

<!DOCTYPE html>
<html lang="zh-CN" >



<head>
  <meta charset="UTF-8">

  <link rel="apple-touch-icon" sizes="76x76" href="/img/Logo.png">
  <link rel="icon" href="/img/Logo.png">
  

  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=5.0, shrink-to-fit=no">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  
  <meta name="theme-color" content="#2f4154">
  <meta name="author" content="向海">
  <meta name="keywords" content="">
  
    <meta name="description" content="一、Principles of cell communicationWays of cell communication: Contact mediated  display molecules on cell surface recognized by receptor on another cell  Non-contact mediated  chemical signal nearby o">
<meta property="og:type" content="article">
<meta property="og:title" content="L15 Overview of Cell Communication">
<meta property="og:url" content="https://zhenyumi.github.io/posts/8e27290e/index.html">
<meta property="og:site_name" content="鹘横海">
<meta property="og:description" content="一、Principles of cell communicationWays of cell communication: Contact mediated  display molecules on cell surface recognized by receptor on another cell  Non-contact mediated  chemical signal nearby o">
<meta property="og:locale" content="zh_CN">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Cell_Biology/Cell_Biology_01.jpg">
<meta property="article:published_time" content="2020-07-08T08:18:00.000Z">
<meta property="article:modified_time" content="2020-07-17T13:36:00.000Z">
<meta property="article:author" content="向海">
<meta property="article:tag" content="Cell Biology">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Cell_Biology/Cell_Biology_01.jpg">
  
  
  
  <title>L15 Overview of Cell Communication - 鹘横海</title>

  <link  rel="stylesheet" href="https://lib.baomitu.com/twitter-bootstrap/4.6.1/css/bootstrap.min.css" />



  <link  rel="stylesheet" href="https://lib.baomitu.com/github-markdown-css/4.0.0/github-markdown.min.css" />

  <link  rel="stylesheet" href="https://lib.baomitu.com/hint.css/2.7.0/hint.min.css" />

  <link  rel="stylesheet" href="https://lib.baomitu.com/fancybox/3.5.7/jquery.fancybox.min.css" />



<!-- 主题依赖的图标库，不要自行修改 -->
<!-- Do not modify the link that theme dependent icons -->

<link rel="stylesheet" href="//at.alicdn.com/t/c/font_1749284_5i9bdhy70f8.css">



<link rel="stylesheet" href="//at.alicdn.com/t/c/font_1736178_k526ubmyhba.css">


<link  rel="stylesheet" href="/css/main.css" />


  <link id="highlight-css" rel="stylesheet" href="/css/highlight.css" />
  




  <script id="fluid-configs">
    var Fluid = window.Fluid || {};
    Fluid.ctx = Object.assign({}, Fluid.ctx)
    var CONFIG = {"hostname":"zhenyumi.github.io","root":"/","version":"1.9.8","typing":{"enable":true,"typeSpeed":70,"cursorChar":"_","loop":false,"scope":["home"]},"anchorjs":{"enable":true,"element":"h1,h2,h3,h4,h5,h6","placement":"left","visible":"hover","icon":""},"progressbar":{"enable":true,"height_px":3,"color":"#29d","options":{"showSpinner":false,"trickleSpeed":100}},"code_language":{"enable":true,"default":"TEXT"},"copy_btn":true,"image_caption":{"enable":true},"image_zoom":{"enable":true,"img_url_replace":["",""]},"toc":{"enable":true,"placement":"right","headingSelector":"h1,h2,h3,h4,h5,h6","collapseDepth":0},"lazyload":{"enable":true,"loading_img":"/img/loading.gif","onlypost":false,"offset_factor":2},"web_analytics":{"enable":false,"follow_dnt":true,"baidu":null,"google":{"measurement_id":null},"tencent":{"sid":null,"cid":null},"leancloud":{"app_id":null,"app_key":null,"server_url":null,"path":"window.location.pathname","ignore_local":false},"umami":{"src":null,"website_id":null,"domains":null,"start_time":"2024-01-01T00:00:00.000Z","token":null,"api_server":null}},"search_path":"/local-search.xml","include_content_in_search":true};

    if (CONFIG.web_analytics.follow_dnt) {
      var dntVal = navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack;
      Fluid.ctx.dnt = dntVal && (dntVal.startsWith('1') || dntVal.startsWith('yes') || dntVal.startsWith('on'));
    }
  </script>
  <script  src="/js/utils.js" ></script>
  <script  src="/js/color-schema.js" ></script>
  


  
<meta name="generator" content="Hexo 7.0.0"></head>


<body>
  

  <header>
    

<div class="header-inner" style="height: 70vh;">
  <nav id="navbar" class="navbar fixed-top  navbar-expand-lg navbar-dark scrolling-navbar">
  <div class="container">
    <a class="navbar-brand" href="/">
      <strong>鹘横海</strong>
    </a>

    <button id="navbar-toggler-btn" class="navbar-toggler" type="button" data-toggle="collapse"
            data-target="#navbarSupportedContent"
            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <div class="animated-icon"><span></span><span></span><span></span></div>
    </button>

    <!-- Collapsible content -->
    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav ml-auto text-center">
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/" target="_self">
                <i class="iconfont icon-home-fill"></i>
                <span>首页</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/archives/" target="_self">
                <i class="iconfont icon-archive-fill"></i>
                <span>归档</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/categories/" target="_self">
                <i class="iconfont icon-category-fill"></i>
                <span>分类</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/tags/" target="_self">
                <i class="iconfont icon-tags-fill"></i>
                <span>标签</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/about/" target="_self">
                <i class="iconfont icon-user-fill"></i>
                <span>关于</span>
              </a>
            </li>
          
        
        
          <li class="nav-item" id="search-btn">
            <a class="nav-link" target="_self" href="javascript:;" data-toggle="modal" data-target="#modalSearch" aria-label="Search">
              <i class="iconfont icon-search"></i>
            </a>
          </li>
          
        
        
      </ul>
    </div>
  </div>
</nav>

  

<div id="banner" class="banner" parallax=true
     style="background: url('https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Cell_Biology/Cell_Biology_02.png') no-repeat center center; background-size: cover;">
  <div class="full-bg-img">
    <div class="mask flex-center" style="background-color: rgba(0, 0, 0, 0.3)">
      <div class="banner-text text-center fade-in-up">
        <div class="h2">
          
            <span id="subtitle">L15 Overview of Cell Communication</span>
          
        </div>

        
          
  <div class="mt-3">
    
    
      <span class="post-meta">
        <i class="iconfont icon-date-fill" aria-hidden="true"></i>
        <time datetime="2020-07-08 16:18" pubdate>
          2020年7月8日 下午
        </time>
      </span>
    
  </div>

  <div class="mt-1">
    
      <span class="post-meta mr-2">
        <i class="iconfont icon-chart"></i>
        
          893 字
        
      </span>
    

    
      <span class="post-meta mr-2">
        <i class="iconfont icon-clock-fill"></i>
        
        
        
          8 分钟
        
      </span>
    

    
    
  </div>


        
      </div>

      
    </div>
  </div>
</div>

</div>

  </header>

  <main>
    
      

<div class="container-fluid nopadding-x">
  <div class="row nomargin-x">
    <div class="side-col d-none d-lg-block col-lg-2">
      

    </div>

    <div class="col-lg-8 nopadding-x-md">
      <div class="container nopadding-x-md" id="board-ctn">
        <div id="board">
          <article class="post-content mx-auto">
            <h1 id="seo-header">L15 Overview of Cell Communication</h1>
            
            
              <div class="markdown-body">
                
                <h1 id="一、Principles-of-cell-communication"><a href="#一、Principles-of-cell-communication" class="headerlink" title="一、Principles of cell communication"></a>一、Principles of cell communication</h1><p>Ways of cell communication:</p>
<p>Contact mediated</p>
<ul>
<li>display molecules on cell surface</li>
<li>recognized by receptor on another cell</li>
</ul>
<p>Non-contact mediated</p>
<ul>
<li>chemical signal</li>
<li>nearby or at a distance</li>
</ul>
<blockquote>
<p>Many bacteria, for example, respond to chemical signals that are secreted by their neighbors and accumulate at higher population density. This process, called <strong>quorum sensing,</strong> allows bacteria to coordinate their behavior, including their <strong>motility</strong>, <strong>antibiotic production</strong>, <strong>spore formation</strong>, and <strong>sexual conjugation</strong></p>
<p>Communication between cells in multicellular organisms is mediated mainly by <strong>extracellular signal molecules.</strong></p>
</blockquote>
<h2 id="Signals-and-receptors"><a href="#Signals-and-receptors" class="headerlink" title="Signals and receptors"></a>Signals and receptors</h2><p>A signaling cell: signaling molecules</p>
<p>A responding cell: receptors</p>
<h3 id="1-Steps-in-cell-signaling"><a href="#1-Steps-in-cell-signaling" class="headerlink" title="1. Steps in cell signaling"></a>1. Steps in cell signaling</h3><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424213047853.png" srcset="/img/loading.gif" lazyload alt="image-20200424213047853" style="zoom: 33%;" />

<p>The functional process of signaling molecule </p>
<ol>
<li>Synthesis</li>
<li>Release by signaling cell</li>
<li>Transport to target cell</li>
<li>Detection by a specific receptor protein</li>
<li>Change by receptor-signal complex (trigger)</li>
</ol>
<h3 id="2-Type-of-extracellular-signals-form-or-molecules"><a href="#2-Type-of-extracellular-signals-form-or-molecules" class="headerlink" title="2. Type of extracellular signals (form or molecules)"></a>2. Type of extracellular signals (form or molecules)</h3><p>One of the key challenges in cell biology is to understand how a cell reacts to and integrates different signals in order to make decisions: to divide, to move, to differentiate, and so on…</p>
<p>Origin:</p>
<ul>
<li>External signal</li>
<li>Internal signal</li>
</ul>
<p>Physical:</p>
<ul>
<li>Light</li>
<li>mechanical force</li>
<li>Heat</li>
</ul>
<p>Chemical:</p>
<ol>
<li><p>water soluble chemical</p>
</li>
<li><p>lipid soluble (diffuses across membrane) chemical</p>
</li>
</ol>
<ul>
<li>proteins, peptides, amino acid derivatives</li>
<li>Nucleotides</li>
<li>steroids, retinoids, fatty acid derivatives</li>
<li>gases( NO, CO), etc.</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424213418609.png" srcset="/img/loading.gif" lazyload alt="image-20200424213418609"></p>
<blockquote>
<p><u>Reception of the signals depends on receptor proteins, usually (but not always) at the cell surface, which bind the signal molecule</u>. </p>
<p>The binding activates the receptor, which in turn activates one or more <strong>intracellular signaling pathways</strong> or <strong>systems</strong>. These systems  depend on <strong>intracellular signaling proteins</strong>, which process the signal inside the receiving cell and distribute it to the appropriate intracellular targets. </p>
<p>The targets that lie at the end of signaling pathways are generally called <strong>effector proteins</strong>,  which are <u>altered in some way</u> by the incoming signal and <u>implement the appropriate change in cell behavior</u>. </p>
<ul>
<li>Depending on the <strong>signal</strong> and the <strong>type</strong> and state of  the receiving cell, these effectors can be <strong>transcription regulators</strong>, <strong>ion channels</strong>,  <strong>components of a metabolic pathway</strong>, or parts of the cytoskeleton</li>
</ul>
</blockquote>
<blockquote>
<p>In most cases, however, signaling cells secrete signal molecules into the <strong>extracellular fluid</strong>. Often, the secreted molecules are <strong>local mediators</strong>, which <u>act only on cells in the local environment of the signaling cell.</u> This is called <strong>paracrine（旁分泌） signaling</strong> (Figure 15–2B). Usually, the signaling and target cells in paracrine signaling are of <u>different cell types</u>, but cells may also produce signals that they  themselves respond to: this is referred to as <strong>autocrine（自分泌） signaling.</strong></p>
<p>A quite different strategy for signaling over long distances makes use of <strong>endocrine cells</strong>, which secrete their signal molecules, called <strong>hormones</strong>, into the <u>bloodstream</u>. The blood carries the molecules <u>far and wide</u>, allowing them to act on target cells that may lie anywhere in the body</p>
</blockquote>
<h3 id="3-Diverse-receptors"><a href="#3-Diverse-receptors" class="headerlink" title="3. Diverse receptors"></a>3. Diverse receptors</h3><blockquote>
<p>Many signal molecules act at very <u>low concentrations</u> (typically ≤ 10^–8^ M), and their receptors usually bind them with high affinity (dissociation constant K<del>d</del> ≤ 10^–8^ M; see Figure 3–44).</p>
<p>In most cases, receptors are <strong>transmembrane proteins</strong> on the target-cell surface</p>
<p>In other cases, the receptor proteins are <strong>inside the target cell</strong>, and the signal molecule has to <u>enter the cell to bind to them</u>: this requires that the signal molecule be sufficiently <u>small and hydrophobic</u> to diffuse across the target cell’s plasma membrane</p>
</blockquote>
<blockquote>
<p>Each Cell Is Programmed to Respond to Specific Combinations of Extracellular Signals</p>
<ul>
<li>Many cells, for example, require a specific combination of extracellular survival factors to allow the cell to continue living; when deprived of these signals, the cell activates a suicide program and kills itself—usually by <strong>apoptosis</strong>, a form of <strong>programmed cell death</strong></li>
<li><strong>differentiation</strong> into a nondividing state (called <strong>terminal differentiation</strong>) frequently requires a different combination of <strong>survival and differentiation signals</strong> that must override any signal to divide.</li>
</ul>
<p>A signal molecule often has different effects on different types of target cells</p>
<ul>
<li>The different effects of acetylcholine (乙酰胆碱) in these cell types result from differences in the <strong>intracellular signaling proteins</strong>, <strong>effector proteins</strong>, and <u>genes that are activated</u>. Thus, an extracellular signal itself has little information content; it simply induces the cell to respond according to its <u>predetermined state</u>, which depends on the cell’s <u>developmental history</u> and the <u>specific genes it expresses</u>.</li>
</ul>
</blockquote>
<p>receptor protein, not a signal molecule, provides a response</p>
<p>Most of signals are proteins which are larger or not hydrophobic and cannot go through the membrane.</p>
<ul>
<li>They bind to membrane receptors activating <strong>intracellular signaling molecules.</strong></li>
</ul>
<p>Two types of cell signaling receptors, <strong>cell-surface receptors</strong> and <strong>intracellular receptors</strong>, which work differently.</p>
<p>Signals impact on effector proteins to alter <u>metabolism, change cell shape or movement or alter gene expression</u>.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426140844296.png" srcset="/img/loading.gif" lazyload alt="image-20200426140844296"></p>
<p>The same signals can trigger different effects, depending on the cell type</p>
<blockquote>
<p>There Are Three Major Classes of Cell-Surface Receptor Proteins</p>
<p>These cell-surface receptors act as <strong>signal transducers</strong> by <u>converting an extracellular ligand-binding event into intracellular signals that alter the behavior of the target cell.</u></p>
<ol>
<li><strong>Ion-channel-coupled receptors</strong>, also known as <strong>transmitter-gated ion channels</strong> or <strong>ionotropic receptors</strong>, are involved in <strong>rapid synaptic signaling</strong> between nerve cells and other electrically excitable target cells such as nerve and muscle cells</li>
<li><strong>G-protein-coupled receptors</strong> act by <strong>indirectly regulating</strong> the activity of a  <strong>separate plasma-membrane-bound target protein</strong>, which is generally either an enzyme or an ion channel. A <strong>trimeric GTP-binding protein</strong> (G protein) mediates the interaction between the <strong>activated receptor</strong> and this <strong>target protein</strong></li>
<li><strong>Enzyme-coupled receptors</strong> either function as enzymes or associate directly with enzymes that they activate (Figure 15–6C). They are usually <strong>single-pass</strong> transmembrane proteins that have their <strong>ligand-binding site</strong> outside the cell and their catalytic or enzyme-binding site inside</li>
</ol>
<ul>
<li>There are also some types of cell-surface receptors that <u>do not fit easily into any of these classes</u> but have important functions in controlling the <strong>specialization</strong> of different cell types <u>during development</u> and in <strong>tissue renewal</strong> and <strong>repair</strong> in adults</li>
</ul>
<ol>
<li><strong>Signal processing</strong> can convert a simple signal into a complex response. <ul>
<li>In many systems, for example, a gradual increase in an extracellular signal is converted into an <strong>abrupt, switch-like response</strong>. </li>
<li>In other cases, a simple input signal is converted into an <strong>oscillatory response</strong>, produced by a repeating series of transient intracellular signals. </li>
<li><strong>Feedback</strong> usually lies at the heart of <u>biochemical switches and oscillators</u>, as we describe later.</li>
</ul>
</li>
<li><strong>Integration</strong> allows a response to be governed by multiple inputs. As discussed earlier, for example, <u>specific combinations of extracellular signals</u>  are generally required to stimulate complex cell behaviors such as cell survival and proliferation (see Figure 15–4). The cell therefore has to <u>integrate information coming from multiple signals</u>, which often depends on intracellular <strong>coincidence detectors</strong>; these proteins are equivalent to <strong>AND gates</strong> in the microprocessor of a computer, in that they are only activated if they  receive multiple converging signals</li>
<li><strong>Coordination</strong> of multiple responses in one cell can be achieved by a single extracellular signal. Some extracellular signal molecules, for example, stimulate a cell to both grow and divide. This coordination generally depends on <u>mechanisms for distributing a signal to multiple effectors, by creating branches</u> in the signaling pathway. <ul>
<li>In some cases, the branching of signaling pathways can allow one signal to modulate the strength of a response to other signals</li>
</ul>
</li>
</ol>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426140911394.png" srcset="/img/loading.gif" lazyload alt="image-20200426140911394"></p>
<p>Receptor protein, not a signal molecule, provides a response</p>
<p>Depending on receptor, response may be completely different (e.g., in case of acetylcholine) </p>
<ul>
<li>Acetylcholine receptors in heart muscle cells and salivary gland cells are identical.</li>
</ul>
<p>However, they result in different effect or proteins activation</p>
<h2 id="Signal-transduction-relay"><a href="#Signal-transduction-relay" class="headerlink" title="Signal transduction&#x2F;relay"></a>Signal transduction&#x2F;relay</h2><p>Three classes of cell surface receptors </p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426141021653.png" srcset="/img/loading.gif" lazyload alt="image-20200426141021653"></p>
<p>Cell surface receptors rely on intracellular signaling proteins to relay the signal!</p>
<blockquote>
<p>Cell-Surface Receptors Relay Signals Via Intracellular Signaling Molecules</p>
<p><u>Some intracellular signaling molecules are small chemicals, which are often called <strong>second messengers</strong> (the “first messengers” being the extracellular signals). They are generated in large amounts in response to receptor activation and diffuse away from their source, spreading the signal to other parts of the cell</u>. </p>
<p>Some, such as <font color="EE0000"><strong>cyclic AMP</strong></font> and <font color="EE0000"><strong>Ca2+</strong></font>, are <strong>water-soluble</strong> and diffuse in the cytosol, while others, such as diacylglycerol, are lipid-soluble and diffuse in the plane of the plasma membrane. In either case, they pass the signal on by <u>binding to and altering the behavior of selected signaling or effector proteins.</u></p>
<ul>
<li>Many of these proteins behave like <strong>molecular switches</strong>. When they receive a signal, they switch from an inactive to an active state, until another process switches them off, returning them to their inactive state.</li>
</ul>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426141045379.png" srcset="/img/loading.gif" lazyload alt="image-20200426141045379" style="zoom: 43%;" />

<h3 id="1-second-messenger"><a href="#1-second-messenger" class="headerlink" title="1. second messenger"></a>1. second messenger</h3><p>Signaling pathways often contain two different types of messengers: </p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426143500292.png" srcset="/img/loading.gif" lazyload alt="image-20200426143500292" style="zoom:25%;" />

<center>cAMP</center>

<ul>
<li><p>The first messenger molecules: extracellular signals (signaling molecule)</p>
</li>
<li><p>The second messenger molecules:</p>
<ul>
<li>small molecules which are generated in large numbers after the receptor activation. </li>
<li>They are either hydrophilic or lipid diffusing:<ul>
<li>Cyclic nucleotides: cAMP, cGMP</li>
<li>Ca2+</li>
<li>Diacylglycerol (DAG): modified lipid activates PKC</li>
<li>Inositol triphosphate (IP<del>3</del>)</li>
</ul>
</li>
<li>Second messenger work on effector proteins and they relay the signals.</li>
</ul>
</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426143533577.png" srcset="/img/loading.gif" lazyload alt="image-20200426143533577" style="zoom:25%;" />

<center>cGMP</center>

<h3 id="2-Intracellular-signaling-pathways"><a href="#2-Intracellular-signaling-pathways" class="headerlink" title="2. Intracellular signaling pathways"></a>2. Intracellular signaling pathways</h3><p>Relay signal forward</p>
<ul>
<li>Scaffold, anchor</li>
<li>Amplify</li>
<li>Integrate multiple signals</li>
<li>Distribute signals to more than one effector</li>
<li>Modulate the activity of signaling proteins</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426143711002.png" srcset="/img/loading.gif" lazyload alt="image-20200426143711002" style="zoom:33%;" />

<p>Steps of intracellular signaling pathway:</p>
<ol>
<li>activated by an extracellular signal (binding to a receptor protein)</li>
<li>The receptor activates one or more intracellular signaling pathways. (involving a series of signaling proteins)</li>
<li>Finally, one or more of the intracellular signaling proteins alter the activity of effector proteins and thereby the behavior of the cell.</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426143806294.png" srcset="/img/loading.gif" lazyload alt="image-20200426143806294"></p>
<p>Many intracellular signal molecules have active and inactive states</p>
<p>The switch between forms is due to </p>
<ul>
<li>Protein phosphorylation (Kinases (serine&#x2F;threonine or tyrosine) attach phosphate, phosphatases dethatch phosphate)</li>
<li>GTP-binding </li>
<li>cAMP or Ca2+ binding</li>
<li>Ubiquitination, etc.</li>
</ul>
<blockquote>
<p>Intracellular Signals Must Be Specific and Precise in a Noisy Cytoplasm</p>
<ul>
<li>The binding of a signaling molecule to the correct target is determined by <strong>precise and complex interactions</strong> between <strong>complementary surfaces</strong> on the two molecules</li>
<li>Another important way that cells avoid responses to unwanted background signals depends on the ability of many downstream target proteins to <u>simply ignore such signals</u>. These proteins respond only when the upstream signal reaches a <strong>high concentration or activity level</strong></li>
</ul>
<p>In these and other ways, intracellular signaling systems <u>filter out noise</u>, generating little or no response to low levels of stimuli</p>
</blockquote>
<blockquote>
<p><u>The largest class of molecular switches consists of proteins that are activated  or inactivated by <strong>phosphorylation</strong></u> (discussed in Chapter 3). For these proteins, the switch is thrown in one direction by a <strong>protein kinase</strong>, which <u>covalently adds one or more phosphate groups to specific amino acids</u> on the signaling protein,  and in the other direction by a <strong>protein phosphatase</strong>, which removes the phosphate groups</p>
<ul>
<li>There are <strong>two main types</strong> of protein kinase. <ol>
<li>The great majority are <strong>serine&#x2F;threonine kinases</strong>, which phosphorylate the <u>hydroxyl groups (-OH) of serines and threonines</u> in their targets. </li>
<li>Others are <strong>tyrosine kinases</strong>, which phosphorylate proteins on tyrosines</li>
</ol>
</li>
</ul>
</blockquote>
<h4 id="Protein-phosphorylation-by-kinases"><a href="#Protein-phosphorylation-by-kinases" class="headerlink" title="Protein phosphorylation by kinases"></a>Protein phosphorylation by kinases</h4><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426143903235.png" srcset="/img/loading.gif" lazyload alt="image-20200426143903235" style="zoom:67%;" />

<p>It is one major way of post-translational modification to regulate protein activity<br>$&gt;$ 30% of all human genome proteins can be phosphorylated<br>$&gt;$ 520 human kinases (kinome)<br>$&gt;$ 150 protein phosphatases</p>
<p>Two categories: Serine&#x2F;Threonine kinase; Tyrosine kinase</p>
<p>Protein kinases are major therapeutic targets in human diseases:</p>
<ul>
<li>e.g. : Acute leukemia:</li>
<li>Gleevec targets BCR-ABL kinase (Uncontrolled Cell Growth)</li>
</ul>
<h4 id="GEF-and-GAP"><a href="#GEF-and-GAP" class="headerlink" title="GEF and GAP"></a>GEF and GAP</h4><blockquote>
<p>The other important class of molecular switches consists of <u><strong>GTP-binding proteins</strong> (discussed in Chapter 3). These proteins switch between an “on” (actively  signaling) state when GTP is bound and an “off” state when GDP is bound. </u></p>
<p><u>In the “on” state, they usually have <strong>intrinsic GTPase activity</strong> and shut themselves off by hydrolyzing their bound GTP to GDP</u> (Figure 15–7B). There are two major types of GTP-binding proteins. </p>
<ol>
<li>Large, <strong>trimeric GTP-binding proteins</strong> (also called <strong>G proteins</strong>) help relay signals from G-protein-coupled receptors that activate them (see  Figure 15–6B). </li>
<li>Small <strong>monomeric GTPases</strong> (also called <strong>monomeric GTP-binding  proteins</strong> help relay signals from many classes of cell-surface receptors.</li>
</ol>
<p>Specific regulatory proteins control both types of GTP-binding proteins. <strong>GTPase-activating proteins (GAPs)</strong> <u>drive the proteins into an “off” state by increasing the <strong>rate</strong> of hydrolysis</u> of bound GTP. Conversely, <strong>guanine nucleotide exchange factors (GEFs)</strong> <u>activate GTP-binding proteins by promoting the <strong>release</strong> of bound GDP</u>, which allows a new GTP to bind. </p>
<p>In the case of trimeric G proteins,  the <strong>activated receptor</strong> serves as the <strong>GEF</strong>. </p>
</blockquote>
<blockquote>
<p>Not all molecular switches in signaling systems depend on phosphorylation or GTP binding</p>
<ul>
<li>We see later that some signaling proteins are switched on or off by the binding of another signaling protein or a second messenger such as <strong>cyclic AMP or Ca2+</strong>, or by <strong>covalent modifications</strong> other than phosphorylation or dephosphorylation, such as <strong>ubiquitylation</strong></li>
<li>Most signaling pathways contain <strong>inhibitory steps</strong>, and a sequence of <u>two inhibitory steps can have the same effect as one activating step</u> (Figure 15–9). This <strong>double-negative activation</strong>  is very common in signaling systems</li>
</ul>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144031209.png" srcset="/img/loading.gif" lazyload alt="image-20200426144031209" style="zoom:40%;" />

<h3 id="3-signaling-complexes"><a href="#3-signaling-complexes" class="headerlink" title="3. signaling complexes"></a>3. signaling complexes</h3><blockquote>
<p>One simple and effective strategy for <strong>enhancing the specificity of interactions</strong>  between signaling molecules is to <u>localize them in the same part</u> of the cell or even within large protein complexes, thereby <u>ensuring that they interact only with each other</u> and not with inappropriate partners. </p>
<p>Such mechanisms often involve <strong>scaffold proteins</strong>, which bring together groups of interacting signaling proteins into <strong>signaling complexes</strong>, often <u>before</u> a signal has been received</p>
</blockquote>
<p>High speed and specificity in signaling is achieved by the formation of “signaling complexes”</p>
<p>Simply bringing intracellular signaling proteins together into close proximity is sometimes sufficient to activate them. </p>
<ul>
<li>A signaling complex brings all the different molecules that interact with each other in close proximity. This greatly <u>reduces the time</u> the components would need to “find each other”.</li>
<li><u>Induced proximity</u>, where a signal triggers assembly of a signaling complex, is commonly used to relay signals from protein to protein along a signaling pathway.</li>
</ul>
<p>Signaling complexes can be established in different ways:</p>
<ul>
<li>Assembly of the complex via scaffolding proteins on an inactive receptor<ul>
<li>Can be pre-regulated</li>
<li>Specific region</li>
</ul>
</li>
<li>Assembly of the complex on an activated receptor</li>
<li>Assembly of the complex at the membrane on phosphoinositide docking sites</li>
</ul>
<h4 id="Pre-formed-“signaling-complex”"><a href="#Pre-formed-“signaling-complex”" class="headerlink" title="Pre-formed “signaling complex”"></a>Pre-formed “signaling complex”</h4><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144328401.png" srcset="/img/loading.gif" lazyload alt="image-20200426144328401" style="zoom: 33%;" />

<ul>
<li>Receptor and some intracellular signaling proteins to be activated in sequence are <strong>pre-assembled</strong> into a signaling complex on the inactive receptor by a <strong>scaffold protein</strong>.</li>
</ul>
<h4 id="Assembly-of-signaling-complex-on-an-activated-receptor"><a href="#Assembly-of-signaling-complex-on-an-activated-receptor" class="headerlink" title="Assembly of signaling complex on an activated receptor"></a>Assembly of signaling complex on an activated receptor</h4><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144403437.png" srcset="/img/loading.gif" lazyload alt="image-20200426144403437"></p>
<ul>
<li>A signaling complex assembles transiently on a receptor only after the binding of an extracellular signal molecule has activated the receptor.</li>
<li>The activated receptor phosphorylates itself at multiple sites, which then act as docking sites for intracellular signaling proteins.</li>
</ul>
<h4 id="Assembly-of-signaling-complex-on-phosphoinositide-docking-sites"><a href="#Assembly-of-signaling-complex-on-phosphoinositide-docking-sites" class="headerlink" title="Assembly of signaling complex on phosphoinositide docking sites"></a>Assembly of signaling complex on phosphoinositide docking sites</h4><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144514145.png" srcset="/img/loading.gif" lazyload alt="image-20200426144514145" style="zoom: 25%;" />

<ul>
<li>Activation of a receptor leads to the phosphorylation of specific <strong>phospholipids</strong> (<strong>phosphoinositides</strong>) in the adjacent plasma membrane.</li>
<li>These phosphoinositides then serve as docking sites for specific intracellular signaling proteins 1 &amp; 2, which can now interact with each other.</li>
</ul>
<h3 id="4-Interaction-Domains"><a href="#4-Interaction-Domains" class="headerlink" title="4. Interaction Domains"></a>4. Interaction Domains</h3><blockquote>
<p><strong>Induced proximity</strong>, where a signal triggers assembly of a signaling complex, is commonly used to <u>relay signals</u> from  protein to protein along a signaling pathway. The assembly of such signaling complexes depends on various highly conserved, small interaction domains, which are found in many <strong>intracellular signaling</strong> proteins.</p>
<p>There are many types of interaction domains in signaling proteins. </p>
<ul>
<li><p><font color="EE0000">Src homology 2 (SH2) domains</font> and <font color="EE0000">phosphotyrosine-binding (PTB) domains</font>, for example, bind to <u><strong>phosphorylated tyrosines</strong> in a particular peptide sequence</u> on activated receptors or intracellular signaling proteins. </p>
</li>
<li><p><font color="EE0000">Src homology 3 (SH3) domains</font> bind to <u>short, proline-rich amino acid sequences</u>. </p>
</li>
<li><p>Some <font color="EE0000">pleckstrin homology (PH)  domains</font> bind to the <u>charged head groups of specific phosphoinositides</u> that are produced in the plasma membrane in response to an extracellular signal;</p>
</li>
</ul>
<p>They enable the protein they are part of to dock on the membrane and interact with other similarly recruited signaling proteins</p>
<ul>
<li>它们使它们所属的蛋白能够停靠在膜上并与其他类似募集的信号蛋白相互作用</li>
</ul>
<p>Some signaling  proteins consist solely of two or more interaction domains and function only as <strong>adaptors</strong> to link two other proteins together in a signaling pathway.</p>
<p>Another way of bringing receptors and intracellular signaling proteins together is to <u>concentrate them in a specific region of the cell</u></p>
</blockquote>
<p>Conserved interaction domains are important for protein binding which modulates intracellular signaling proteins</p>
<ol>
<li><p><font color="EE0000">Src homology 2 (<strong>SH2</strong>) domains</font> and <font color="EE0000">Phosphotyrosine-binding (<strong>PTB</strong>) domains</font> - Bind <strong>phosphotyrosine</strong> (p-Y)</p>
</li>
<li><p><font color="EE0000">Src homology 3 (<strong>SH3</strong>) domains</font>: bind <strong>proline</strong> rich domains</p>
</li>
<li><p><font color="EE0000">Pleckstrin homology (<strong>PH</strong>) domains</font>: bind <strong>phosphoinositides</strong></p>
</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144647524.png" srcset="/img/loading.gif" lazyload alt="image-20200426144647524"></p>
<h3 id="5-Signaling-Cascade"><a href="#5-Signaling-Cascade" class="headerlink" title="5. Signaling Cascade"></a>5. Signaling Cascade</h3><blockquote>
<p>Many intracellular signaling proteins controlled by phosphorylation are themselves <strong>protein kinases</strong>, and these are often organized into <strong>kinase cascades</strong>. In such a cascade, one protein kinase, activated by phosphorylation, phosphorylates the next protein kinase in the sequence, and so on, <u>relaying</u> the signal onward and, in some cases, <u>amplifying</u> it or <u>spreading</u> it to other signaling pathways.</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144743105.png" srcset="/img/loading.gif" lazyload alt="image-20200426144743105" style="zoom: 50%;" />



<h4 id="Phosphorylation-cascade"><a href="#Phosphorylation-cascade" class="headerlink" title="Phosphorylation cascade"></a>Phosphorylation cascade</h4><p>Signaling protein itself is a kinase which can phosphorylate and activate downstream effectors</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144812501.png" srcset="/img/loading.gif" lazyload alt="image-20200426144812501"></p>
<ol>
<li><p>The <font color="0000EE"><strong>MAP kinase module</strong></font> is activated by <font color="EE0000"><strong>Ras</strong></font>. </p>
<ul>
<li>The three-component module begins with the <font color="0000EE"><strong>MAP kinase kinase kinase</strong></font> <font color="EE0000"><strong>Raf</strong></font>. </li>
<li>Ras recruits Raf to the plasma membrane and helps to activate it.</li>
</ul>
</li>
<li><p><font color="EE0000"><strong>Raf</strong></font> activates the <font color="0000EE"><strong>MAP kinase kinase</strong></font> <font color="EE0000"><strong>MEK</strong></font>, which activates the <font color="0000EE"><strong>MAP kinase</strong></font> <font color="EE0000"><strong>Erk</strong></font>. </p>
</li>
<li><p><font color="EE0000"><strong>Erk</strong></font> in turn phosphorylates a variety of <strong>downstream proteins</strong>, including other protein kinases, as well as transcription regulators in the nucleus.</p>
</li>
</ol>
<p>The resulting changes in protein activities and gene expression cause complex changes in cell behavior</p>
<h3 id="6-Signal-integration"><a href="#6-Signal-integration" class="headerlink" title="6. Signal integration"></a>6. Signal integration</h3><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144934995.png" srcset="/img/loading.gif" lazyload alt="image-20200426144934995" style="zoom: 50%;" />

<ul>
<li>Coincidence of multiple signals</li>
</ul>
<h3 id="7-Characteristics-of-signal-transduction"><a href="#7-Characteristics-of-signal-transduction" class="headerlink" title="7. Characteristics of signal transduction"></a>7. Characteristics of signal transduction</h3><blockquote>
<p>The Relationship Between Signal and Response Varies in Different Signaling Pathways</p>
<p>The function of an intracellular signaling system is to <u>detect</u> and <u>measure</u> a specific stimulus in one location of a cell and then <u>generate</u> an appropriately <u>timed and measured response</u> at another location</p>
<ol>
<li><p><strong>Response timing</strong> varies dramatically <u>in different signaling systems,</u> according to the <u>speed required for the response</u>. </p>
<ul>
<li>In some cases, such as synaptic signaling (see Figure 15–2C), the response can occur within milliseconds.  </li>
<li>In other cases, as in the control of cell fate by morphogens during development, a full response can require hours or days.</li>
</ul>
</li>
<li><p><strong>Sensitivity</strong> to extracellular signals can vary greatly. </p>
<ul>
<li><strong>Hormones</strong> tend to act at <u>very low concentrations</u> on their distant target cells, which are therefore highly sensitive to low concentrations of signal. </li>
<li><strong>Neurotransmitters</strong>, on the other hand, operate at <u>much higher concentrations at a synapse</u>, reducing the need for high sensitivity in postsynaptic receptors. </li>
<li>Sensitivity is often controlled by changes in the <strong>number</strong> or <strong>affinity</strong> of the receptors on the target cell. A particularly important mechanism for <u>increasing the sensitivity  of a signaling system is signal <strong>amplification</strong>,</u> whereby a small number of  activated cell-surface receptors evoke a large intracellular response either by producing large amounts of a second messenger or by activating many copies of a downstream signaling protein.</li>
</ul>
</li>
<li><p><strong>Dynamic range</strong> of a signaling system is <u>related to its sensitivity</u>. Some systems, like those involved in simple developmental decisions, are responsive over a narrow range of extracellular signal concentrations. Other systems,  like those controlling vision or the metabolic response to some hormones, are highly responsive over a much broader range of signal strengths. We will see that broad dynamic range is often achieved by <strong>adaptation mechanisms</strong> that adjust the responsiveness of the system according to the prevailing amount of signal.</p>
<hr>
</li>
<li><p><strong>Persistence</strong> of a response can vary greatly. A transient response of less than a second is appropriate in some synaptic responses, for example, while a prolonged or even permanent response is required in cell fate decisions during development. </p>
<p><u>Numerous mechanisms, including positive feedback, can be used to alter the duration and reversibility of a response.</u></p>
</li>
<li><p><u><strong>Signal processing</strong> can convert a simple signal into a complex response.</u> In many systems, for example, a gradual increase in an extracellular signal is converted into an abrupt, switch-like response. In other cases, a simple input signal is converted into an oscillatory response, produced by a  repeating series of transient intracellular signals. Feedback usually lies at the heart of biochemical switches and oscillators, as we describe later.</p>
</li>
<li><p><strong>Integration</strong> allows a response to be governed by multiple inputs. As discussed earlier, for example, specific combinations of extracellular signals  are generally required to stimulate complex cell behaviors such as cell survival and proliferation (see Figure 15–4). <u>The cell therefore has to integrate  information coming from multiple signals, which often depends on <strong>intracellular coincidence detectors</strong>;</u> these proteins are equivalent to AND gates in the microprocessor of a computer, in that they are only activated if they  receive multiple converging signals</p>
</li>
<li><p><u><strong>Coordination</strong> of multiple responses in one cell can be achieved by a single  extracellular signal.</u> Some extracellular signal molecules, for example, stimulate a cell to both grow and divide. This <u>coordination generally depends  on mechanisms for distributing a signal to multiple effectors, by creating  branches in the signaling pathway.</u> In some cases, the branching of signaling pathways can allow one signal to modulate the strength of a response to  other signals.</p>
</li>
</ol>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145021852.png" srcset="/img/loading.gif" lazyload alt="image-20200426145021852"></p>
<h4 id="Speed-of-responses-slow-or-fast-response"><a href="#Speed-of-responses-slow-or-fast-response" class="headerlink" title="Speed of responses: slow or fast response"></a>Speed of responses: slow or fast response</h4><blockquote>
<p>The Speed of a Response Depends on the Turnover of Signaling  Molecules</p>
<p>The <strong>speed</strong> of any signaling response depends on the nature of the intracellular signaling molecules that <u>carry out the target cell’s response</u>. When the response requires <u>only changes in proteins already present in the cell,</u> it can occur very rapidly</p>
<p>When the response involves changes in <strong>gene expression</strong> and the <u>synthesis of new proteins</u>, however, it usually requires many minutes or  hours, regardless of the mode of signal delivery</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145207682.png" srcset="/img/loading.gif" lazyload alt="image-20200426145207682" style="zoom:67%;" />

<p><strong>Fast response</strong>: alter protein function</p>
<ul>
<li>Alteration of <u>protein function</u> (i.e. activation&#x2F;inactivation&#x2F;modification) is immediate (i.e. seconds or less) upon signal reception, leading to fast responses.</li>
</ul>
<blockquote>
<p>These <strong>inactivation processes</strong> play a crucial part in determining the magnitude, rapidity, and duration of the response.</p>
</blockquote>
<p><strong>Slow response</strong>: through transcription&#x2F;translation</p>
<ul>
<li>Changes in <u>gene expression and protein synthesis</u> (i.e. de novo protein) start 1 hour or later after signal reception, resulting in slow responses.</li>
</ul>
<p>Proteins that have higher turn over rate react to stimuli in a faster manner</p>
<ul>
<li>The amount and the activity of signaling molecules are important!</li>
<li>The importance of fast turnover (synthesis&#x2F;degradation).</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145412235.png" srcset="/img/loading.gif" lazyload alt="image-20200426145412235" style="zoom: 33%;" />

<ol>
<li>high turn over: fast&#x2F;high synthesis rate and quick degradation</li>
<li>low turn over: slow&#x2F;low synthesis rate and slow degradation</li>
</ol>
<blockquote>
<p><strong>turnover rate</strong> can determine the <strong>promptness</strong> of the response when an extracellular signal arrives.</p>
</blockquote>
<p>Many proteins in signaling have short half lives:</p>
<ul>
<li>ensures&#x2F;allows for quicker responses</li>
<li>allows high frequency of the signals(…similar to the low affinity binding of the receptors)</li>
</ul>
<p>Many signaling proteins undergo conversion between inactive and active states:</p>
<ul>
<li>ensures&#x2F;allows for quicker responses than de novo protein synthesis</li>
</ul>
<blockquote>
<p>Cells Can Respond Abruptly to a Gradually Increasing Signal</p>
<p><u>Some signaling systems are capable of generating a <strong>smoothly graded response</strong> over a wide range of extracellular signal concentrations</u> (Figure 15–15, blue line);  such systems are useful, for example, in the <u>fine tuning of metabolic processes</u>  by some hormones. <u>Other signaling systems generate significant responses only when the signal concentration rises <strong>beyond some threshold value</strong>.</u> These abrupt responses are of two types. </p>
<ol>
<li>One is a <strong>sigmoidal response</strong>, in which <u>low concentrations of stimulus do not have much effect</u>, but then the response <u>rises steeply</u> and  continuously at intermediate stimulus levels (Figure 15–15, red line). Such systems provide a <strong>filter</strong> to reduce inappropriate responses to low-level background <strong>signals</strong> but respond with high sensitivity when the stimulus falls within a small range of physiological signal concentrations. </li>
<li>A second type of abrupt response is the <strong>discontinuous</strong> or <strong>all-or-none response</strong>, in which the response switches  on completely (and often irreversibly) when the signal reaches some <strong>threshold concentration</strong> (Figure 15–15, green line). Such responses are particularly useful for controlling the choice <u>between two alternative cell states</u>, and they generally involve <strong>positive feedback,</strong> as we describe in more detail shortly.</li>
</ol>
</blockquote>
<blockquote>
<p>A similar sharpening of response is seen when the activation of an intracellular signaling protein requires <u>phosphorylation at more than one site</u>. Such responses <u>become sharper</u> as the <u>number of required molecules or phosphate groups increases</u>, and if the number is large  enough, responses become almost all-or-none </p>
<p>Responses are also <u>sharpened</u> when an intracellular signaling molecule <u>activates</u> one enzyme and also <u>inhibits</u> another enzyme that catalyzes the <u>opposite</u> reaction</p>
</blockquote>
<h2 id="Positive-and-negative-feedback"><a href="#Positive-and-negative-feedback" class="headerlink" title="Positive and negative feedback"></a>Positive and negative feedback</h2><p>Signaling can cause both, an “all-or-none” response or a “smoothly-graded” response”</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145442277.png" srcset="/img/loading.gif" lazyload alt="image-20200426145442277" style="zoom:45%;" />

<h4 id="“All-or-none”-response"><a href="#“All-or-none”-response" class="headerlink" title="“All-or-none” response"></a>“All-or-none” response</h4><p>Cooperative responses that reach a certain threshold:</p>
<ul>
<li>e. g. 4 cAMPs binding to <font color="EE0000">PKA</font> and multiple sites of phosphorylation</li>
</ul>
<p>Concerted effects that activate one enzyme and simultaneous inhibit another enzyme catalyzing the opposing reaction</p>
<p>Positive feedback can generate an “all-or none” response.</p>
<h3 id="1-Positive-feedback-gives-switch-like-response"><a href="#1-Positive-feedback-gives-switch-like-response" class="headerlink" title="1. Positive feedback gives switch-like response"></a>1. Positive feedback gives switch-like response</h3><blockquote>
<p>Positive Feedback Can Generate an All-or-None Response</p>
<p>In <strong>positive feedback</strong>, the output stimulates its own production; in <strong>negative feedback</strong>, the output inhibits its own production</p>
<p>Those that regulate cell signaling can either operate  exclusively <u>within</u> the target cell or <u>involve the secretion</u> of extracellular signals.</p>
<p>If the positive feedback is of only <strong>moderate strength</strong>, its effect will  be simply to steepen the response to the signal, generating a <strong>sigmoidal response</strong>  like those described earlier; but if the feedback is <strong>strong enough</strong>, it can produce  an <strong>all-or-none response</strong></p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145600788.png" srcset="/img/loading.gif" lazyload alt="image-20200426145600788"></p>
<p>A stimulus activates protein A, which, in turn, activates protein B, while protein B then acts back to increase the activity of A.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145627556.png" srcset="/img/loading.gif" lazyload alt="image-20200426145627556" style="zoom: 67%;" />

<h4 id="Effects-of-positive-feedback"><a href="#Effects-of-positive-feedback" class="headerlink" title="Effects of positive feedback"></a>Effects of positive feedback</h4><blockquote>
<p>Through <strong>positive feedback,</strong> a <u>transient</u> extracellular signal can induce <strong>long-term changes</strong> in cells and their progeny that can persist for the lifetime of the organism.</p>
<p>all cells in a population <u>do not respond identically to the same concentration of extracellular signal</u>, especially at intermediate signal concentrations where the receptor is only partially occupied.</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145758273.png" srcset="/img/loading.gif" lazyload alt="image-20200426145758273"></p>
<ul>
<li>Positive feedback amplify the signal</li>
</ul>
<p>Activated E kinase acts back to promote its own phosphorylation and activation!</p>
<p>The signal kinase switches the system from an “off” state to an “on” state. With the positive feedback, the “on” status persists - even after the stimulus has stopped.</p>
<p>Positive feedback converts a short signal in a long-term effects even after the stimulus has been removed (or the signal drops back below its critical value). </p>
<ul>
<li>It memorizes the signal!</li>
</ul>
<h3 id="2-Negative-Feedback"><a href="#2-Negative-Feedback" class="headerlink" title="2. Negative Feedback"></a>2. Negative Feedback</h3><blockquote>
<p>Negative Feedback is a Common Motif in Signaling Systems</p>
<p>By contrast with positive feedback, negative feedback counteracts the effect of a stimulus and thereby <strong>abbreviates</strong> and <strong>limits</strong> the level of the response, making the system <u>less sensitive to perturbations</u></p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145701732.png" srcset="/img/loading.gif" lazyload alt="image-20200426145701732" style="zoom: 25%;" />
+ Such reactions can <u>convert a short-term signal in a long-term answer!</u>

<p>A stimulus activates protein A, which, in turn, activates protein B, while protein B then acts back to increase the activity of A.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145758273.png" srcset="/img/loading.gif" lazyload alt="image-20200426145758273"></p>
<ul>
<li>Negative feedback change signal to oscillations</li>
</ul>
<p>The basal activity of the I phosphatase dephosphorylates (deactivates) the activated E kinase at a steady, low rate. </p>
<p>With a short delay, the system shows a strong, brief response when the signal is abruptly changed, and the feedback then drives the response back down to a lower level. </p>
<blockquote>
<p>A <strong>delayed negative feedback</strong> with a <strong>long enough delay</strong> can produce responses that oscillate.</p>
</blockquote>
<p>With a long delay, the feedback produces sustained oscillations for as long as the stimulus is present, then no further response.</p>
<h4 id="How-Negative-Feedback"><a href="#How-Negative-Feedback" class="headerlink" title="How Negative Feedback"></a>How Negative Feedback</h4><p>Detects changes of concentration of signals.</p>
<p>There are several ways to achieve these:</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150043835.png" srcset="/img/loading.gif" lazyload alt="image-20200426150043835"></p>
<blockquote>
<p>Cells Can Adjust Their Sensitivity to a Signal</p>
<p>The target cells accomplish this through a reversible process of <strong>adaptation</strong>, or <strong>desensitization</strong>, whereby a prolonged exposure to a stimulus <strong>decreases</strong> the cells’ response to that level of stimulus. In chemical signaling, adaptation enables cells to <u>respond to changes in the concentration</u> of an extracellular signal molecule  (rather than to the absolute concentration of the signal) over a very wide range  of signal concentrations.</p>
<p><strong>Receptors</strong> can also become <strong>inactivated</strong> on the cell surface—for example, by becoming <strong>phosphorylated</strong>—with a short delay following their activation. Adaptation can also  occur at sites <u>downstream of the receptors</u>, either by a change in <strong>intracellular  signaling proteins</strong> involved in transducing the extracellular signal or by the <u>production of an inhibitor protein</u> that blocks the signal transduction process.</p>
</blockquote>
<p>Some ways in which target cells can become adapted (desensitized) to an extracellular signal molecule</p>
<ol>
<li>Receptor sequestration</li>
<li>Receptor down-regulation</li>
<li>Receptor inactivation</li>
<li>Inactivation of signaling protein</li>
<li>Production of inhibitory protein</li>
</ol>
<h1 id="二、General-methods-to-study-cell-signaling"><a href="#二、General-methods-to-study-cell-signaling" class="headerlink" title="二、General methods to study cell signaling"></a>二、General methods to study cell signaling</h1><h2 id="co-IP"><a href="#co-IP" class="headerlink" title="co-IP"></a>co-IP</h2><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150134992.png" srcset="/img/loading.gif" lazyload alt="image-20200426150134992" style="zoom: 25%;" />

<p>Applications for co-IPs:</p>
<ul>
<li>Protein-protein interactions in general</li>
<li>To find other protein interaction partners </li>
<li>To analyze receptor-ligand interaction</li>
<li>To analyze kinase-substrate interaction</li>
</ul>
<p>Step:</p>
<ol>
<li>Incubation with an antibody</li>
<li>Antibody binds to antigen (target molecules)</li>
<li>Protein A-conjugated beads bind to the antibody</li>
<li>Sedimentation of beads by centrifugation</li>
</ol>
<h2 id="Western-Blotting"><a href="#Western-Blotting" class="headerlink" title="Western Blotting"></a>Western Blotting</h2><p>Western blotting (WB) (SDS-PAGE followed by WB and immunodetection)</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150230177.png" srcset="/img/loading.gif" lazyload alt="image-20200426150230177"></p>
<p>Western blotting analyzing activation of cell signaling</p>
<h2 id="In-vitro-kinase-assay"><a href="#In-vitro-kinase-assay" class="headerlink" title="In vitro kinase assay"></a><em>In vitro</em> kinase assay</h2><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150334330.png" srcset="/img/loading.gif" lazyload alt="image-20200426150334330" style="zoom:50%;" />

<p>Samples:<br>analysis of the kinase activity of three polo-like kinases (PLKs) using 32P in a substrate (casein) phosphorylation reaction in the absence (-) or presence (+) of the kinase inhibitor BI 2536.</p>
<p>Procedure: </p>
<ol>
<li>Purify protein in vitro (either isolation or production of recombinant proteins)Set up in vitro protein assay with substrates and necessary components such as ATP, etc.Upper panel: 32P-phosphorylated casein </li>
<li>Analyze protein activity by comparing signal strength.</li>
</ol>
<h2 id="shRNA-siRNA-mediated-“knock-down”-or-using-inhibitors"><a href="#shRNA-siRNA-mediated-“knock-down”-or-using-inhibitors" class="headerlink" title="shRNA&#x2F;siRNA-mediated “knock-down” or using inhibitors"></a>shRNA&#x2F;siRNA-mediated “knock-down” or using inhibitors</h2><p>sh (short hairpin) RNA &#x2F;si (small interfering) RNA mediated depletion of specific mRNA via RNA interference (RNAi).</p>
<p>Many enzymes have relatively specific inhibitors.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150451437.png" srcset="/img/loading.gif" lazyload alt="image-20200426150451437"></p>
<ul>
<li>Example: cGMP-dependent protein kinase G (PKG-Ia) can phosphorylate VASP and Src upon EGF perception by PKG-I$\alpha$.</li>
</ul>
<h2 id="Phenotype-rescue-assay"><a href="#Phenotype-rescue-assay" class="headerlink" title="Phenotype rescue assay"></a>Phenotype rescue assay</h2><p>Application:<br>To prove that one signaling protein locates upstream or downstream of another.</p>
<ol>
<li>Deletion of A or B leads to a certain defect </li>
<li>Expression of activated C can rescue this defect.</li>
</ol>
<h2 id="Summary"><a href="#Summary" class="headerlink" title="Summary"></a>Summary</h2><p>General test for protein-protein interaction</p>
<ul>
<li>Protein co-immunoprecipitation (co-IP)</li>
<li>“pull-down” assays</li>
<li>SDS-PAGE (Polyacrylamide gel electrophoresis) followed by Western blotting (WB) and immunodetection using specific antibodies</li>
<li>In vitro kinase assay</li>
</ul>
<p>RNA interference (RNAi)</p>
<ul>
<li>short hairpin (shRNA)&#x2F;short interfering (siRNA) induced “knock-down” </li>
<li>Functional analysis using chemical inhibitors </li>
<li>Rescue analysis of defects</li>
<li>Life cell imaging analysis and spectro-microscopical analysis</li>
</ul>
<h1 id="三、Classification-of-receptors"><a href="#三、Classification-of-receptors" class="headerlink" title="三、Classification of receptors"></a>三、Classification of receptors</h1><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200429211822116.png" srcset="/img/loading.gif" lazyload alt="image-20200429211822116"></p>
<ol>
<li>G-protein coupled receptor<ol>
<li>via <font color="EE0000">cAMP-PKA</font></li>
<li>via phospholipids: <font color="EE0000">IP<del>3</del> </font>and <font color="EE0000">DAG</font>, <font color="EE0000">Ca2+</font></li>
<li>Via <font color="EE0000">NO</font></li>
</ol>
</li>
<li>Enzyme-coupled receptor<ol>
<li><font color="EE0000">Receptor tyrosine kinase (RTK)</font></li>
<li><font color="EE0000">Ras</font> signaling</li>
<li><font color="EE0000">Rho</font> family GTPases</li>
<li><font color="EE0000">PI<del>3</del>K</font>&#x2F;<font color="EE0000">Akt</font> signaling</li>
<li><font color="EE0000">TGF-$\beta$</font> superfamily</li>
<li><font color="EE0000">Cytosolic tyrosine kinase</font></li>
<li><font color="EE0000">Cytokine receptor&#x2F;JAK-STAT</font> signaling</li>
</ol>
</li>
<li>Other signaling<ol>
<li><font color="EE0000">Notch</font> signaling</li>
<li><font color="EE0000">Wnt</font> signaling</li>
<li><font color="EE0000">Hedgehog</font> signaling</li>
<li><font color="EE0000">NF$\kappa$B</font> </li>
<li>Circadian clocks</li>
</ol>
</li>
<li>Nuclear receptors</li>
</ol>
<h2 id="GPCRs"><a href="#GPCRs" class="headerlink" title="GPCRs"></a>GPCRs</h2><p>G (GTP-binding)-protein-coupled receptors (GPCRs) </p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150849931.png" srcset="/img/loading.gif" lazyload alt="image-20200426150849931"></p>
<p>The activation of a G-protein-coupled receptor triggers indirectly the activation of another membrane protein (via a G-protein).</p>
<ul>
<li>The G-protein “mediates” between the activated receptor and the other membrane protein.</li>
<li>That’s why this receptor type it is called “G-protein-coupled” receptor.</li>
</ul>
<h2 id="Enzyme-coupled-receptors"><a href="#Enzyme-coupled-receptors" class="headerlink" title="Enzyme-coupled receptors"></a>Enzyme-coupled receptors</h2><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150934050.png" srcset="/img/loading.gif" lazyload alt="image-20200426150934050"></p>
<p>These receptors either are enzymes themselves and are activated by the signaling molecule or they associate with enzymes, which they activate.</p>
<h2 id="Nuclear-receptors-intracellular-receptors"><a href="#Nuclear-receptors-intracellular-receptors" class="headerlink" title="Nuclear receptors (intracellular receptors)"></a>Nuclear receptors (intracellular receptors)</h2><p>Nuclear receptors are Ligand-modulated gene regulatory proteins</p>
<p>Ligands of nuclear receptors:</p>
<p>Steroid hormones (made from cholesterol) : </p>
<ul>
<li><strong>Cortisol</strong> ( secreted from cortex to adrenal gland)</li>
<li><strong>Sex hormones</strong> (estradiol, testosterone, progesterone</li>
<li><strong>Vitamin D</strong> (synthesized in the skin under sunlight)</li>
<li>molting hormone ecdysone (insects)</li>
</ul>
<p>Thyroid hormone: (made from tyrosine)</p>
<p>Retinoids ( made from vitamin A)</p>
<h3 id="1-how-nuclear-receptors-work"><a href="#1-how-nuclear-receptors-work" class="headerlink" title="1. how nuclear receptors work"></a>1. how nuclear receptors work</h3><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426192901261.png" srcset="/img/loading.gif" lazyload alt="image-20200426192901261"></p>
<h3 id="2-Common-features-of-nuclear-receptors"><a href="#2-Common-features-of-nuclear-receptors" class="headerlink" title="2. Common features of nuclear receptors"></a>2. Common features of nuclear receptors</h3><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426192918945.png" srcset="/img/loading.gif" lazyload alt="image-20200426192918945" style="zoom:25%;" />

<p>Work either as homodimer of heterodimer</p>
<p>Serve both as ligand receptor and as gene transcription factor</p>
<p>All of these receptors possess 3 functional domains:</p>
<ul>
<li>DNA binding domain </li>
<li>Gene transactivation domain</li>
<li>Ligand binding domain</li>
</ul>
<h2 id="Signaling-molecules-that-can-cross-the-membrane"><a href="#Signaling-molecules-that-can-cross-the-membrane" class="headerlink" title="Signaling molecules that can cross the membrane"></a>Signaling molecules that can cross the membrane</h2><p>Signaling molecules: small hydrophobic molecules that can cross the plasma membrane.</p>
<ul>
<li>Examples: <ul>
<li>Nitric oxide (NO) gas, carbon monoxide (CO) gas</li>
<li>Steroid hormones, thyroid hormones, retinoids, vitamin D, etc.</li>
<li>Steroid hormones are small enough (+ hydrophobic) to go through the membrane</li>
<li>They bind to receptor proteins which are transcription regulators and may be located in nucleus or in cytoplasm</li>
</ul>
</li>
</ul>
<p>These signaling molecules bind to an intracellular receptor.</p>
<h3 id="1-Steroid-Responses"><a href="#1-Steroid-Responses" class="headerlink" title="1. Steroid Responses"></a>1. Steroid Responses</h3><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426192717358.png" srcset="/img/loading.gif" lazyload alt="image-20200426192717358" style="zoom: 33%;" />

<p>Cytosol location</p>
<ul>
<li>receptor bound to inhibitor</li>
<li>ligand binding activates receptor</li>
<li>translocates to nucleus on ligand binding</li>
</ul>
<p>Nuclear location</p>
<ul>
<li>binds ligand and DNA</li>
<li>becomes transcription factor</li>
</ul>
<p>Some nongaseous signal molecules that bind to intracellular receptors</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426192811249.png" srcset="/img/loading.gif" lazyload alt="image-20200426192811249"></p>
<ul>
<li>All of them are small and hydrophobic!</li>
</ul>

                
              </div>
            
            <hr/>
            <div>
              <div class="post-metas my-3">
  
    <div class="post-meta mr-3 d-flex align-items-center">
      <i class="iconfont icon-category"></i>
      

<span class="category-chains">
  
  
    
      <span class="category-chain">
        
  <a href="/categories/Academic-Notebook/" class="category-chain-item">Academic Notebook</a>
  
  
    <span>></span>
    
  <a href="/categories/Academic-Notebook/Cell-Biology/" class="category-chain-item">Cell Biology</a>
  
  

  

      </span>
    
  
</span>

    </div>
  
  
    <div class="post-meta">
      <i class="iconfont icon-tags"></i>
      
        <a href="/tags/Cell-Biology/" class="print-no-link">#Cell Biology</a>
      
    </div>
  
</div>


              
  

  <div class="license-box my-3">
    <div class="license-title">
      <div>L15 Overview of Cell Communication</div>
      <div>https://zhenyumi.github.io/posts/8e27290e/</div>
    </div>
    <div class="license-meta">
      
        <div class="license-meta-item">
          <div>作者</div>
          <div>向海</div>
        </div>
      
      
        <div class="license-meta-item license-meta-date">
          <div>发布于</div>
          <div>2020年7月8日</div>
        </div>
      
      
      
        <div class="license-meta-item">
          <div>许可协议</div>
          <div>
            
              
              
                <a class="print-no-link" target="_blank" href="https://creativecommons.org/licenses/by/4.0/">
                  <span class="hint--top hint--rounded" aria-label="BY - 署名">
                    <i class="iconfont icon-cc-by"></i>
                  </span>
                </a>
              
            
          </div>
        </div>
      
    </div>
    <div class="license-icon iconfont"></div>
  </div>



              
                <div class="post-prevnext my-3">
                  <article class="post-prev col-6">
                    
                    
                  </article>
                  <article class="post-next col-6">
                    
                    
                      <a href="/posts/f6992a6c/" title="L11 Membrane trafficking">
                        <span class="hidden-mobile">L11 Membrane trafficking</span>
                        <span class="visible-mobile">下一篇</span>
                        <i class="iconfont icon-arrowright"></i>
                      </a>
                    
                  </article>
                </div>
              
            </div>

            
          </article>
        </div>
      </div>
    </div>

    <div class="side-col d-none d-lg-block col-lg-2">
      
  <aside class="sidebar" style="margin-left: -1rem">
    <div id="toc">
  <p class="toc-header">
    <i class="iconfont icon-list"></i>
    <span>目录</span>
  </p>
  <div class="toc-body" id="toc-body"></div>
</div>



  </aside>


    </div>
  </div>
</div>





  



  



  



  



  


  
  









    

    
      <a id="scroll-top-button" aria-label="TOP" href="#" role="button">
        <i class="iconfont icon-arrowup" aria-hidden="true"></i>
      </a>
    

    
      <div class="modal fade" id="modalSearch" tabindex="-1" role="dialog" aria-labelledby="ModalLabel"
     aria-hidden="true">
  <div class="modal-dialog modal-dialog-scrollable modal-lg" role="document">
    <div class="modal-content">
      <div class="modal-header text-center">
        <h4 class="modal-title w-100 font-weight-bold">搜索</h4>
        <button type="button" id="local-search-close" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body mx-3">
        <div class="md-form mb-5">
          <input type="text" id="local-search-input" class="form-control validate">
          <label data-error="x" data-success="v" for="local-search-input">关键词</label>
        </div>
        <div class="list-group" id="local-search-result"></div>
      </div>
    </div>
  </div>
</div>

    

    
  </main>

  <footer>
    <div class="footer-inner">
  
    <div class="footer-content">
       <a href="https://hexo.io" target="_blank" rel="nofollow noopener"><span>Hexo</span></a> <i class="iconfont icon-love"></i> <a href="https://github.com/fluid-dev/hexo-theme-fluid" target="_blank" rel="nofollow noopener"><span>Fluid</span></a> 
    </div>
  
  
  
</div>

  </footer>

  <!-- Scripts -->
  
  <script  src="https://lib.baomitu.com/nprogress/0.2.0/nprogress.min.js" ></script>
  <link  rel="stylesheet" href="https://lib.baomitu.com/nprogress/0.2.0/nprogress.min.css" />

  <script>
    NProgress.configure({"showSpinner":false,"trickleSpeed":100})
    NProgress.start()
    window.addEventListener('load', function() {
      NProgress.done();
    })
  </script>


<script  src="https://lib.baomitu.com/jquery/3.6.4/jquery.min.js" ></script>
<script  src="https://lib.baomitu.com/twitter-bootstrap/4.6.1/js/bootstrap.min.js" ></script>
<script  src="/js/events.js" ></script>
<script  src="/js/plugins.js" ></script>





  
    <script  src="/js/img-lazyload.js" ></script>
  




  
<script>
  Fluid.utils.createScript('https://lib.baomitu.com/tocbot/4.20.1/tocbot.min.js', function() {
    var toc = jQuery('#toc');
    if (toc.length === 0 || !window.tocbot) { return; }
    var boardCtn = jQuery('#board-ctn');
    var boardTop = boardCtn.offset().top;

    window.tocbot.init(Object.assign({
      tocSelector     : '#toc-body',
      contentSelector : '.markdown-body',
      linkClass       : 'tocbot-link',
      activeLinkClass : 'tocbot-active-link',
      listClass       : 'tocbot-list',
      isCollapsedClass: 'tocbot-is-collapsed',
      collapsibleClass: 'tocbot-is-collapsible',
      scrollSmooth    : true,
      includeTitleTags: true,
      headingsOffset  : -boardTop,
    }, CONFIG.toc));
    if (toc.find('.toc-list-item').length > 0) {
      toc.css('visibility', 'visible');
    }

    Fluid.events.registerRefreshCallback(function() {
      if ('tocbot' in window) {
        tocbot.refresh();
        var toc = jQuery('#toc');
        if (toc.length === 0 || !tocbot) {
          return;
        }
        if (toc.find('.toc-list-item').length > 0) {
          toc.css('visibility', 'visible');
        }
      }
    });
  });
</script>


  <script src=https://lib.baomitu.com/clipboard.js/2.0.11/clipboard.min.js></script>

  <script>Fluid.plugins.codeWidget();</script>


  
<script>
  Fluid.utils.createScript('https://lib.baomitu.com/anchor-js/5.0.0/anchor.min.js', function() {
    window.anchors.options = {
      placement: CONFIG.anchorjs.placement,
      visible  : CONFIG.anchorjs.visible
    };
    if (CONFIG.anchorjs.icon) {
      window.anchors.options.icon = CONFIG.anchorjs.icon;
    }
    var el = (CONFIG.anchorjs.element || 'h1,h2,h3,h4,h5,h6').split(',');
    var res = [];
    for (var item of el) {
      res.push('.markdown-body > ' + item.trim());
    }
    if (CONFIG.anchorjs.placement === 'left') {
      window.anchors.options.class = 'anchorjs-link-left';
    }
    window.anchors.add(res.join(', '));

    Fluid.events.registerRefreshCallback(function() {
      if ('anchors' in window) {
        anchors.removeAll();
        var el = (CONFIG.anchorjs.element || 'h1,h2,h3,h4,h5,h6').split(',');
        var res = [];
        for (var item of el) {
          res.push('.markdown-body > ' + item.trim());
        }
        if (CONFIG.anchorjs.placement === 'left') {
          anchors.options.class = 'anchorjs-link-left';
        }
        anchors.add(res.join(', '));
      }
    });
  });
</script>


  
<script>
  Fluid.utils.createScript('https://lib.baomitu.com/fancybox/3.5.7/jquery.fancybox.min.js', function() {
    Fluid.plugins.fancyBox();
  });
</script>


  <script>Fluid.plugins.imageCaption();</script>

  
      <script>
        if (!window.MathJax) {
          window.MathJax = {
            tex    : {
              inlineMath: { '[+]': [['$', '$']] }
            },
            loader : {
              load: ['ui/lazy']
            },
            options: {
              renderActions: {
                insertedScript: [200, () => {
                  document.querySelectorAll('mjx-container').forEach(node => {
                    let target = node.parentNode;
                    if (target.nodeName.toLowerCase() === 'li') {
                      target.parentNode.classList.add('has-jax');
                    }
                  });
                }, '', false]
              }
            }
          };
        } else {
          MathJax.startup.document.state(0);
          MathJax.texReset();
          MathJax.typeset();
          MathJax.typesetPromise();
        }

        Fluid.events.registerRefreshCallback(function() {
          if ('MathJax' in window && MathJax.startup.document && typeof MathJax.startup.document.state === 'function') {
            MathJax.startup.document.state(0);
            MathJax.texReset();
            MathJax.typeset();
            MathJax.typesetPromise();
          }
        });
      </script>
    

  <script  src="https://lib.baomitu.com/mathjax/3.2.2/es5/tex-mml-chtml.js" ></script>

  <script  src="/js/local-search.js" ></script>





<!-- 主题的启动项，将它保持在最底部 -->
<!-- the boot of the theme, keep it at the bottom -->
<script  src="/js/boot.js" ></script>


  

  <noscript>
    <div class="noscript-warning">博客在允许 JavaScript 运行的环境下浏览效果更佳</div>
  </noscript>
</body>
</html>
