

<!DOCTYPE html>
<html lang="zh-CN" >



<head>
  <meta charset="UTF-8">

  <link rel="apple-touch-icon" sizes="76x76" href="/img/Logo.png">
  <link rel="icon" href="/img/Logo.png">
  

  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=5.0, shrink-to-fit=no">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  
  <meta name="theme-color" content="#2f4154">
  <meta name="author" content="向海">
  <meta name="keywords" content="">
  
    <meta name="description" content="No such things">
<meta property="og:type" content="article">
<meta property="og:title" content="L05 Bioinformatics">
<meta property="og:url" content="https://zhenyumi.github.io/posts/52351e2/index.html">
<meta property="og:site_name" content="鹘横海">
<meta property="og:description" content="No such things">
<meta property="og:locale" content="zh_CN">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Molecular%20Biology/20200724_%EC%BA%AC%EB%A3%A8%ED%83%88%EC%B6%9C.png">
<meta property="article:published_time" content="2020-07-24T09:20:09.000Z">
<meta property="article:modified_time" content="2020-07-24T09:20:09.000Z">
<meta property="article:author" content="向海">
<meta property="article:tag" content="Molecular Biology">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Molecular%20Biology/20200724_%EC%BA%AC%EB%A3%A8%ED%83%88%EC%B6%9C.png">
  
  
  
  <title>L05 Bioinformatics - 鹘横海</title>

  <link  rel="stylesheet" href="https://lib.baomitu.com/twitter-bootstrap/4.6.1/css/bootstrap.min.css" />



  <link  rel="stylesheet" href="https://lib.baomitu.com/github-markdown-css/4.0.0/github-markdown.min.css" />

  <link  rel="stylesheet" href="https://lib.baomitu.com/hint.css/2.7.0/hint.min.css" />

  <link  rel="stylesheet" href="https://lib.baomitu.com/fancybox/3.5.7/jquery.fancybox.min.css" />



<!-- 主题依赖的图标库，不要自行修改 -->
<!-- Do not modify the link that theme dependent icons -->

<link rel="stylesheet" href="//at.alicdn.com/t/c/font_1749284_5i9bdhy70f8.css">



<link rel="stylesheet" href="//at.alicdn.com/t/c/font_1736178_k526ubmyhba.css">


<link  rel="stylesheet" href="/css/main.css" />


  <link id="highlight-css" rel="stylesheet" href="/css/highlight.css" />
  




  <script id="fluid-configs">
    var Fluid = window.Fluid || {};
    Fluid.ctx = Object.assign({}, Fluid.ctx)
    var CONFIG = {"hostname":"zhenyumi.github.io","root":"/","version":"1.9.8","typing":{"enable":true,"typeSpeed":70,"cursorChar":"_","loop":false,"scope":["home"]},"anchorjs":{"enable":true,"element":"h1,h2,h3,h4,h5,h6","placement":"left","visible":"hover","icon":""},"progressbar":{"enable":true,"height_px":3,"color":"#29d","options":{"showSpinner":false,"trickleSpeed":100}},"code_language":{"enable":true,"default":"TEXT"},"copy_btn":true,"image_caption":{"enable":true},"image_zoom":{"enable":true,"img_url_replace":["",""]},"toc":{"enable":true,"placement":"right","headingSelector":"h1,h2,h3,h4,h5,h6","collapseDepth":0},"lazyload":{"enable":true,"loading_img":"/img/loading.gif","onlypost":false,"offset_factor":2},"web_analytics":{"enable":false,"follow_dnt":true,"baidu":null,"google":{"measurement_id":null},"tencent":{"sid":null,"cid":null},"leancloud":{"app_id":null,"app_key":null,"server_url":null,"path":"window.location.pathname","ignore_local":false},"umami":{"src":null,"website_id":null,"domains":null,"start_time":"2024-01-01T00:00:00.000Z","token":null,"api_server":null}},"search_path":"/local-search.xml","include_content_in_search":true};

    if (CONFIG.web_analytics.follow_dnt) {
      var dntVal = navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack;
      Fluid.ctx.dnt = dntVal && (dntVal.startsWith('1') || dntVal.startsWith('yes') || dntVal.startsWith('on'));
    }
  </script>
  <script  src="/js/utils.js" ></script>
  <script  src="/js/color-schema.js" ></script>
  


  
<meta name="generator" content="Hexo 7.0.0"></head>


<body>
  

  <header>
    

<div class="header-inner" style="height: 70vh;">
  <nav id="navbar" class="navbar fixed-top  navbar-expand-lg navbar-dark scrolling-navbar">
  <div class="container">
    <a class="navbar-brand" href="/">
      <strong>鹘横海</strong>
    </a>

    <button id="navbar-toggler-btn" class="navbar-toggler" type="button" data-toggle="collapse"
            data-target="#navbarSupportedContent"
            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <div class="animated-icon"><span></span><span></span><span></span></div>
    </button>

    <!-- Collapsible content -->
    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav ml-auto text-center">
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/" target="_self">
                <i class="iconfont icon-home-fill"></i>
                <span>首页</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/archives/" target="_self">
                <i class="iconfont icon-archive-fill"></i>
                <span>归档</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/categories/" target="_self">
                <i class="iconfont icon-category-fill"></i>
                <span>分类</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/tags/" target="_self">
                <i class="iconfont icon-tags-fill"></i>
                <span>标签</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/about/" target="_self">
                <i class="iconfont icon-user-fill"></i>
                <span>关于</span>
              </a>
            </li>
          
        
        
          <li class="nav-item" id="search-btn">
            <a class="nav-link" target="_self" href="javascript:;" data-toggle="modal" data-target="#modalSearch" aria-label="Search">
              <i class="iconfont icon-search"></i>
            </a>
          </li>
          
        
        
      </ul>
    </div>
  </div>
</nav>

  

<div id="banner" class="banner" parallax=true
     style="background: url('https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Molecular%20Biology/20200724_Sunlight%281%29.jpg') no-repeat center center; background-size: cover;">
  <div class="full-bg-img">
    <div class="mask flex-center" style="background-color: rgba(0, 0, 0, 0.3)">
      <div class="banner-text text-center fade-in-up">
        <div class="h2">
          
            <span id="subtitle">L05 Bioinformatics</span>
          
        </div>

        
          
  <div class="mt-3">
    
    
      <span class="post-meta">
        <i class="iconfont icon-date-fill" aria-hidden="true"></i>
        <time datetime="2020-07-24 17:20" pubdate>
          2020年7月24日 下午
        </time>
      </span>
    
  </div>

  <div class="mt-1">
    
      <span class="post-meta mr-2">
        <i class="iconfont icon-chart"></i>
        
          1.8k 字
        
      </span>
    

    
      <span class="post-meta mr-2">
        <i class="iconfont icon-clock-fill"></i>
        
        
        
          15 分钟
        
      </span>
    

    
    
  </div>


        
      </div>

      
    </div>
  </div>
</div>

</div>

  </header>

  <main>
    
      

<div class="container-fluid nopadding-x">
  <div class="row nomargin-x">
    <div class="side-col d-none d-lg-block col-lg-2">
      

    </div>

    <div class="col-lg-8 nopadding-x-md">
      <div class="container nopadding-x-md" id="board-ctn">
        <div id="board">
          <article class="post-content mx-auto">
            <h1 id="seo-header">L05 Bioinformatics</h1>
            
            
              <div class="markdown-body">
                
                <h1 id="一、Bioinformatics"><a href="#一、Bioinformatics" class="headerlink" title="一、Bioinformatics"></a>一、Bioinformatics</h1><p>Analysis of DNA or protein sequences using computational tools.</p>
<h2 id="Sequencing-Techniques"><a href="#Sequencing-Techniques" class="headerlink" title="Sequencing Techniques"></a>Sequencing Techniques</h2><ol>
<li>454 sequencing technology</li>
<li>SBS (Sequencing-by-synthesis)</li>
<li>Illumina sequencing</li>
</ol>
<h2 id="Definition-of-Annotation"><a href="#Definition-of-Annotation" class="headerlink" title="Definition of Annotation"></a>Definition of Annotation</h2><ul>
<li><u>Interpreting</u> raw sequence data into useful biological information.</li>
<li><u>Addition</u> of information to describe a sequence.</li>
<li><u>Information</u> attached to genomic coordinates with start and end point.</li>
<li><strong>Identification</strong>, structural description, characterization of putative protein products and other features in primary genomic sequence.</li>
</ul>
<h3 id="1-Structural-Annotation"><a href="#1-Structural-Annotation" class="headerlink" title="1. Structural Annotation"></a>1. Structural Annotation</h3><p>Finding genes and other biologically <strong>relevant sites</strong> <u>thus building up a model of genome as objects with specific locations</u></p>
<h3 id="2-Functional-Annotation"><a href="#2-Functional-Annotation" class="headerlink" title="2. Functional Annotation"></a>2. Functional Annotation</h3><p>Objects are used in database searches (and experiments); the aim is to attribute biologically relevant information <u>to whole sequence and individual objects</u></p>
<p>Searching Databases is a kind of functional annotation of sequences</p>
<h2 id="Features-that-can-be-annotated-in-DNA"><a href="#Features-that-can-be-annotated-in-DNA" class="headerlink" title="Features that can be annotated in DNA"></a>Features that can be annotated in DNA</h2><ol>
<li><p>Genes: </p>
<ol>
<li><p>Protein Binding Sites</p>
</li>
<li><p>Alternative Poly-A sites</p>
</li>
<li><p>Alternative Splice sites</p>
</li>
<li><p>Poly-A site</p>
</li>
<li><p><strong>Enhancers:</strong> DNA上一小段可与蛋白质（反式作用因子；trans-acting factor）结合的区域，与蛋白质结合之后，基因的转录作用将会加强。增强子可能位于基因上游，也可能位于下游。且不一定接近所要作用的基因，甚至不一定与基因位于同一染色体。这是因为染色质的缠绕结构，使序列上相隔很远的位置也有机会相互接触。 </p>
</li>
<li><p><strong>Promoters</strong>: 启动子（promoter）是指一段能使特定基因进行转录的脱氧核糖核酸（DNA）序列。启动子可以被RNA聚合酶辨认，并开始转录合成RNA。在核糖核酸（RNA）合成中，启动子可以和调控基因转录的转录因子产生相互作用，控制基因表达（转录）的起始时间和表达的程度，包含核心启动子区域和调控区域，就像“开关”，决定基因的活动，继而控制细胞开始生产哪一种蛋白质。<br> 启动子位于控制基因表达的调控序列中、基因转录起始位点的上游（DNA反义链的5′方向），长约100~1000个碱基对。启动子本身并无编译功能，但它拥有对基因翻译氨基酸的指挥作用，就像一面旗帜，其核心部分是非编码区上游的RNA聚合酶结合位点，指挥聚合酶的合成，这种酶指导RNA的复制合成。因此该段位的启动子发生突变（变异），将对基因的表达有着毁灭性作用。<br> 完全的启动子称为规范序列。 </p>
</li>
<li><p><strong>3’UTR</strong>: 3’非翻译区（英语：three prime untranslated region）简称3’-UTR，<u>是信使核糖核酸（mRNA）中，紧接在编码区之后的序列。</u>mRNA上除了可以翻译成蛋白质的序列外，还有一些不被翻译的序列，如5端（5’）的5′端帽与5′非翻译区（5’-UTR），以及3端（3’）的3′非翻译区（3’-UTR）与多聚腺苷酸尾。3’-UTR上含有AAUAAA的序列，可被特定蛋白识别，以启动多腺苷酸化，即切割mRNA，并于切割位点的3端加上约250个单磷酸腺苷，即多聚腺苷酸尾</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Mature_mRNA.png" srcset="/img/loading.gif" lazyload alt="img" style="zoom: 33%;" />
</li>
<li><p>Exon and Intron</p>
</li>
</ol>
</li>
<li><p>Non-genic Encoded RNA</p>
<ol>
<li>ncRNA（非编码核糖核酸）</li>
<li>**MicroRNA: ** miRNA通过与目标信使核糖核酸（mRNA）结合，进而抑制转录后的基因表达，在调控基因表达、细胞周期、生物体发育时序等方面起重要作用。</li>
<li>**Anti-sense RNA: **In genetics, a sense strand, or coding strand, is the segment within double-stranded DNA that runs from 5’ to 3’, and which is complementary to the antisense strand of DNA, or template strand, which runs from 3’ to 5’. <u>The sense strand is the strand of DNA that has the same sequence as the mRNA,</u> which takes the antisense strand as its template during transcription, and eventually undergoes (typically, not always) translation into a protein.</li>
<li>rRNA</li>
<li>tRNA</li>
</ol>
</li>
<li><p>Large Genomic Regions</p>
<ol>
<li><strong>Microsatellites:</strong> A microsatellite is a tract of repetitive DNA in which certain DNA motifs (ranging in length from one to six or more base pairs) are repeated, typically 5–50 times.（串联重复）</li>
<li>**Transposons: **A transposable element (TE, transposon, or jumping gene) is a DNA sequence that can change its position within a genome, sometimes creating or reversing mutations and altering the cell’s genetic identity and genome size</li>
<li><strong>LTRs: (long terminal repeat，长端重复序列)</strong><br>Long terminal repeats (LTRs) are identical sequences of DNA that repeat hundreds or thousands of times found <u>at either end of retrotransposons or proviral DNA</u> formed by reverse transcription of retroviral RNA. <u>They are used by viruses to insert their genetic material into the host genomes.</u></li>
<li>heterochromatic knob repeats</li>
</ol>
</li>
<li><p>Chromosomal Features</p>
<ol>
<li>**Telomeres: **端粒（英语：Telomere）是真核生物染色体末端的DNA重复序列，作用是保持染色体的完整性和控制细胞分裂周期。 由于DNA复制的机制，每次染色体复制后，延迟股上的染色体末端必无法被复制。</li>
<li>Centromere</li>
<li>Matrix Attachment Regions</li>
</ol>
</li>
<li><p>Comparative Aspects</p>
<ol>
<li>Indels</li>
<li>SNPs</li>
<li>Rearrangements</li>
<li>Duplications</li>
<li>Synonymous VS. Non-synonymous Substitutions</li>
</ol>
</li>
</ol>
<h2 id="Gene-finding-Programs"><a href="#Gene-finding-Programs" class="headerlink" title="Gene-finding Programs"></a>Gene-finding Programs</h2><p>   GRAIL, MZEF, FGENES, glimmer, GENSCAN, GeneMark, HMMgene…. </p>
<h3 id="1-Types-of-Methods"><a href="#1-Types-of-Methods" class="headerlink" title="1. Types of Methods:"></a>1. Types of Methods:</h3><ol>
<li><p>Content-based</p>
<ul>
<li>codon usage</li>
<li>substitutions</li>
<li>composition</li>
<li>repeats</li>
</ul>
</li>
<li><p>Site-based</p>
<ul>
<li>presence&#x2F;absence of motif – splice sites</li>
<li>TF binding sites</li>
<li>start&#x2F;stop codons.</li>
</ul>
</li>
<li><p>Comparative</p>
<ul>
<li>make determinations based on sequence homology</li>
</ul>
</li>
</ol>
<h3 id="2-ORF"><a href="#2-ORF" class="headerlink" title="2. ORF"></a>2. ORF</h3><p>In molecular genetics, an <strong>open reading frame</strong> (ORF) is the part of a reading frame that has the ability to be translated. </p>
<p>An ORF is a continuous stretch of codons that begins with a start codon (usually AUG) and ends at a stop codon (usually UAA, UAG or UGA).</p>
<p>In molecular genetics, an open reading frame (ORF) is the part of a reading frame that has the potential to code for a protein or peptide. An ORF is a continuous stretch of codons that do not contain a stop codon (usually UAA, UAG or UGA). </p>
<h4 id="ORF-in-cDNA"><a href="#ORF-in-cDNA" class="headerlink" title="ORF in cDNA"></a>ORF in cDNA</h4><p>A cDNA molecule should have only one ORF, meaning that the protein is encoded  between one ATG and one stop codon. Other ATGs may be found, including some in the 3’ UTR , but there will be very few ORFs (only one?) per mRNA&#x2F;cDNA.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191104192109954.png" srcset="/img/loading.gif" lazyload alt="image-20191104192109954" style="zoom: 33%;" />

<h3 id="3-Steps-to-Find-Gene"><a href="#3-Steps-to-Find-Gene" class="headerlink" title="3. Steps to Find Gene"></a>3. Steps to Find Gene</h3><p>The first step in gene finding is to <u>identify all of the ORFs</u> (open reading frames).  These start with ATG, ends with in-frame TAA, TGA, or TAG. </p>
<p>Only a fraction of these will include the entire gene, but many will be exons. </p>
<p>As a general rule, for a given gene, there is only one ATG that is used as a  translational start. However, there may be internal ATGs that are just normal  methionine codons.</p>
<h2 id="Searching-Databases"><a href="#Searching-Databases" class="headerlink" title="Searching Databases"></a>Searching Databases</h2><p>Functional annotation of sequences: Searching Databases</p>
<p>实验室内的实验很花时间，因此想要得到某些信息的时候，搜索database很方便快捷，同时也很cheap</p>
<ol>
<li>Databases can <u>indicate known function</u> of related sequences (<u>predict function</u> based on similarities in structure) </li>
<li>Database homology searches can <u>annotate large numbers of unknown sequences</u> (e.g. whole genomes or sets of ESTs).</li>
</ol>
<h3 id="1-BLAST-–-The-king-of-search-tools"><a href="#1-BLAST-–-The-king-of-search-tools" class="headerlink" title="1. BLAST – The king of search tools"></a>1. BLAST – The king of search tools</h3><p><strong>Basic Local Alignment Search Tool – most commonly used</strong>  <strong>program due to large size of databases.</strong> </p>
<p>Heuristic algorithm to accelerate searches; aligns small bits of DNA  and looks for the best overall match between the “query” sequence  and an entry in the database.  </p>
<p>The BLAST programs are very <strong>fast</strong> because they only calculate optimal alignment scores for most closely related sequences. </p>
<p><strong>BLAST compares short segments (words) of the query and</strong>  <strong>database to find alignments that exceed a particular score.</strong> </p>
<h3 id="2-Different-Favors-of-BLAST"><a href="#2-Different-Favors-of-BLAST" class="headerlink" title="2. Different Favors of BLAST"></a>2. Different Favors of BLAST</h3><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191104192601363.png" srcset="/img/loading.gif" lazyload alt="image-20191104192601363" style="zoom: 33%;" />

<p><strong>All comparisons except BLASTN are carried out as protein&#x2F;protein comparisons.</strong></p>
<h3 id="3-How-to-USE-–-Searching-Genbank-At-NCBI"><a href="#3-How-to-USE-–-Searching-Genbank-At-NCBI" class="headerlink" title="3. How to USE – Searching Genbank At NCBI"></a>3. How to USE – Searching Genbank At NCBI</h3><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191104192727667.png" srcset="/img/loading.gif" lazyload alt="image-20191104192727667" style="zoom: 33%;" />

<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191225214547484.png" srcset="/img/loading.gif" lazyload alt="image-20191225214547484"></p>
<h3 id="4-Schematic-of-BLASTN-results"><a href="#4-Schematic-of-BLASTN-results" class="headerlink" title="4. Schematic of BLASTN results"></a>4. Schematic of BLASTN results</h3><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191104192918887.png" srcset="/img/loading.gif" lazyload alt="image-20191104192918887" style="zoom:33%;" />

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191104192943494.png" srcset="/img/loading.gif" lazyload alt="image-20191104192943494" style="zoom: 50%;" />

<h3 id="5-How-to-read-the-BLAST-Result"><a href="#5-How-to-read-the-BLAST-Result" class="headerlink" title="5. How to read the BLAST Result"></a>5. How to read the BLAST Result</h3><ol>
<li><p><strong>Bit score</strong> – normalized from the raw score, and comparable  between alignments. The raw score “S” of an alignment is  usually calculated by summing the scores for each letter-to-letter and letter-to-null position in the alignment.  </p>
</li>
<li><p><strong>P-value</strong>: The number of random local alignment scores equal or  greater than S is described by the Poisson distribution. This P  value is associated with the score S. </p>
<p>Highly significant scores have P values close to zero.  </p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191104193502556.png" srcset="/img/loading.gif" lazyload alt="image-20191104193502556" style="zoom: 25%;" />
</li>
<li><p><strong>The Expect value (E)</strong> is a parameter describing the number of hits  “expected” by chance when searching a database of a particular  size. Decreases exponentially with the score. The E value  describes the random background noise that exists for matches  between sequences.</p>
</li>
</ol>
<h2 id="Sequence-Alignment-序列比对"><a href="#Sequence-Alignment-序列比对" class="headerlink" title="Sequence Alignment 序列比对"></a>Sequence Alignment 序列比对</h2><p>DNA and Protein sequence homology can be used to infer <u>evolutionary</u> relations.</p>
<h1 id="二、The-characteristic-used-for-bioinformatic"><a href="#二、The-characteristic-used-for-bioinformatic" class="headerlink" title="二、The characteristic used for bioinformatic"></a>二、The characteristic used for bioinformatic</h1><h2 id="Motifs"><a href="#Motifs" class="headerlink" title="Motifs"></a>Motifs</h2><p>Protein motifs are <u>short conserved</u> segments that <u>have a particular function</u>. </p>
<p>The presence of certain motifs can be used to predict basic characteristics of a protein that otherwise you may know nothing about.</p>
<p>Similarities in protein sequence motifs indicate related functions</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/1570720727362.png" srcset="/img/loading.gif" lazyload alt="1570720727362"></p>
<h2 id="Homologous"><a href="#Homologous" class="headerlink" title="Homologous"></a>Homologous</h2><p>DNA and protein homology can be used to infer evolutionary relationships</p>
<p><u>Coding sequences are more highly conserved than non-coding sequences</u></p>
<p>Natural selection acts on the protein because that is where the function is., but the result of the selection is stored in the DNA. If a protein is advantageous, it is the gene that is conserved. </p>
<blockquote>
<p>As shown below, there is little conservation in introns, because there are no or few known functions in introns that selection can act on.</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191225214937183.png" srcset="/img/loading.gif" lazyload alt="image-20191225214937183"></p>
<p>Homologous proteins resulted from gene duplication, not convergence. Related organisms show sequence homology at the level of DNA and Protein</p>
<blockquote>
<p>Mutations occur randomly and at a very low frequency along the gene due to mistakes in replication of DNA during cell division</p>
</blockquote>
<p>Proteins with similar sequences will have similar structure and&#x2F;or function (broadly). </p>
<ul>
<li>True biological function needs to be determined experimentally!</li>
</ul>
<h1 id="三、Tools-For-Analyzing-Gene-Expression"><a href="#三、Tools-For-Analyzing-Gene-Expression" class="headerlink" title="三、Tools For Analyzing Gene Expression"></a>三、Tools For Analyzing Gene Expression</h1><h2 id="Transient-Stable-expression-arrays"><a href="#Transient-Stable-expression-arrays" class="headerlink" title="Transient&#x2F;Stable expression arrays"></a>Transient&#x2F;Stable expression arrays</h2><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191225215205049.png" srcset="/img/loading.gif" lazyload alt="image-20191225215205049"></p>
<h2 id="Reporter-gene-报告基因"><a href="#Reporter-gene-报告基因" class="headerlink" title="Reporter gene 报告基因"></a>Reporter gene 报告基因</h2><p>Reporter genes encode proteins or enzymatic functions not typically found in a cell.</p>
<p>报告基因（英语：reporter gene，又称报导基因）是一个分子生物学概念，它是指一类在细胞、组织&#x2F;器官或个体<u>处于特定情况下会表达并使得他们产生易于检测、且实验材料原本不会产生的性状的基因</u>。较常用的报告基因有绿色萤光蛋白基因（GFP，<strong>Green Fluorescent Protein</strong>）、β-半乳糖苷酶基因（LacZ等）等。 </p>
<p>Example：</p>
<ol>
<li>B-galactosidase (color) Luciferase from fireflies Chloramphenicol  acetyltransferase (CAT)  <ul>
<li><u>CAT activity can be measured  by acetylation of  chloramphenicol and  separation by thin layer  chromatography.</u></li>
</ul>
</li>
<li>GFP</li>
<li>β-glucuronidase (GUS) gene<ul>
<li>Leaf vascular patterning in Arabidopsis shown via expression of the β-glucuronidase (GUS) gene in Arabidopsis leaves</li>
<li><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191225220133449.png" srcset="/img/loading.gif" lazyload alt="image-20191225220133449" style="zoom: 80%;" /></li>
</ul>
</li>
</ol>
<h3 id="1-GFP-and-GFP-like"><a href="#1-GFP-and-GFP-like" class="headerlink" title="1. GFP and GFP-like"></a>1. GFP and GFP-like</h3><p>GFP is one of the most commonly used reporter genes.</p>
<p>GFP contains a fluorescent compound  (fluorophore). Different mutations result in  altered spectra of fluorescence. <u>Excitation takes  place at one wavelength and fluorescence at  another.</u></p>
<p>GFP can be used for GFP expression plasmid or GFP expression protein.</p>
<p>Some GFP-like proteins can attach to specific RNA, then make the RNA molecular emit light.</p>
<h3 id="2-Fluorescence-Dye-In-Situ-Hybridization"><a href="#2-Fluorescence-Dye-In-Situ-Hybridization" class="headerlink" title="2. Fluorescence Dye (In Situ Hybridization)"></a>2. Fluorescence Dye (In Situ Hybridization)</h3><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191104195237836.png" srcset="/img/loading.gif" lazyload alt="image-20191104195237836" style="zoom:33%;" />

<p>The fluorescence dye can be used to DNA and RNA </p>
<h2 id="Co-translation-Assay"><a href="#Co-translation-Assay" class="headerlink" title="Co-translation Assay"></a>Co-translation Assay</h2><p>Tests for the ability of a DNA-binding protein to activate a reporter gene regulated by the  control region of interest.</p>
<h2 id="Hybridization-probe-杂交探针"><a href="#Hybridization-probe-杂交探针" class="headerlink" title="Hybridization probe 杂交探针"></a>Hybridization probe 杂交探针</h2><p>杂交探针（英语：Hybridization probe）是一小段单链DNA片段（十几到几百个碱基），用于检测与其互补的核酸序列。双链DNA加热变性成为单链，随后用放射性同位素（通常用32P）、荧光染料或者酶（如辣根过氧化物酶）标记成为探针。32P通常被掺入组成DNA的四种核苷酸之一的磷酸基团中，而萤光染料和酶与核酸序列以共价键相连。<br>当将探针与样品杂交时，探针和与其互补的核酸（DNA或RNA）序列通过氢键紧密相连，随后，未被杂交的多余探针被洗去。最后，根据探针的种类，可进行放射自显影、荧光显微镜、酶联放大等方法来判断样品中是否含有，或者何位置含有被测序列（即与探针互补的序列）。 </p>
<h2 id="Purification-and-Detect-Protein-Tags"><a href="#Purification-and-Detect-Protein-Tags" class="headerlink" title="Purification and Detect Protein Tags"></a>Purification and Detect Protein Tags</h2><h2 id="Microscopy"><a href="#Microscopy" class="headerlink" title="Microscopy"></a>Microscopy</h2><h2 id="Protein-Gel-Electrophoresis"><a href="#Protein-Gel-Electrophoresis" class="headerlink" title="Protein Gel Electrophoresis"></a>Protein Gel Electrophoresis</h2><h2 id="Polyclonal-Antibody"><a href="#Polyclonal-Antibody" class="headerlink" title="Polyclonal Antibody"></a>Polyclonal Antibody</h2><h2 id="Analyze-at-RNA-Level"><a href="#Analyze-at-RNA-Level" class="headerlink" title="Analyze at RNA Level"></a>Analyze at RNA Level</h2><ol>
<li><strong>Northern blots</strong> for size and abundance.</li>
<li><strong>In situ hybridizations</strong> for cellular localization.</li>
<li><strong>RNase protection assay</strong> – an RNA probe is hybridized to total RNA, protecting the target from digestion by RNase. </li>
<li><strong>RT-PCR</strong> for rapid abundance data.</li>
</ol>
<h1 id="四、Methods-to-Analyze-Protein-Expression"><a href="#四、Methods-to-Analyze-Protein-Expression" class="headerlink" title="四、Methods to Analyze Protein Expression"></a>四、Methods to Analyze Protein Expression</h1><ol>
<li><strong>Western blots</strong>  (immunoblotting) for  size and specificity. </li>
<li><strong>In situ hybridization</strong> for  localization using  secondary, labeled  antibodies.  </li>
<li><strong>Enzyme-linked immunosorbent assay  (ELISA, 酶联免疫吸附试验)</strong> – a  colorimetric assay for  protein levels.</li>
</ol>
<h1 id="五、Inhabitance-or-Interference-to-Gene-Expression"><a href="#五、Inhabitance-or-Interference-to-Gene-Expression" class="headerlink" title="五、Inhabitance or Interference to Gene Expression"></a>五、Inhabitance or Interference to Gene Expression</h1><h2 id="Antisense-oligo-反义寡核苷酸-Mediated-Inhibition-of-Gene-Expression"><a href="#Antisense-oligo-反义寡核苷酸-Mediated-Inhibition-of-Gene-Expression" class="headerlink" title="Antisense-oligo(反义寡核苷酸) Mediated Inhibition of Gene Expression"></a>Antisense-oligo(反义寡核苷酸) Mediated Inhibition of Gene Expression</h2><p>Antisense oligos complementary  to mRNAs can <u>target mRNA  for degradation or block  translation</u>.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191104202059902.png" srcset="/img/loading.gif" lazyload alt="image-20191104202059902" style="zoom:50%;" />

<h2 id="RNA-Interference"><a href="#RNA-Interference" class="headerlink" title="RNA Interference"></a>RNA Interference</h2><p><u>Processing of dsRNA  from viral or exogenous（外源的） sources.</u> </p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191104202221377.png" srcset="/img/loading.gif" lazyload alt="image-20191104202221377" style="zoom: 60%;" />

<p>Artificial microRNA can be used to down-regulate Genes or virus</p>
<h1 id="六、Methods-for-analyzing-DNA-Protein-Interactions"><a href="#六、Methods-for-analyzing-DNA-Protein-Interactions" class="headerlink" title="六、Methods for analyzing DNA-Protein Interactions"></a>六、Methods for analyzing DNA-Protein Interactions</h1><ol>
<li>Electrophoretic mobility shift assay. DNA is labelled with 32P. If bound by protein, it causes shift in size.  </li>
<li>DNase footprinting. Protein binding “protects” DNA from digestion by Dnase.</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191225224846784.png" srcset="/img/loading.gif" lazyload alt="image-20191225224846784"></p>
<h1 id="七、Methods-for-analyzing-Protein-Protein-Interactions"><a href="#七、Methods-for-analyzing-Protein-Protein-Interactions" class="headerlink" title="七、Methods for analyzing Protein-Protein Interactions"></a>七、Methods for analyzing Protein-Protein Interactions</h1><ol>
<li>GST pull-down assay to see which proteins interact with a GST-tagged protein of interest.</li>
<li>Yeast two-hybrid. Interaction of two proteins activates expression of reporter.</li>
<li>Co-IP to see if one protein can precipitate another through a direct interaction.</li>
<li>Fluorescence resonance energy transfer. Energy is only transferred when two tagged proteins are within 10 nm.</li>
</ol>
<h1 id="八、Structural-Analyze-for-Protein-X-ray-crystallography"><a href="#八、Structural-Analyze-for-Protein-X-ray-crystallography" class="headerlink" title="八、Structural Analyze for Protein X-ray crystallography"></a>八、Structural Analyze for Protein X-ray crystallography</h1><ol>
<li><strong>X-ray crystallography</strong> uses diffraction to determine crystal structures</li>
<li><strong>AFM</strong> determines the properties of the surface of a material.</li>
</ol>
<h1 id="九、Analysis-of-Gene-Expression-In-Plants"><a href="#九、Analysis-of-Gene-Expression-In-Plants" class="headerlink" title="九、Analysis of Gene Expression In Plants"></a>九、Analysis of Gene Expression In Plants</h1><h2 id="Assessing-Transcript-Abundance-From-Extracted-RNA（评估提取的RNA的转录丰度）"><a href="#Assessing-Transcript-Abundance-From-Extracted-RNA（评估提取的RNA的转录丰度）" class="headerlink" title="Assessing Transcript Abundance From Extracted RNA（评估提取的RNA的转录丰度）"></a>Assessing Transcript Abundance From Extracted RNA（评估提取的RNA的转录丰度）</h2><p>Poly(A)+RNA can be separated from other RNAs by FRactionation on An Oligo(dT) Column</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191225225552479.png" srcset="/img/loading.gif" lazyload alt="image-20191225225552479" style="zoom:72%;" />

<h2 id="Northern-Blots"><a href="#Northern-Blots" class="headerlink" title="Northern Blots"></a>Northern Blots</h2><p>北方墨点法（英语：Northern blot）是分子生物学、生物化学中常用的实验方法，通过探测样品中的RNA（或隔离的mRNA）来研究基因表现。</p>
<p>利用探针侦测由凝胶电泳分离出来的RNA片段，寻找含有特定串行的RNA片段。与南方墨点法相似，RNA样品经过凝胶电泳后会按照大小分离，然后将样品从凝胶转印到膜上，并用与目标串行互补的标记的探针来探测。实验结果可以根据所用探针的不同以多种方式来观察，但大多数都显示的是样品中被探测的RNA条带的相对位置，也就是分子大小；而条带的强度则与样品中目标RNA的含量相关。这一方法可以测量目标RNA在不同样品中的情况，因此已经被普遍用于研究特定基因在生物体中表现的时刻和表现量，也是这类研究中最基本的手段。</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191225225659889.png" srcset="/img/loading.gif" lazyload alt="image-20191225225659889"></p>
<h2 id="Real-Time-PCR"><a href="#Real-Time-PCR" class="headerlink" title="Real-Time PCR"></a>Real-Time PCR</h2><p>PCR used to measure amplification curve of individual genes</p>
<p>Amplification performed with dye-labeled primers to enable comparisons within a tube</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191225225832810.png" srcset="/img/loading.gif" lazyload alt="image-20191225225832810" style="zoom:67%;" />

<h2 id="Microarrays"><a href="#Microarrays" class="headerlink" title="Microarrays"></a>Microarrays</h2><p>A microarray is a multiplex lab-on-a-chip. It is a 2D array on a solid substrate (usually a glass slide or silicon thin-film cell) that assays large amounts of biological material using high-throughput screening miniaturized, multiplexed and parallel processing and detection methods.</p>
<p>Types:</p>
<ol>
<li>cDNA microarray</li>
<li>High-density oligonucleotide microarrays</li>
</ol>
<h2 id="Sequencing-Methods"><a href="#Sequencing-Methods" class="headerlink" title="Sequencing Methods"></a>Sequencing Methods</h2><h3 id="1-ESTs"><a href="#1-ESTs" class="headerlink" title="1. ESTs"></a>1. ESTs</h3><p>Sequencing of mRNAs, converted to cDNA</p>
<p>“Digital” measurements because counting frequency of transcripts instead of “analog” measurements of hybridization intensity</p>
<p>EST sequences were used as <u>starting material for microarray design</u></p>
<p>Cloned cDNAs used for EST sequencing were spotted on arrays</p>
<h3 id="2-SAGE"><a href="#2-SAGE" class="headerlink" title="2. SAGE"></a>2. SAGE</h3><p>The first “digital” transcriptome sequencing method to use specifically captured short “tags” of cDNAs.</p>
<p>Allowed sequencing of many tags per Sanger sequencing read.</p>
<p>Tags mapped back to longer ESTs or genomic sequences to measure abundances</p>
<h3 id="3-DGE-Digital-Gene-Expression"><a href="#3-DGE-Digital-Gene-Expression" class="headerlink" title="3. DGE (Digital Gene Expression)"></a>3. DGE (Digital Gene Expression)</h3><p>Like SAGE, “digital gene expression” methods capture short tags from specific locations at the 3’ end of end of the transcript.</p>
<p>Longer tags than most SAGE, and optimal for Illumina sequencing </p>
<p>These tags are useful for quantification by counting, or for measuring specific types of transcripts.</p>
<h3 id="4-RNA-SEQ"><a href="#4-RNA-SEQ" class="headerlink" title="4. RNA-SEQ"></a>4. RNA-SEQ</h3><p>The most popular off all methods at the present time 一 fragmented mRNA&#x2F;cDNA, deeply sequenced by next-gen methods.</p>
<h3 id="5-PARE"><a href="#5-PARE" class="headerlink" title="5. PARE"></a>5. PARE</h3><p>Data analysis of sequencing data is more “tricky” than microarray data due to the requirement to match reads back to the gene or genome, then add them up and normalize them.</p>
<h2 id="Experimental-Design"><a href="#Experimental-Design" class="headerlink" title="Experimental Design"></a>Experimental Design</h2><p>An important aspect of both array and sequencing experiments</p>
<p>Considerations:</p>
<p>Scientific</p>
<ul>
<li><p>Aim of the experiment.</p>
</li>
<li><p>Specific questions to be answered and how they are prioritized.</p>
</li>
<li><p>How will the experiments answer the posed questions?</p>
</li>
</ul>
<p>Practical (logistic)</p>
<ul>
<li><p>Types of mRNA samples: reference, control, treatment 1 (T1), and so on.</p>
</li>
<li><p>Amount of material available: count the amount of mRNA involved in one </p>
</li>
<li><p>channel of one hybridization as one unit.</p>
</li>
<li><p>Number of slides available for the experiment.</p>
</li>
</ul>
<p>Other factors</p>
<ul>
<li><p>The experimental process before hybridization: sample isolation, mRNA </p>
</li>
<li><p>extraction, amplification and labelling.</p>
</li>
<li><p>Controls planned: positive, negative, ratio, and so on.</p>
</li>
<li><p>Verification method: northern blot, reverse transcriptase (RT)-PCR, in situ hybridization, and so on.</p>
</li>
</ul>

                
              </div>
            
            <hr/>
            <div>
              <div class="post-metas my-3">
  
    <div class="post-meta mr-3 d-flex align-items-center">
      <i class="iconfont icon-category"></i>
      

<span class="category-chains">
  
  
    
      <span class="category-chain">
        
  <a href="/categories/Academic-Notebook/" class="category-chain-item">Academic Notebook</a>
  
  
    <span>></span>
    
  <a href="/categories/Academic-Notebook/Molecular-Biology/" class="category-chain-item">Molecular Biology</a>
  
  

  

      </span>
    
  
</span>

    </div>
  
  
    <div class="post-meta">
      <i class="iconfont icon-tags"></i>
      
        <a href="/tags/Molecular-Biology/" class="print-no-link">#Molecular Biology</a>
      
    </div>
  
</div>


              
  

  <div class="license-box my-3">
    <div class="license-title">
      <div>L05 Bioinformatics</div>
      <div>https://zhenyumi.github.io/posts/52351e2/</div>
    </div>
    <div class="license-meta">
      
        <div class="license-meta-item">
          <div>作者</div>
          <div>向海</div>
        </div>
      
      
        <div class="license-meta-item license-meta-date">
          <div>发布于</div>
          <div>2020年7月24日</div>
        </div>
      
      
      
        <div class="license-meta-item">
          <div>许可协议</div>
          <div>
            
              
              
                <a class="print-no-link" target="_blank" href="https://creativecommons.org/licenses/by/4.0/">
                  <span class="hint--top hint--rounded" aria-label="BY - 署名">
                    <i class="iconfont icon-cc-by"></i>
                  </span>
                </a>
              
            
          </div>
        </div>
      
    </div>
    <div class="license-icon iconfont"></div>
  </div>



              
                <div class="post-prevnext my-3">
                  <article class="post-prev col-6">
                    
                    
                      <a href="/posts/29f349b4/" title="L04 Recombinat DNA">
                        <i class="iconfont icon-arrowleft"></i>
                        <span class="hidden-mobile">L04 Recombinat DNA</span>
                        <span class="visible-mobile">上一篇</span>
                      </a>
                    
                  </article>
                  <article class="post-next col-6">
                    
                    
                      <a href="/posts/808992df/" title="L06 DNA Replication">
                        <span class="hidden-mobile">L06 DNA Replication</span>
                        <span class="visible-mobile">下一篇</span>
                        <i class="iconfont icon-arrowright"></i>
                      </a>
                    
                  </article>
                </div>
              
            </div>

            
          </article>
        </div>
      </div>
    </div>

    <div class="side-col d-none d-lg-block col-lg-2">
      
  <aside class="sidebar" style="margin-left: -1rem">
    <div id="toc">
  <p class="toc-header">
    <i class="iconfont icon-list"></i>
    <span>目录</span>
  </p>
  <div class="toc-body" id="toc-body"></div>
</div>



  </aside>


    </div>
  </div>
</div>





  



  



  



  



  


  
  









    

    
      <a id="scroll-top-button" aria-label="TOP" href="#" role="button">
        <i class="iconfont icon-arrowup" aria-hidden="true"></i>
      </a>
    

    
      <div class="modal fade" id="modalSearch" tabindex="-1" role="dialog" aria-labelledby="ModalLabel"
     aria-hidden="true">
  <div class="modal-dialog modal-dialog-scrollable modal-lg" role="document">
    <div class="modal-content">
      <div class="modal-header text-center">
        <h4 class="modal-title w-100 font-weight-bold">搜索</h4>
        <button type="button" id="local-search-close" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body mx-3">
        <div class="md-form mb-5">
          <input type="text" id="local-search-input" class="form-control validate">
          <label data-error="x" data-success="v" for="local-search-input">关键词</label>
        </div>
        <div class="list-group" id="local-search-result"></div>
      </div>
    </div>
  </div>
</div>

    

    
  </main>

  <footer>
    <div class="footer-inner">
  
    <div class="footer-content">
       <a href="https://hexo.io" target="_blank" rel="nofollow noopener"><span>Hexo</span></a> <i class="iconfont icon-love"></i> <a href="https://github.com/fluid-dev/hexo-theme-fluid" target="_blank" rel="nofollow noopener"><span>Fluid</span></a> 
    </div>
  
  
  
</div>

  </footer>

  <!-- Scripts -->
  
  <script  src="https://lib.baomitu.com/nprogress/0.2.0/nprogress.min.js" ></script>
  <link  rel="stylesheet" href="https://lib.baomitu.com/nprogress/0.2.0/nprogress.min.css" />

  <script>
    NProgress.configure({"showSpinner":false,"trickleSpeed":100})
    NProgress.start()
    window.addEventListener('load', function() {
      NProgress.done();
    })
  </script>


<script  src="https://lib.baomitu.com/jquery/3.6.4/jquery.min.js" ></script>
<script  src="https://lib.baomitu.com/twitter-bootstrap/4.6.1/js/bootstrap.min.js" ></script>
<script  src="/js/events.js" ></script>
<script  src="/js/plugins.js" ></script>





  
    <script  src="/js/img-lazyload.js" ></script>
  




  
<script>
  Fluid.utils.createScript('https://lib.baomitu.com/tocbot/4.20.1/tocbot.min.js', function() {
    var toc = jQuery('#toc');
    if (toc.length === 0 || !window.tocbot) { return; }
    var boardCtn = jQuery('#board-ctn');
    var boardTop = boardCtn.offset().top;

    window.tocbot.init(Object.assign({
      tocSelector     : '#toc-body',
      contentSelector : '.markdown-body',
      linkClass       : 'tocbot-link',
      activeLinkClass : 'tocbot-active-link',
      listClass       : 'tocbot-list',
      isCollapsedClass: 'tocbot-is-collapsed',
      collapsibleClass: 'tocbot-is-collapsible',
      scrollSmooth    : true,
      includeTitleTags: true,
      headingsOffset  : -boardTop,
    }, CONFIG.toc));
    if (toc.find('.toc-list-item').length > 0) {
      toc.css('visibility', 'visible');
    }

    Fluid.events.registerRefreshCallback(function() {
      if ('tocbot' in window) {
        tocbot.refresh();
        var toc = jQuery('#toc');
        if (toc.length === 0 || !tocbot) {
          return;
        }
        if (toc.find('.toc-list-item').length > 0) {
          toc.css('visibility', 'visible');
        }
      }
    });
  });
</script>


  <script src=https://lib.baomitu.com/clipboard.js/2.0.11/clipboard.min.js></script>

  <script>Fluid.plugins.codeWidget();</script>


  
<script>
  Fluid.utils.createScript('https://lib.baomitu.com/anchor-js/5.0.0/anchor.min.js', function() {
    window.anchors.options = {
      placement: CONFIG.anchorjs.placement,
      visible  : CONFIG.anchorjs.visible
    };
    if (CONFIG.anchorjs.icon) {
      window.anchors.options.icon = CONFIG.anchorjs.icon;
    }
    var el = (CONFIG.anchorjs.element || 'h1,h2,h3,h4,h5,h6').split(',');
    var res = [];
    for (var item of el) {
      res.push('.markdown-body > ' + item.trim());
    }
    if (CONFIG.anchorjs.placement === 'left') {
      window.anchors.options.class = 'anchorjs-link-left';
    }
    window.anchors.add(res.join(', '));

    Fluid.events.registerRefreshCallback(function() {
      if ('anchors' in window) {
        anchors.removeAll();
        var el = (CONFIG.anchorjs.element || 'h1,h2,h3,h4,h5,h6').split(',');
        var res = [];
        for (var item of el) {
          res.push('.markdown-body > ' + item.trim());
        }
        if (CONFIG.anchorjs.placement === 'left') {
          anchors.options.class = 'anchorjs-link-left';
        }
        anchors.add(res.join(', '));
      }
    });
  });
</script>


  
<script>
  Fluid.utils.createScript('https://lib.baomitu.com/fancybox/3.5.7/jquery.fancybox.min.js', function() {
    Fluid.plugins.fancyBox();
  });
</script>


  <script>Fluid.plugins.imageCaption();</script>

  
      <script>
        if (!window.MathJax) {
          window.MathJax = {
            tex    : {
              inlineMath: { '[+]': [['$', '$']] }
            },
            loader : {
              load: ['ui/lazy']
            },
            options: {
              renderActions: {
                insertedScript: [200, () => {
                  document.querySelectorAll('mjx-container').forEach(node => {
                    let target = node.parentNode;
                    if (target.nodeName.toLowerCase() === 'li') {
                      target.parentNode.classList.add('has-jax');
                    }
                  });
                }, '', false]
              }
            }
          };
        } else {
          MathJax.startup.document.state(0);
          MathJax.texReset();
          MathJax.typeset();
          MathJax.typesetPromise();
        }

        Fluid.events.registerRefreshCallback(function() {
          if ('MathJax' in window && MathJax.startup.document && typeof MathJax.startup.document.state === 'function') {
            MathJax.startup.document.state(0);
            MathJax.texReset();
            MathJax.typeset();
            MathJax.typesetPromise();
          }
        });
      </script>
    

  <script  src="https://lib.baomitu.com/mathjax/3.2.2/es5/tex-mml-chtml.js" ></script>

  <script  src="/js/local-search.js" ></script>





<!-- 主题的启动项，将它保持在最底部 -->
<!-- the boot of the theme, keep it at the bottom -->
<script  src="/js/boot.js" ></script>


  

  <noscript>
    <div class="noscript-warning">博客在允许 JavaScript 运行的环境下浏览效果更佳</div>
  </noscript>
</body>
</html>
