

<!DOCTYPE html>
<html lang="zh-CN" >



<head>
  <meta charset="UTF-8">

  <link rel="apple-touch-icon" sizes="76x76" href="/img/Logo.png">
  <link rel="icon" href="/img/Logo.png">
  

  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=5.0, shrink-to-fit=no">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  
  <meta name="theme-color" content="#2f4154">
  <meta name="author" content="向海">
  <meta name="keywords" content="">
  
    <meta name="description" content="Wed Nov 11 2020 08:33:53 GMT+0800 (中国标准时间)">
<meta property="og:type" content="article">
<meta property="og:title" content="L4 Qualification and Quantification of DNA&amp;RNA">
<meta property="og:url" content="https://zhenyumi.github.io/posts/44583cb1/index.html">
<meta property="og:site_name" content="鹘横海">
<meta property="og:description" content="Wed Nov 11 2020 08:33:53 GMT+0800 (中国标准时间)">
<meta property="og:locale" content="zh_CN">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018130700852.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131057045.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131554115.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131643390.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131826138.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018132558842.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018132850268.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018133000046.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018133017561.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018144429617.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145408023.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145415029.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145433430.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145456455.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145507310.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018150200305.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018150425639.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151608412.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018150749025.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151325377.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151500072.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151540170.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151608412.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151617739.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151623310.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151639945.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151825489.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018152945692.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018152953782.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154101724.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154109739.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154116713.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154123396.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154622953.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154807551.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155029409.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155103826.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155220833.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155233565.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155301313.png">
<meta property="article:published_time" content="2020-11-11T00:33:51.000Z">
<meta property="article:modified_time" content="2020-11-11T00:33:53.000Z">
<meta property="article:author" content="向海">
<meta property="article:tag" content="Genetic Engineering">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018130700852.png">
  
  
  
  <title>L4 Qualification and Quantification of DNA&amp;RNA - 鹘横海</title>

  <link  rel="stylesheet" href="https://lib.baomitu.com/twitter-bootstrap/4.6.1/css/bootstrap.min.css" />



  <link  rel="stylesheet" href="https://lib.baomitu.com/github-markdown-css/4.0.0/github-markdown.min.css" />

  <link  rel="stylesheet" href="https://lib.baomitu.com/hint.css/2.7.0/hint.min.css" />

  <link  rel="stylesheet" href="https://lib.baomitu.com/fancybox/3.5.7/jquery.fancybox.min.css" />



<!-- 主题依赖的图标库，不要自行修改 -->
<!-- Do not modify the link that theme dependent icons -->

<link rel="stylesheet" href="//at.alicdn.com/t/c/font_1749284_5i9bdhy70f8.css">



<link rel="stylesheet" href="//at.alicdn.com/t/c/font_1736178_k526ubmyhba.css">


<link  rel="stylesheet" href="/css/main.css" />


  <link id="highlight-css" rel="stylesheet" href="/css/highlight.css" />
  




  <script id="fluid-configs">
    var Fluid = window.Fluid || {};
    Fluid.ctx = Object.assign({}, Fluid.ctx)
    var CONFIG = {"hostname":"zhenyumi.github.io","root":"/","version":"1.9.8","typing":{"enable":true,"typeSpeed":70,"cursorChar":"_","loop":false,"scope":["home"]},"anchorjs":{"enable":true,"element":"h1,h2,h3,h4,h5,h6","placement":"left","visible":"hover","icon":""},"progressbar":{"enable":true,"height_px":3,"color":"#29d","options":{"showSpinner":false,"trickleSpeed":100}},"code_language":{"enable":true,"default":"TEXT"},"copy_btn":true,"image_caption":{"enable":true},"image_zoom":{"enable":true,"img_url_replace":["",""]},"toc":{"enable":true,"placement":"right","headingSelector":"h1,h2,h3,h4,h5,h6","collapseDepth":0},"lazyload":{"enable":true,"loading_img":"/img/loading.gif","onlypost":false,"offset_factor":2},"web_analytics":{"enable":false,"follow_dnt":true,"baidu":null,"google":{"measurement_id":null},"tencent":{"sid":null,"cid":null},"leancloud":{"app_id":null,"app_key":null,"server_url":null,"path":"window.location.pathname","ignore_local":false},"umami":{"src":null,"website_id":null,"domains":null,"start_time":"2024-01-01T00:00:00.000Z","token":null,"api_server":null}},"search_path":"/local-search.xml","include_content_in_search":true};

    if (CONFIG.web_analytics.follow_dnt) {
      var dntVal = navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack;
      Fluid.ctx.dnt = dntVal && (dntVal.startsWith('1') || dntVal.startsWith('yes') || dntVal.startsWith('on'));
    }
  </script>
  <script  src="/js/utils.js" ></script>
  <script  src="/js/color-schema.js" ></script>
  


  
<meta name="generator" content="Hexo 7.0.0"></head>


<body>
  

  <header>
    

<div class="header-inner" style="height: 70vh;">
  <nav id="navbar" class="navbar fixed-top  navbar-expand-lg navbar-dark scrolling-navbar">
  <div class="container">
    <a class="navbar-brand" href="/">
      <strong>鹘横海</strong>
    </a>

    <button id="navbar-toggler-btn" class="navbar-toggler" type="button" data-toggle="collapse"
            data-target="#navbarSupportedContent"
            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <div class="animated-icon"><span></span><span></span><span></span></div>
    </button>

    <!-- Collapsible content -->
    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav ml-auto text-center">
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/" target="_self">
                <i class="iconfont icon-home-fill"></i>
                <span>首页</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/archives/" target="_self">
                <i class="iconfont icon-archive-fill"></i>
                <span>归档</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/categories/" target="_self">
                <i class="iconfont icon-category-fill"></i>
                <span>分类</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/tags/" target="_self">
                <i class="iconfont icon-tags-fill"></i>
                <span>标签</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/about/" target="_self">
                <i class="iconfont icon-user-fill"></i>
                <span>关于</span>
              </a>
            </li>
          
        
        
          <li class="nav-item" id="search-btn">
            <a class="nav-link" target="_self" href="javascript:;" data-toggle="modal" data-target="#modalSearch" aria-label="Search">
              <i class="iconfont icon-search"></i>
            </a>
          </li>
          
        
        
      </ul>
    </div>
  </div>
</nav>

  

<div id="banner" class="banner" parallax=true
     style="background: url('https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/post/post2.jpg') no-repeat center center; background-size: cover;">
  <div class="full-bg-img">
    <div class="mask flex-center" style="background-color: rgba(0, 0, 0, 0.3)">
      <div class="banner-text text-center fade-in-up">
        <div class="h2">
          
            <span id="subtitle">L4 Qualification and Quantification of DNA&RNA</span>
          
        </div>

        
          
  <div class="mt-3">
    
    
      <span class="post-meta">
        <i class="iconfont icon-date-fill" aria-hidden="true"></i>
        <time datetime="2020-11-11 08:33" pubdate>
          2020年11月11日 早上
        </time>
      </span>
    
  </div>

  <div class="mt-1">
    
      <span class="post-meta mr-2">
        <i class="iconfont icon-chart"></i>
        
          603 字
        
      </span>
    

    
      <span class="post-meta mr-2">
        <i class="iconfont icon-clock-fill"></i>
        
        
        
          6 分钟
        
      </span>
    

    
    
  </div>


        
      </div>

      
    </div>
  </div>
</div>

</div>

  </header>

  <main>
    
      

<div class="container-fluid nopadding-x">
  <div class="row nomargin-x">
    <div class="side-col d-none d-lg-block col-lg-2">
      

    </div>

    <div class="col-lg-8 nopadding-x-md">
      <div class="container nopadding-x-md" id="board-ctn">
        <div id="board">
          <article class="post-content mx-auto">
            <h1 id="seo-header">L4 Qualification and Quantification of DNA&amp;RNA</h1>
            
            
              <div class="markdown-body">
                
                <h1 id="一、Handling-and-Quantification-of-DNA-RNA"><a href="#一、Handling-and-Quantification-of-DNA-RNA" class="headerlink" title="一、Handling and Quantification of DNA&#x2F;RNA"></a>一、Handling and Quantification of DNA&#x2F;RNA</h1><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018130700852.png" srcset="/img/loading.gif" lazyload alt="image-20201018130700852" style="zoom: 33%;" />

<ul>
<li>Pure DNA&#x2F;RNA only form tiny pellet after precipitation. - semi transparent and easy to be dissolved</li>
</ul>
<p>Mainly two types of quantification methods</p>
<ol>
<li>Absorbance based methods<ul>
<li>the principle is simple, the key is the <strong>ratio</strong>.</li>
</ul>
</li>
<li>Fluorescence based methods</li>
</ol>
<h2 id="The-Absorbance-Based-Methods"><a href="#The-Absorbance-Based-Methods" class="headerlink" title="The Absorbance Based Methods"></a>The Absorbance Based Methods</h2><p>The <strong>Beer‐Lambert Law</strong> — draws a direct correlation between absorbance and concentration. While nucleic acids absorb at many wavelengths, <u>they have a peak absorbance of UV light at 260nm</u>. Thus, the amount of light absorbed in this region can be used to determine the concentration of RNA or DNA in solution<br>$$<br>A &#x3D; \varepsilon c l<br>$$</p>
<ul>
<li>A&#x3D;absorbance, </li>
<li>ԑ&#x3D;extinction coefficient </li>
<li>c&#x3D;concentration</li>
<li>l&#x3D;path length</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131057045.png" srcset="/img/loading.gif" lazyload alt="image-20201018131057045" style="zoom:67%;" />

<ul>
<li>The <strong>nucleic acids</strong> (DNA, RNA and oligo) solutions with different concentration has different <u>ability absorbing light.</u><ul>
<li>1 OD<del>260</del> Unit $\rarr$ 50ug&#x2F;ml for double stranded DNA</li>
<li>1 OD<del>260</del> Unit $\rarr$ 40ug&#x2F;ml for single stranded RNA</li>
<li>1 OD<del>260</del> Unit $\rarr$ 33ug&#x2F;ml for single stranded DNA (ssDNA)</li>
</ul>
</li>
</ul>
<h3 id="1-The-Ideal-Ratios"><a href="#1-The-Ideal-Ratios" class="headerlink" title="1. The Ideal Ratios"></a>1. The Ideal Ratios</h3><p>What would be the ideal ratios for pure DNA and or RNA in solution?</p>
<ul>
<li><p>A260&#x2F;280 ratio </p>
<ul>
<li>A l<strong>ow A260&#x2F;A280 ratio</strong> often indicate <u>protein</u> contamination, Aromatic amino acid side chains are responsible for 280nm absorbance.</li>
<li>For DNA: A260&#x2F;280 is nearly 1.8</li>
<li>For RNA: A260&#x2F;280 is nearly 2.</li>
<li><strong>Reason</strong>: RNA will typically have a higher 260&#x2F;280 ratio due to the higher ratio of <strong>Uracil</strong> compared to that of <strong>Thymine</strong></li>
</ul>
</li>
<li><p><strong>A260&#x2F;230 ratio</strong>, ideally more than 1.8</p>
<ul>
<li>The A260&#x2F;230 ratio indicates the presence of organic contaminants which could <u>interfere with downstream applications</u></li>
<li><strong>Low A260&#x2F;A230</strong> often indicate presence of following impurities</li>
</ul>
<ol>
<li><strong>Plant polyphenol</strong> – common contaminants when doing RNA&#x2F;DNA extraction from plant tissues.</li>
<li>Phenol</li>
<li>Chaotropic salts from your lysis buffer</li>
</ol>
</li>
</ul>
<h3 id="2-Real-World-Examples"><a href="#2-Real-World-Examples" class="headerlink" title="2. Real World Examples"></a>2. Real World Examples</h3><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131554115.png" srcset="/img/loading.gif" lazyload alt="image-20201018131554115" style="zoom: 50%;" />

<ul>
<li>the A260&#x2F;230 ratio of 1.29 is significantly lower than 2—indicating some sort of organic contaminants present in the sample</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131643390.png" srcset="/img/loading.gif" lazyload alt="image-20201018131643390" style="zoom:50%;" />

<ul>
<li>This RNA has low A260&#x2F;280 and A260&#x2F;230 ratios. Additionally, the maximum peak is at 270 nm. Most likely due to presence of high concentration of phenol</li>
<li>Note that the concentration of RNA returned by the spectrophotometer is not at all reliable due to the contamination</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131826138.png" srcset="/img/loading.gif" lazyload alt="image-20201018131826138" style="zoom:50%;" />

<h3 id="3-How-to-Keep-The-Ratio-Constant"><a href="#3-How-to-Keep-The-Ratio-Constant" class="headerlink" title="3. How to Keep The Ratio Constant"></a>3. How to Keep The Ratio Constant</h3><ol>
<li>Constant ionic strength – same buffer as blank and dilution</li>
<li>Constant pH – pH 8.0 1xTE buffer is recommanded</li>
</ol>
<h2 id="Fluorescence-Based-Methods"><a href="#Fluorescence-Based-Methods" class="headerlink" title="Fluorescence Based Methods"></a>Fluorescence Based Methods</h2><h3 id="1-The-shortage-of-absorbance-based-method"><a href="#1-The-shortage-of-absorbance-based-method" class="headerlink" title="1. The shortage of absorbance based method"></a>1. The shortage of absorbance based method</h3><ol>
<li>Can’t distinguish between DNA&#x2F;RNA&#x2F;free nucleotide </li>
<li>Might not be accurate when concentration of DNA&#x2F;RNA is low, due to the presence of other impurities. </li>
<li>No size information</li>
<li>Might not be sensitive – below 10ng&#x2F;ul</li>
</ol>
<h3 id="2-Stains"><a href="#2-Stains" class="headerlink" title="2. Stains"></a>2. Stains</h3><p><strong>Fluorescent DNA&#x2F;RNA Stains</strong>– Bind nucleic acids and emit fluorescence when excited. </p>
<ul>
<li>DNA(RNA) Stains are fluorescent dyes that bind nucleic acids and have a wide range of applications, including in flow cytometry, cell-cycle studies, chromosome and nuclei counterstaining, as an indicator of apoptosis, and to quantify DNA</li>
</ul>
<p>The technology involved:</p>
<ul>
<li>Gel electrophoresis</li>
<li>DNA or RNA specific dye enables accurate quantification</li>
<li>Capillary electrophoresis + DNA&#x2F;RNA specific dye</li>
</ul>
<h4 id="Gel-Electrophoresis"><a href="#Gel-Electrophoresis" class="headerlink" title="Gel Electrophoresis"></a>Gel Electrophoresis</h4><p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018132558842.png" srcset="/img/loading.gif" lazyload alt="image-20201018132558842"></p>
<ul>
<li>agarose gel electrophoresis (<strong>SAGE</strong>)</li>
<li>Polyacrylamide-based gel electrophoresis (<strong>PAGE</strong>)</li>
</ul>
<p><strong>Gel electrophoresis</strong> is a typical fluorescence based methods</p>
<ul>
<li>Matrix of choice – <strong>Agarose</strong> vs <strong>Low-melting point agarose</strong>, <strong>Polyacrylamide</strong></li>
<li>Buffer of choice – TAE vs TBE buffer</li>
</ul>
<p>The migration of DNA fragments through an agarose gel</p>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018132850268.png" srcset="/img/loading.gif" lazyload alt="image-20201018132850268"></p>
<p>Polyacrylamide Gel Electrophoresis</p>
<ul>
<li>在制备的过程中，加入了cross-linker</li>
<li>因此制造出了more ordered struacture，同时也更难配置</li>
</ul>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018133000046.png" srcset="/img/loading.gif" lazyload alt="image-20201018133000046"></p>
<p>Migration of dsDNA in gels with different concentration of acrylamide</p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018133017561.png" srcset="/img/loading.gif" lazyload alt="image-20201018133017561" style="zoom: 33%;" />

<p><strong>Gel electrophoresis</strong> is a typical fluorescence based methods. </p>
<ul>
<li>Matrix of choice – Agarose vs Low-melting point agarose, Polyacrylamide</li>
<li><strong>Agarose gel electrophoresis (SAGE)</strong> – Work well for a wide range of DNA length. <u>Maxima resolution is around 10bp, High Capacity.</u></li>
<li><strong>Low-melting-point agarose</strong>, you may choose it for various reason. It is a lot more transparent and have more even pore size compared with normal agarose gel, so it is <u>a better matrix for the detection of DNA&#x2F;RNA in low concentration</u></li>
<li><strong>Polyacrylamide-based gel electrophoresis (PAGE)</strong> – High resolution, <u>1bp resolution is possible</u>. Low Capacity. Good for separate small RNAs and used in DNA sequencing. <ul>
<li>Denatured and Native</li>
</ul>
</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018144429617.png" srcset="/img/loading.gif" lazyload alt="image-20201018144429617" style="zoom:67%;" />

<h4 id="Buffer-of-Choice"><a href="#Buffer-of-Choice" class="headerlink" title="Buffer of Choice"></a>Buffer of Choice</h4><p>Buffer of choice – <strong>TAE</strong> vs <strong>TBE</strong> buffer</p>
<ul>
<li><strong>TAE</strong> is advantageous for high resolution of long nucleic acid fragments (qs) on agarose gels.</li>
<li><strong>TBE</strong> has a greater buffering capacity and will give sharper resolution than TAE, especially when separating small fragments of DNA.</li>
</ul>
<h4 id="Stains"><a href="#Stains" class="headerlink" title="Stains"></a>Stains</h4><p><strong>Fluorescent DNA&#x2F;RNA Stains</strong>– Bind nucleic acids and emit fluorescence when excited</p>
<p>We often use two types of DNA stains in gel electrophoresis and other applications. </p>
<ul>
<li>UV light excited - Ethidium bromide, <u>Gel Red,</u> </li>
<li>Blue light excited - <u>SYBR Green, Gene Green, Gel Green etc</u></li>
</ul>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145408023.png" srcset="/img/loading.gif" lazyload alt="image-20201018145408023"></p>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145415029.png" srcset="/img/loading.gif" lazyload alt="image-20201018145415029"></p>
<p><strong>DNA or RNA specific fluorescent Dye enables accurate quantification of DNA or RNA</strong> </p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145433430.png" srcset="/img/loading.gif" lazyload alt="image-20201018145433430" style="zoom: 67%;" />

<p>The Qubit 1X dsDNA HS (High-Sensitivity) Assay Kit is designed specifically for use with the Qubit Fluorometer. The assay is highly selective for double-stranded DNA (dsDNA) over RNA and is designed to be accurate for initial sample concentrations from 10 pg&#x2F;µL to 100 ng&#x2F;µL.</p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145456455.png" srcset="/img/loading.gif" lazyload alt="image-20201018145456455" style="zoom:67%;" />

<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145507310.png" srcset="/img/loading.gif" lazyload alt="image-20201018145507310" style="zoom:67%;" />

<p>Advantages:</p>
<ol>
<li>accurate – highly selective</li>
<li>sensitive – can deal with sample of very low concentration</li>
</ol>
<p>Disadvantages:</p>
<ol>
<li>Good for quantification but not for qualification (length, integrity).</li>
<li>Concentration rely on external standard</li>
</ol>
<h4 id="Capillary-electrophoresis-DNA-RNA-specific-dye"><a href="#Capillary-electrophoresis-DNA-RNA-specific-dye" class="headerlink" title="Capillary electrophoresis + DNA&#x2F;RNA specific dye"></a>Capillary electrophoresis + DNA&#x2F;RNA specific dye</h4><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018150200305.png" srcset="/img/loading.gif" lazyload alt="image-20201018150200305" style="zoom:67%;" />

<p>Advantages: You get almost all the information you need ! </p>
<p>Disadvantages: Operation with the instrument can be complicated, Expensive (both instrument and reagent)</p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018150425639.png" srcset="/img/loading.gif" lazyload alt="image-20201018150425639" style="zoom:67%;" />



<h1 id="二、Detection-of-Nucleic-acids-through-blotting-techniques-Nucleic-acids-hybridization"><a href="#二、Detection-of-Nucleic-acids-through-blotting-techniques-Nucleic-acids-hybridization" class="headerlink" title="二、Detection of Nucleic acids through blotting techniques (Nucleic acids hybridization)"></a>二、Detection of Nucleic acids through blotting techniques (Nucleic acids hybridization)</h1><p>Nucleic Acid Hybridization: Detection of nucleic acids with known sequence from complex samples !</p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151608412.png" srcset="/img/loading.gif" lazyload alt="image-20201018151608412" style="zoom:50%;" />

<h2 id="Introduction"><a href="#Introduction" class="headerlink" title="Introduction"></a>Introduction</h2><p>A basic technique in molecular biology. in which single stranded Nucleic acids are allowed to interact so that complexes called <strong>HYBRIDS</strong> are formed by molecules with similar complementary sequences</p>
<p>The essence of nucleic acids hybridization is the denaturation and renaturation (<strong>annealing</strong>) of DNA (or RNA). </p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018150749025.png" srcset="/img/loading.gif" lazyload alt="image-20201018150749025" style="zoom:50%;" />

<p>The principle of hybridization is the addition of a probe to a complex mixture of target DNA. The mixture is incubated under conditions that promote the formation of hydrogen bonds between complementary strands.</p>
<h2 id="Different-Blotting"><a href="#Different-Blotting" class="headerlink" title="Different Blotting"></a>Different Blotting</h2><p>Nucleic acid hybridization – A brief history of Southern&#x2F;Northern&#x2F;Western blot</p>
<ul>
<li>In 1975 when Ed Southern invented his method of using a radiolabeled <font color="FF0000">DNA probe</font> to <u>detect a specific DNA sequence within a DNA sample</u> (e.g. a fractionated genome) and named it after himself – the <strong>Southern blot</strong></li>
<li>In 1977, J.C. Alwine, a biologist with a sense of humor, developed <strong>Northern Blot</strong>, a technique analogous to the Southern blot, this time for the <u>identification of a specific RNA within a complex RNA sample</u> using a <font color="FF0000">radio-labelled DNA probe</font>.</li>
<li>W. Neal Burnette, a post-doc working in the Nowinski group at the Hutchinson Cancer Center in Seattle, developed a technique call western<ul>
<li>Seattle sitting in the west cost of US. If it is in New York, it is another story perhaps.</li>
<li>Burnette was searching for a way to combine the powers of radio immunoassay and SDS-PAGE electrophoresis so that he could <u>pinpoint specific antigens in a complex protein mixture</u></li>
</ul>
</li>
</ul>
<h3 id="1-Southern-Blotting"><a href="#1-Southern-Blotting" class="headerlink" title="1. Southern Blotting"></a>1. Southern Blotting</h3><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151325377.png" srcset="/img/loading.gif" lazyload alt="image-20201018151325377" style="zoom:50%;" />

<ul>
<li>The first type of blotting to be discovered</li>
<li>Plays significant role in <strong>Recombinant DNA Technology</strong> as well as <strong>Molecular Biology</strong>.</li>
<li>Used to <strong>detect target DNA</strong> in a sample</li>
</ul>
<p><strong>Usage of Southern blot:</strong> To detect <strong>a specific DNA sequence</strong> <strong>within a complex DNA sample.</strong></p>
<ul>
<li>Detection of <strong>copy number</strong> of a certain gene</li>
<li><strong>Genotyping</strong> – different restriction patterns between two alleles of a same gene</li>
<li>Detection of replication intermediate – 2D gel + southern blot.</li>
</ul>
<p>Detection of an RFLP (Restriction fragment length polymorphism) by Southern Blotting:</p>
<blockquote>
<p>In molecular biology, restriction fragment length polymorphism (RFLP) is a technique that exploits variations in homologous DNA sequences, known as polymorphisms, in order to distinguish individuals, populations, or species or to pinpoint the locations of genes within a sequence.</p>
</blockquote>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151500072.png" srcset="/img/loading.gif" lazyload alt="image-20201018151500072"></p>
<p>Detection of DNA defect &amp; DNA segment (Gene Defect) (Detection of the sickle cell globin gene)</p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151540170.png" srcset="/img/loading.gif" lazyload alt="image-20201018151540170" style="zoom: 33%;" />

<ul>
<li>Using restriction enzymes we can prove that they are different types of genes.</li>
<li>Normal DNA- 2 segments</li>
<li>Sickle cell- 1</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151608412.png" srcset="/img/loading.gif" lazyload alt="image-20201018151608412" style="zoom: 33%;" />

<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151617739.png" srcset="/img/loading.gif" lazyload alt="image-20201018151617739"></p>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151623310.png" srcset="/img/loading.gif" lazyload alt="image-20201018151623310"></p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151639945.png" srcset="/img/loading.gif" lazyload alt="image-20201018151639945" style="zoom: 25%;" />

<h4 id="Denature-of-dsDNA-Alkaline-denaturation"><a href="#Denature-of-dsDNA-Alkaline-denaturation" class="headerlink" title="Denature of dsDNA: Alkaline denaturation"></a>Denature of dsDNA: Alkaline denaturation</h4><p>At <u>high pH</u>, then, the solution is <u>rich in hydroxide ions</u>, and these negatively-charged ions can <u>pull hydrogen ions off</u> of molecules like <u>the base pairs in DNA</u>. This process disrupts the hydrogen bonding that holds the two DNA strands together, causing them to separate.</p>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151825489.png" srcset="/img/loading.gif" lazyload alt="image-20201018151825489"></p>
<p>Unlike RNA, DNA lacks <strong>a hydroxyl group on the 2’ position in each</strong> <strong>sugar group</strong>. This difference makes DNA much more stable in alkaline solution. </p>
<p><u>In RNA</u>, the hydroxyl group on the 2’ position can <u>give up a hydrogen ion</u> to the solution at high pH, creating <strong>a highly</strong> <strong>reactive</strong> <strong>alkoxide ion</strong> that <u>attacks the phosphate group</u> holding two neighboring nucleotides together. </p>
<ul>
<li>DNA does not suffer from this defect and thus enjoys remarkable stability at high pH.</li>
</ul>
<h3 id="2-Northern-Blot"><a href="#2-Northern-Blot" class="headerlink" title="2. Northern Blot"></a>2. Northern Blot</h3><p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018152945692.png" srcset="/img/loading.gif" lazyload alt="image-20201018152945692"></p>
<h3 id="3-Western-Blot"><a href="#3-Western-Blot" class="headerlink" title="3. Western Blot"></a>3. Western Blot</h3><p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018152953782.png" srcset="/img/loading.gif" lazyload alt="image-20201018152953782"></p>
<ol>
<li>Load and separate protein samples on SDS-PAGE</li>
<li>Electrophoretically transfer fractionated proteins onto PVDF membrane</li>
<li>Block the membrane with neutral protein (BSA or milk casein)</li>
<li>Incubate the membrane with primary antibody specific to target protein</li>
<li>Incubate the membrane with Hrp-labeled secondary antibody specific to primary antibody</li>
<li>Incubate the blot with chemiluminescent HRP substrate and expose to film</li>
</ol>
<h3 id="4-Reporter-Enzymes-in-Blotting-HRP-vs-AP"><a href="#4-Reporter-Enzymes-in-Blotting-HRP-vs-AP" class="headerlink" title="4. Reporter Enzymes in Blotting- HRP vs AP"></a>4. Reporter Enzymes in Blotting- HRP vs AP</h3><p>The most common reporter enzymes are <strong>horseradish peroxidase (HRP)</strong> or <strong>alkaline phosphatase (AP)</strong>. Chemiluminescent substrates for <u>both</u> HRP and AP are <u>very sensitive</u> and can allow picogram to femtogram level detection.</p>
<ul>
<li>The maximum signal generated from <u>HRP substrates is very fast, often within 5 minutes</u>. </li>
<li>The signal from AP substrates, however, gradually increases over time with a signal plateau around 60 minutes. The signal from AP substrates is very stable and can last up to several days, which can be useful if multiple exposures are needed</li>
</ul>
<p>Generally, dilution factors for the AP 2° antibodies can be greater than with HRP antibodies, thus saving reagents. The disadvantages of either <u>HRP or AP system are interferences</u>, such as azides for HRP or phosphate buffers for AP. There is also the possibility that endogenous phosphatases will react with AP substrates.</p>
<h2 id="Type-of-Filters-in-blotting-techniques"><a href="#Type-of-Filters-in-blotting-techniques" class="headerlink" title="Type of Filters in blotting techniques"></a>Type of Filters in blotting techniques</h2><p><strong>Nitrocellulose</strong> Membrane (NC) - For the detection of nucleic acid and Protein. </p>
<ul>
<li>DNA&#x2F;RNA covalently bound to membrane.</li>
</ul>
<p><strong>Positively charged <u>Nylon Membrane</u></strong> – For DNA&#x2F;RNA only</p>
<p><strong>PVDF Membrane</strong> – For Protein only, a pre-treatment with methanol is required</p>
<h2 id="Principles-of-molecular-hybridization"><a href="#Principles-of-molecular-hybridization" class="headerlink" title="Principles of molecular hybridization"></a>Principles of molecular hybridization</h2><p>Hybridization involves annealing between two strand of DNA and or RNA with partially or fully complementary sequence. </p>
<ul>
<li>T<del>m</del> (Melting Curve)<ul>
<li><strong>Tm</strong>, Heating the DNA solution above a characteristic <strong>temperature</strong> can <u>separate the two strands</u> of a double helix. That temperature is called the melting temperature</li>
<li>T<del>m50</del>: 50% DNA become single stranded</li>
<li>Factors affect Tm would determine the <u>stringency</u> of hybridization<ul>
<li><strong>High stringency</strong> means a condition only allows <u>perfect match</u> between two piece of ssDNA. (高严格度)</li>
</ul>
</li>
</ul>
</li>
<li>Stringency of hybridization (Influenced by the following reasons)<ul>
<li>GC content<ul>
<li><strong>GC content</strong>, Because a G–C base pair has three hydrogen bonds and an A–T pair only has two, nucleic acid double helices with a <strong>high G+C</strong> content have a <strong>higher Tm</strong> than do those with a greater proportion of A+T.</li>
</ul>
</li>
<li>Salt concentration<ul>
<li>Salt: High Salt concentration reduce stringency, Low salt concentration enhance stringency.</li>
<li><u>A high concentration of salt</u> (such as NaCl) <u>raises the Tm</u> of a DNA duplex, because the positive Na+ ions shield the <u>negative charges on the phosphodiester backbone from repelling each other</u> </li>
<li><u>High salt decrease the stringency</u> &amp; Low salt increase the stringency – of hybridization<ul>
<li>Oppose to the protein-protein interaction</li>
</ul>
</li>
</ul>
</li>
<li>Solvent<ul>
<li><strong>Organic Solvent</strong>: DMSO, DMF, Formamide etc, certain organic solvents can cause the negative charges on the phosphates to repel more strongly, thus <u>reducing the Tm.</u> </li>
<li>Create hydrophobic environment $\rarr$ de-stable hydrogen bonds</li>
</ul>
</li>
</ul>
</li>
</ul>
<hr>
<p>Blotting techniques have been widely employed for more than 30 years and have provided the foundation of our understanding of molecular biology.</p>
<ul>
<li>However, these techniques have been largely—and in some cases completely— usurped by new technologies .especially the next generation sequencing techniques</li>
</ul>
<p>Southern blots have been replaced by multiple techniques.</p>
<ol>
<li><strong>Real-time PCR</strong>: boasts incredible sensitivity; theoretically, this method is able to detect even a single copy of the target sequence and compare relative copy numbers across samples rapidly and reliably, with little technical expertise required</li>
<li><strong>Fluorescent <em>in situ</em> hybridization (FISH)</strong> allows detection of specific sequences within a tissue sample with high sensitivity and precise localization</li>
</ol>
<p>Northern blots have given way to <strong>reverse-transcription real-time PCR</strong>, again a more sensitive and more user-friendly technique.</p>
<h1 id="三、Labeling-methods-of-Nucleic-Acid"><a href="#三、Labeling-methods-of-Nucleic-Acid" class="headerlink" title="三、Labeling methods of Nucleic Acid"></a>三、Labeling methods of Nucleic Acid</h1><p>Radioactive and non-radioactive labelling</p>
<ul>
<li>Radioactive labelling: <strong>Unbeatable sensitivity</strong>, Common radioactive isotope used in in molecular biology: ^32^P, ^3^H, ^125^I, ^35^S. ^14^C.</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154101724.png" srcset="/img/loading.gif" lazyload alt="image-20201018154101724" style="zoom:67%;" />

<ul>
<li>Gamma-^32^P ATP used in the End Labelling</li>
<li>Alpha- ^32^P dNTP can be used in any elongation reaction mediated by polymerase</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154109739.png" srcset="/img/loading.gif" lazyload alt="image-20201018154109739" style="zoom:50%;" />

<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154116713.png" srcset="/img/loading.gif" lazyload alt="image-20201018154116713" style="zoom:67%;" />

<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154123396.png" srcset="/img/loading.gif" lazyload alt="image-20201018154123396"></p>
<h2 id="Common-labeling-methods-of-Nucleic-Acid"><a href="#Common-labeling-methods-of-Nucleic-Acid" class="headerlink" title="Common labeling methods of Nucleic Acid"></a>Common labeling methods of Nucleic Acid</h2><ul>
<li>End labelling 末端标记</li>
<li>Nick translation 切口平移</li>
<li>Primer extension 引物延伸</li>
<li>PCR labeling</li>
</ul>
<h3 id="1-5’-End-Labeling"><a href="#1-5’-End-Labeling" class="headerlink" title="1. 5’ End Labeling"></a>1. 5’ End Labeling</h3><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154622953.png" srcset="/img/loading.gif" lazyload alt="image-20201018154622953" />

<ul>
<li>AP: Alkaline phosphatase 碱性磷酸酶</li>
<li>PNK: Polynucleotide Kinase 多聚核苷酸激酶<ul>
<li>Highly efficient, accurate</li>
<li>Specific activity is relatively low (因为只标记了末端，中间部分没有标记)</li>
</ul>
</li>
</ul>
<h3 id="2-Nick-Translation"><a href="#2-Nick-Translation" class="headerlink" title="2. Nick Translation"></a>2. Nick Translation</h3><p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154807551.png" srcset="/img/loading.gif" lazyload alt="image-20201018154807551"></p>
<p>Nick: Low concentration of <strong>DNase I</strong> breaks the DNA strand, remove one nucleotide and leaves 5’ P and 3’ OH.</p>
<ul>
<li><em>E.coli</em> DNA polymerase I <u>has 5’ to 3’ exonuclease activity Polymerase activity</u></li>
<li>One strand is degraded and replaced by an newly synthesized strand</li>
</ul>
<p>Produce relatively short DNA(Radioactively labelled).</p>
<ul>
<li>Need lots of templates (no amplification)</li>
<li>Accurate – amplification free</li>
<li>Label both strand 可以实现对两条链都进行标记</li>
</ul>
<h3 id="3-Primer-extension"><a href="#3-Primer-extension" class="headerlink" title="3. Primer extension"></a>3. Primer extension</h3><p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155029409.png" srcset="/img/loading.gif" lazyload alt="image-20201018155029409"></p>
<ol>
<li>DNA is denatured to give single-stranded molecules.</li>
<li>An oligonucleotide primer is then added to give a short double-stranded region with a free 3-OH group. </li>
<li>The <strong>Klenow fragment</strong> of DNA polymerase I can then synthesize a copy of the template strand from the primer, incorporating [α-^32^P]DNTP(solid circles)to produce a labelled molecule with a very high specific activity</li>
</ol>
<p>Primer；</p>
<ul>
<li>Klenow fragment<ul>
<li>Strand-specific</li>
<li>High specific activity</li>
</ul>
</li>
</ul>
<h3 id="4-Random-prime-Labeling"><a href="#4-Random-prime-Labeling" class="headerlink" title="4. Random prime Labeling"></a>4. Random prime Labeling</h3><p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155103826.png" srcset="/img/loading.gif" lazyload alt="image-20201018155103826"></p>
<p>在DNA单链条件下，加入随机的（并不局限于特定序列的）引物，进行扩增。最终实现标记。</p>
<h2 id="Non-radioactive-labeling"><a href="#Non-radioactive-labeling" class="headerlink" title="Non-radioactive labeling"></a>Non-radioactive labeling</h2><ul>
<li><strong>Digoxigenin</strong>-11-dUTP Anti-DIG-AP</li>
<li><strong>Biotin</strong>-11-dUTP Streptavidin-HRP</li>
<li><strong>Cy3&#x2F;Cy5</strong> – UTP Excitation-Emission</li>
<li>这些方法适用于copy数高，表达量高的基因</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155220833.png" srcset="/img/loading.gif" lazyload alt="image-20201018155220833" style="zoom:67%;" />

<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155233565.png" srcset="/img/loading.gif" lazyload alt="image-20201018155233565" style="zoom:67%;" />

<h2 id="Detection-of-signal-after-hybridization"><a href="#Detection-of-signal-after-hybridization" class="headerlink" title="Detection of signal after hybridization"></a>Detection of signal after hybridization</h2><p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155301313.png" srcset="/img/loading.gif" lazyload alt="image-20201018155301313"></p>
<ul>
<li>^32^P mainly emit beta particles</li>
<li>This can be detected by <strong>X-ray film</strong> or The phosphor screen <ul>
<li>Phosphor screen can stably absorb energy and then release it upon laser stimulation</li>
</ul>
</li>
<li>Fluorescence is often being detected if nonradioactive labelling method is used</li>
</ul>
<hr>
<h1 id="Summary"><a href="#Summary" class="headerlink" title="Summary"></a>Summary</h1><p>Handling and Quantification of DNA&#x2F;RNA</p>
<ul>
<li>Absorbance vs Fluorescence based – OD260&#x2F;280, OD260&#x2F;230 Electrophoresis (Pros and Cons of different matrix)</li>
</ul>
<p>Nucleic acids hybridization – different blotting techniques</p>
<ul>
<li>Southern&#x2F;Northern blot&#x2F;Western, Principle of hybridization (Idea of stringency), Type of Membrane, Alkaline Denaturation&#x2F;Hydrolysis</li>
</ul>
<p>Labeling methods of Nucleic Acid</p>
<ul>
<li>Radioactive VS Non-radioactive – common isotopes – common labeling methods –Signal detections.</li>
</ul>

                
              </div>
            
            <hr/>
            <div>
              <div class="post-metas my-3">
  
    <div class="post-meta mr-3 d-flex align-items-center">
      <i class="iconfont icon-category"></i>
      

<span class="category-chains">
  
  
    
      <span class="category-chain">
        
  <a href="/categories/Academic-Notebook/" class="category-chain-item">Academic Notebook</a>
  
  
    <span>></span>
    
  <a href="/categories/Academic-Notebook/Genetic-Engineering/" class="category-chain-item">Genetic Engineering</a>
  
  

  

      </span>
    
  
</span>

    </div>
  
  
    <div class="post-meta">
      <i class="iconfont icon-tags"></i>
      
        <a href="/tags/Genetic-Engineering/" class="print-no-link">#Genetic Engineering</a>
      
    </div>
  
</div>


              
  

  <div class="license-box my-3">
    <div class="license-title">
      <div>L4 Qualification and Quantification of DNA&amp;RNA</div>
      <div>https://zhenyumi.github.io/posts/44583cb1/</div>
    </div>
    <div class="license-meta">
      
        <div class="license-meta-item">
          <div>作者</div>
          <div>向海</div>
        </div>
      
      
        <div class="license-meta-item license-meta-date">
          <div>发布于</div>
          <div>2020年11月11日</div>
        </div>
      
      
      
        <div class="license-meta-item">
          <div>许可协议</div>
          <div>
            
              
              
                <a class="print-no-link" target="_blank" href="https://creativecommons.org/licenses/by/4.0/">
                  <span class="hint--top hint--rounded" aria-label="BY - 署名">
                    <i class="iconfont icon-cc-by"></i>
                  </span>
                </a>
              
            
          </div>
        </div>
      
    </div>
    <div class="license-icon iconfont"></div>
  </div>



              
                <div class="post-prevnext my-3">
                  <article class="post-prev col-6">
                    
                    
                      <a href="/posts/44c53e21/" title="L3  Isolation of DNA and RNAs">
                        <i class="iconfont icon-arrowleft"></i>
                        <span class="hidden-mobile">L3  Isolation of DNA and RNAs</span>
                        <span class="visible-mobile">上一篇</span>
                      </a>
                    
                  </article>
                  <article class="post-next col-6">
                    
                    
                      <a href="/posts/dd37126d/" title="L5 DNA sequencing">
                        <span class="hidden-mobile">L5 DNA sequencing</span>
                        <span class="visible-mobile">下一篇</span>
                        <i class="iconfont icon-arrowright"></i>
                      </a>
                    
                  </article>
                </div>
              
            </div>

            
          </article>
        </div>
      </div>
    </div>

    <div class="side-col d-none d-lg-block col-lg-2">
      
  <aside class="sidebar" style="margin-left: -1rem">
    <div id="toc">
  <p class="toc-header">
    <i class="iconfont icon-list"></i>
    <span>目录</span>
  </p>
  <div class="toc-body" id="toc-body"></div>
</div>



  </aside>


    </div>
  </div>
</div>





  



  



  



  



  


  
  









    

    
      <a id="scroll-top-button" aria-label="TOP" href="#" role="button">
        <i class="iconfont icon-arrowup" aria-hidden="true"></i>
      </a>
    

    
      <div class="modal fade" id="modalSearch" tabindex="-1" role="dialog" aria-labelledby="ModalLabel"
     aria-hidden="true">
  <div class="modal-dialog modal-dialog-scrollable modal-lg" role="document">
    <div class="modal-content">
      <div class="modal-header text-center">
        <h4 class="modal-title w-100 font-weight-bold">搜索</h4>
        <button type="button" id="local-search-close" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body mx-3">
        <div class="md-form mb-5">
          <input type="text" id="local-search-input" class="form-control validate">
          <label data-error="x" data-success="v" for="local-search-input">关键词</label>
        </div>
        <div class="list-group" id="local-search-result"></div>
      </div>
    </div>
  </div>
</div>

    

    
  </main>

  <footer>
    <div class="footer-inner">
  
    <div class="footer-content">
       <a href="https://hexo.io" target="_blank" rel="nofollow noopener"><span>Hexo</span></a> <i class="iconfont icon-love"></i> <a href="https://github.com/fluid-dev/hexo-theme-fluid" target="_blank" rel="nofollow noopener"><span>Fluid</span></a> 
    </div>
  
  
  
</div>

  </footer>

  <!-- Scripts -->
  
  <script  src="https://lib.baomitu.com/nprogress/0.2.0/nprogress.min.js" ></script>
  <link  rel="stylesheet" href="https://lib.baomitu.com/nprogress/0.2.0/nprogress.min.css" />

  <script>
    NProgress.configure({"showSpinner":false,"trickleSpeed":100})
    NProgress.start()
    window.addEventListener('load', function() {
      NProgress.done();
    })
  </script>


<script  src="https://lib.baomitu.com/jquery/3.6.4/jquery.min.js" ></script>
<script  src="https://lib.baomitu.com/twitter-bootstrap/4.6.1/js/bootstrap.min.js" ></script>
<script  src="/js/events.js" ></script>
<script  src="/js/plugins.js" ></script>





  
    <script  src="/js/img-lazyload.js" ></script>
  




  
<script>
  Fluid.utils.createScript('https://lib.baomitu.com/tocbot/4.20.1/tocbot.min.js', function() {
    var toc = jQuery('#toc');
    if (toc.length === 0 || !window.tocbot) { return; }
    var boardCtn = jQuery('#board-ctn');
    var boardTop = boardCtn.offset().top;

    window.tocbot.init(Object.assign({
      tocSelector     : '#toc-body',
      contentSelector : '.markdown-body',
      linkClass       : 'tocbot-link',
      activeLinkClass : 'tocbot-active-link',
      listClass       : 'tocbot-list',
      isCollapsedClass: 'tocbot-is-collapsed',
      collapsibleClass: 'tocbot-is-collapsible',
      scrollSmooth    : true,
      includeTitleTags: true,
      headingsOffset  : -boardTop,
    }, CONFIG.toc));
    if (toc.find('.toc-list-item').length > 0) {
      toc.css('visibility', 'visible');
    }

    Fluid.events.registerRefreshCallback(function() {
      if ('tocbot' in window) {
        tocbot.refresh();
        var toc = jQuery('#toc');
        if (toc.length === 0 || !tocbot) {
          return;
        }
        if (toc.find('.toc-list-item').length > 0) {
          toc.css('visibility', 'visible');
        }
      }
    });
  });
</script>


  <script src=https://lib.baomitu.com/clipboard.js/2.0.11/clipboard.min.js></script>

  <script>Fluid.plugins.codeWidget();</script>


  
<script>
  Fluid.utils.createScript('https://lib.baomitu.com/anchor-js/5.0.0/anchor.min.js', function() {
    window.anchors.options = {
      placement: CONFIG.anchorjs.placement,
      visible  : CONFIG.anchorjs.visible
    };
    if (CONFIG.anchorjs.icon) {
      window.anchors.options.icon = CONFIG.anchorjs.icon;
    }
    var el = (CONFIG.anchorjs.element || 'h1,h2,h3,h4,h5,h6').split(',');
    var res = [];
    for (var item of el) {
      res.push('.markdown-body > ' + item.trim());
    }
    if (CONFIG.anchorjs.placement === 'left') {
      window.anchors.options.class = 'anchorjs-link-left';
    }
    window.anchors.add(res.join(', '));

    Fluid.events.registerRefreshCallback(function() {
      if ('anchors' in window) {
        anchors.removeAll();
        var el = (CONFIG.anchorjs.element || 'h1,h2,h3,h4,h5,h6').split(',');
        var res = [];
        for (var item of el) {
          res.push('.markdown-body > ' + item.trim());
        }
        if (CONFIG.anchorjs.placement === 'left') {
          anchors.options.class = 'anchorjs-link-left';
        }
        anchors.add(res.join(', '));
      }
    });
  });
</script>


  
<script>
  Fluid.utils.createScript('https://lib.baomitu.com/fancybox/3.5.7/jquery.fancybox.min.js', function() {
    Fluid.plugins.fancyBox();
  });
</script>


  <script>Fluid.plugins.imageCaption();</script>

  
      <script>
        if (!window.MathJax) {
          window.MathJax = {
            tex    : {
              inlineMath: { '[+]': [['$', '$']] }
            },
            loader : {
              load: ['ui/lazy']
            },
            options: {
              renderActions: {
                insertedScript: [200, () => {
                  document.querySelectorAll('mjx-container').forEach(node => {
                    let target = node.parentNode;
                    if (target.nodeName.toLowerCase() === 'li') {
                      target.parentNode.classList.add('has-jax');
                    }
                  });
                }, '', false]
              }
            }
          };
        } else {
          MathJax.startup.document.state(0);
          MathJax.texReset();
          MathJax.typeset();
          MathJax.typesetPromise();
        }

        Fluid.events.registerRefreshCallback(function() {
          if ('MathJax' in window && MathJax.startup.document && typeof MathJax.startup.document.state === 'function') {
            MathJax.startup.document.state(0);
            MathJax.texReset();
            MathJax.typeset();
            MathJax.typesetPromise();
          }
        });
      </script>
    

  <script  src="https://lib.baomitu.com/mathjax/3.2.2/es5/tex-mml-chtml.js" ></script>

  <script  src="/js/local-search.js" ></script>





<!-- 主题的启动项，将它保持在最底部 -->
<!-- the boot of the theme, keep it at the bottom -->
<script  src="/js/boot.js" ></script>


  

  <noscript>
    <div class="noscript-warning">博客在允许 JavaScript 运行的环境下浏览效果更佳</div>
  </noscript>
</body>
</html>
