

<!DOCTYPE html>
<html lang="zh-CN" >



<head>
  <meta charset="UTF-8">

  <link rel="apple-touch-icon" sizes="76x76" href="/img/Logo.png">
  <link rel="icon" href="/img/Logo.png">
  

  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=5.0, shrink-to-fit=no">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  
  <meta name="theme-color" content="#2f4154">
  <meta name="author" content="向海">
  <meta name="keywords" content="">
  
    <meta name="description" content="Enzyme-coupled cell surface receptors Like GPCRs, enzyme-coupled receptors are transmembrane proteins with their ligand-binding domain on the outer surface of the plasma membrane Instead of  having a">
<meta property="og:type" content="article">
<meta property="og:title" content="L17 Cell communication via enzyme-coupled receptors">
<meta property="og:url" content="https://zhenyumi.github.io/posts/1362cf09/index.html">
<meta property="og:site_name" content="鹘横海">
<meta property="og:description" content="Enzyme-coupled cell surface receptors Like GPCRs, enzyme-coupled receptors are transmembrane proteins with their ligand-binding domain on the outer surface of the plasma membrane Instead of  having a">
<meta property="og:locale" content="zh_CN">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Cell_Biology/Cell_Biology_01.jpg">
<meta property="article:published_time" content="2020-07-18T10:49:00.000Z">
<meta property="article:modified_time" content="2020-07-18T10:49:00.000Z">
<meta property="article:author" content="向海">
<meta property="article:tag" content="Cell Biology">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Cell_Biology/Cell_Biology_01.jpg">
  
  
  
  <title>L17 Cell communication via enzyme-coupled receptors - 鹘横海</title>

  <link  rel="stylesheet" href="https://lib.baomitu.com/twitter-bootstrap/4.6.1/css/bootstrap.min.css" />



  <link  rel="stylesheet" href="https://lib.baomitu.com/github-markdown-css/4.0.0/github-markdown.min.css" />

  <link  rel="stylesheet" href="https://lib.baomitu.com/hint.css/2.7.0/hint.min.css" />

  <link  rel="stylesheet" href="https://lib.baomitu.com/fancybox/3.5.7/jquery.fancybox.min.css" />



<!-- 主题依赖的图标库，不要自行修改 -->
<!-- Do not modify the link that theme dependent icons -->

<link rel="stylesheet" href="//at.alicdn.com/t/c/font_1749284_5i9bdhy70f8.css">



<link rel="stylesheet" href="//at.alicdn.com/t/c/font_1736178_k526ubmyhba.css">


<link  rel="stylesheet" href="/css/main.css" />


  <link id="highlight-css" rel="stylesheet" href="/css/highlight.css" />
  




  <script id="fluid-configs">
    var Fluid = window.Fluid || {};
    Fluid.ctx = Object.assign({}, Fluid.ctx)
    var CONFIG = {"hostname":"zhenyumi.github.io","root":"/","version":"1.9.8","typing":{"enable":true,"typeSpeed":70,"cursorChar":"_","loop":false,"scope":["home"]},"anchorjs":{"enable":true,"element":"h1,h2,h3,h4,h5,h6","placement":"left","visible":"hover","icon":""},"progressbar":{"enable":true,"height_px":3,"color":"#29d","options":{"showSpinner":false,"trickleSpeed":100}},"code_language":{"enable":true,"default":"TEXT"},"copy_btn":true,"image_caption":{"enable":true},"image_zoom":{"enable":true,"img_url_replace":["",""]},"toc":{"enable":true,"placement":"right","headingSelector":"h1,h2,h3,h4,h5,h6","collapseDepth":0},"lazyload":{"enable":true,"loading_img":"/img/loading.gif","onlypost":false,"offset_factor":2},"web_analytics":{"enable":false,"follow_dnt":true,"baidu":null,"google":{"measurement_id":null},"tencent":{"sid":null,"cid":null},"leancloud":{"app_id":null,"app_key":null,"server_url":null,"path":"window.location.pathname","ignore_local":false},"umami":{"src":null,"website_id":null,"domains":null,"start_time":"2024-01-01T00:00:00.000Z","token":null,"api_server":null}},"search_path":"/local-search.xml","include_content_in_search":true};

    if (CONFIG.web_analytics.follow_dnt) {
      var dntVal = navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack;
      Fluid.ctx.dnt = dntVal && (dntVal.startsWith('1') || dntVal.startsWith('yes') || dntVal.startsWith('on'));
    }
  </script>
  <script  src="/js/utils.js" ></script>
  <script  src="/js/color-schema.js" ></script>
  


  
<meta name="generator" content="Hexo 7.0.0"></head>


<body>
  

  <header>
    

<div class="header-inner" style="height: 70vh;">
  <nav id="navbar" class="navbar fixed-top  navbar-expand-lg navbar-dark scrolling-navbar">
  <div class="container">
    <a class="navbar-brand" href="/">
      <strong>鹘横海</strong>
    </a>

    <button id="navbar-toggler-btn" class="navbar-toggler" type="button" data-toggle="collapse"
            data-target="#navbarSupportedContent"
            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <div class="animated-icon"><span></span><span></span><span></span></div>
    </button>

    <!-- Collapsible content -->
    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav ml-auto text-center">
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/" target="_self">
                <i class="iconfont icon-home-fill"></i>
                <span>首页</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/archives/" target="_self">
                <i class="iconfont icon-archive-fill"></i>
                <span>归档</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/categories/" target="_self">
                <i class="iconfont icon-category-fill"></i>
                <span>分类</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/tags/" target="_self">
                <i class="iconfont icon-tags-fill"></i>
                <span>标签</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/about/" target="_self">
                <i class="iconfont icon-user-fill"></i>
                <span>关于</span>
              </a>
            </li>
          
        
        
          <li class="nav-item" id="search-btn">
            <a class="nav-link" target="_self" href="javascript:;" data-toggle="modal" data-target="#modalSearch" aria-label="Search">
              <i class="iconfont icon-search"></i>
            </a>
          </li>
          
        
        
      </ul>
    </div>
  </div>
</nav>

  

<div id="banner" class="banner" parallax=true
     style="background: url('https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Cell_Biology/Cell_Biology_02.png') no-repeat center center; background-size: cover;">
  <div class="full-bg-img">
    <div class="mask flex-center" style="background-color: rgba(0, 0, 0, 0.3)">
      <div class="banner-text text-center fade-in-up">
        <div class="h2">
          
            <span id="subtitle">L17 Cell communication via enzyme-coupled receptors</span>
          
        </div>

        
          
  <div class="mt-3">
    
    
      <span class="post-meta">
        <i class="iconfont icon-date-fill" aria-hidden="true"></i>
        <time datetime="2020-07-18 18:49" pubdate>
          2020年7月18日 晚上
        </time>
      </span>
    
  </div>

  <div class="mt-1">
    
      <span class="post-meta mr-2">
        <i class="iconfont icon-chart"></i>
        
          1.5k 字
        
      </span>
    

    
      <span class="post-meta mr-2">
        <i class="iconfont icon-clock-fill"></i>
        
        
        
          13 分钟
        
      </span>
    

    
    
  </div>


        
      </div>

      
    </div>
  </div>
</div>

</div>

  </header>

  <main>
    
      

<div class="container-fluid nopadding-x">
  <div class="row nomargin-x">
    <div class="side-col d-none d-lg-block col-lg-2">
      

    </div>

    <div class="col-lg-8 nopadding-x-md">
      <div class="container nopadding-x-md" id="board-ctn">
        <div id="board">
          <article class="post-content mx-auto">
            <h1 id="seo-header">L17 Cell communication via enzyme-coupled receptors</h1>
            
            
              <div class="markdown-body">
                
                <h2 id="Enzyme-coupled-cell-surface-receptors"><a href="#Enzyme-coupled-cell-surface-receptors" class="headerlink" title="Enzyme-coupled cell surface receptors"></a>Enzyme-coupled cell surface receptors</h2><blockquote>
<p>Like GPCRs, <strong>enzyme-coupled receptors</strong> are transmembrane proteins with their ligand-binding domain on the <u>outer surface</u> of the plasma membrane</p>
<p>Instead of  having a <strong>cytosolic domain</strong> that associates with a trimeric G protein, however, their cytosolic domain either has <strong>intrinsic enzyme activity</strong> or <strong>associates directly with an enzyme</strong>. Whereas a GPCR has <u>seven</u> transmembrane segments, each subunit of an enzyme-coupled receptor typically has only <u>one</u>.</p>
</blockquote>
<p>All these receptors are <strong>single</strong> transmembrane receptors, which are either an enzyme themselves or are directly associated with an enzyme.</p>
<p>Enzyme-coupled receptors are divided as follows: </p>
<ol>
<li>Receptor tyrosine kinases (RTKs)</li>
<li>Receptors without enzyme activity but are coupled to cytoplasmic tyrosine kinase</li>
<li>Receptor Ser&#x2F;Thr kinases</li>
<li>Cytokine receptors that activates JAK-STAT signaling: cytokines, growth hormones etc..</li>
<li>Cytokine receptors that activates NF$\kappa$B signaling:</li>
<li>Histidine-kinase-associated receptors</li>
</ol>
<h1 id="一、Receptor-tyrosine-kinases-RTKs-and-downstream-signaling"><a href="#一、Receptor-tyrosine-kinases-RTKs-and-downstream-signaling" class="headerlink" title="一、Receptor tyrosine kinases (RTKs) and downstream signaling"></a>一、Receptor tyrosine kinases (RTKs) and downstream signaling</h1><blockquote>
<p>Many extracellular signal proteins act through <strong>receptor tyrosine kinases (RTKs)</strong>.  These include many <strong>secreted</strong> and <strong>cell-surface-bound proteins</strong> that control cell  behavior in developing and adult animals.</p>
</blockquote>
<h2 id="Signaling-via-RTKs"><a href="#Signaling-via-RTKs" class="headerlink" title="Signaling via RTKs"></a>Signaling via RTKs</h2><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708195645624.png" srcset="/img/loading.gif" lazyload alt="image-20200708195645624"></p>
<table>
<thead>
<tr>
<th>Signal protein family</th>
<th>Receptor family</th>
<th>Some representative responses</th>
</tr>
</thead>
<tbody><tr>
<td>Epidermal growth factor (EGF)</td>
<td>EGF receptors</td>
<td>Stimulates cell survival, growth, proliferation, or differentiation  of various cell types; acts as inductive signal in development</td>
</tr>
<tr>
<td>Insulin</td>
<td>Insulin receptor</td>
<td>Stimulates carbohydrate utilization and protein synthesis</td>
</tr>
<tr>
<td>Insulin-like growth factor (IGF1)</td>
<td>IGF receptor-1</td>
<td>Stimulates cell growth and survival in many cell types</td>
</tr>
<tr>
<td>Nerve growth factor (NGF)</td>
<td>Trk receptors</td>
<td>Stimulates survival and growth of some neurons</td>
</tr>
<tr>
<td>Platelet-derived growth factor (PDGF)</td>
<td>PDGF receptors</td>
<td>Stimulates survival, growth, proliferation, and migration of  various cell types</td>
</tr>
<tr>
<td>Macrophage-colony-stimulating factor (MCSF)</td>
<td>MCSF receptor</td>
<td>Stimulates monocyte&#x2F;macrophage proliferation and  differentiation</td>
</tr>
<tr>
<td>Fibroblast growth factor (FGF)</td>
<td>FGF receptors</td>
<td>Stimulates proliferation of various cell types; inhibits  differentiation of some precursor cells; acts as inductive  signal in development</td>
</tr>
<tr>
<td>Vascular endothelial growth factor (VEGF)</td>
<td>VEGF receptors</td>
<td>Stimulates angiogenesis</td>
</tr>
<tr>
<td>Ephrin</td>
<td>Eph receptors</td>
<td>Stimulates angiogenesis; guides cell and axon migration</td>
</tr>
</tbody></table>
<blockquote>
<p>For most RTKs, ligand binding causes the receptors to <strong>dimerize</strong>, bringing the two cytoplasmic kinase domains together and thereby <u>promoting their activation</u></p>
<p>Receptor for <strong>epidermal growth factor (EGF)</strong>, the kinase is <u>not activated by phosphorylation</u> but by <u>conformational changes</u> brought about by interactions between the two kinase domains outside their active sites</p>
</blockquote>
<h2 id="Domain-structure-of-some-RTKs"><a href="#Domain-structure-of-some-RTKs" class="headerlink" title="Domain structure of some RTKs"></a>Domain structure of some RTKs</h2><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708195657596.png" srcset="/img/loading.gif" lazyload alt="image-20200708195657596"></p>
<p>RTKs are <strong>single</strong> transmembrane receptors.</p>
<p>RTKs are kinases themselves and possess three domains:</p>
<ul>
<li>Extracellular region: interact with ligand</li>
<li>One single transmembrane domain</li>
<li>Intracellular region: tyrosine kinase activity</li>
</ul>
<p>Human genome encodes ~ <strong>60</strong> RTK genes</p>
<p>Upon ligand binding, RTK usually <strong>dimerize</strong> and <strong>trans-autophosphorylate</strong> themselves</p>
<h2 id="Activation-of-RTKs"><a href="#Activation-of-RTKs" class="headerlink" title="Activation of RTKs"></a>Activation of RTKs</h2><blockquote>
<p>Phosphorylated Tyrosines on RTKs Serve as Docking Sites for Intracellular Signaling Proteins</p>
<p>Once the <strong>kinase domains</strong> of an RTK dimer are <u>activated</u>, they phosphorylate multiple <u>additional sites in the cytosolic parts</u> of the receptors, typically in disordered regions outside the kinase domain</p>
<ul>
<li>This phosphorylation creates <strong>high-affinity docking sites</strong> for intracellular signaling proteins</li>
</ul>
<p>Once bound to the activated RTK, a signaling protein may become <u>phosphorylated on tyrosines</u> and thereby activated. In many cases, however, the <u>binding  alone</u> may be sufficient to activate the docked signaling protein, by either <u>inducing a conformational change</u> in the protein or simply <u>bringing it near</u> the protein  that is next in the signaling pathway.</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708195809484.png" srcset="/img/loading.gif" lazyload alt="image-20200708195809484"></p>
<p>Phosphorylation on RTK has <strong>dual roles</strong>: </p>
<ul>
<li>Activates RTK kinase activity </li>
<li>Introduce phospho-Tyr that can <u>recruit</u> other protein factors to relay signals through <font color="EE0000">SH2</font>, <font color="EE0000">PTB</font> (phospho-Tyr binding) domain, or <font color="EE0000">SH3</font> (Pro-rich Binding) domains, etc.</li>
</ul>
<blockquote>
<p>Some RTKs use <strong>additional docking proteins</strong> to enlarge the signaling complex at activated receptors.</p>
<ul>
<li>Insulin and <font color="EE0000"><strong>IGF1</strong></font> receptor signaling, for example, depend on a specialized docking protein called <strong>insulin receptor substrate 1 (<font color="EE0000">IRS1</font>)</strong></li>
<li><strong>IRS1</strong> associates with <strong>phosphorylated tyrosines</strong> on the activated receptor and is then <u>phosphorylated at multiple sites</u>, thereby creating many more docking sites than could be accommodated on the receptor alone</li>
</ul>
</blockquote>
<h3 id="1-Mutant-RTKs-can-act-in-a-dominant-negative-manner"><a href="#1-Mutant-RTKs-can-act-in-a-dominant-negative-manner" class="headerlink" title="1. Mutant RTKs can act in a dominant negative manner"></a>1. Mutant RTKs can act in a dominant negative manner</h3><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708200044974.png" srcset="/img/loading.gif" lazyload alt="image-20200708200044974"></p>
<p>Dominant negative (DN) mutant: </p>
<ul>
<li>a mutant protein that exhibits an inhibitory effect on the wild type protein</li>
<li>mutant 使得功能丧失</li>
</ul>
<h3 id="2-Signaling-pathways-downstream-of-RTKs-the-Ras-signaling-pathway"><a href="#2-Signaling-pathways-downstream-of-RTKs-the-Ras-signaling-pathway" class="headerlink" title="2. Signaling pathways downstream of RTKs: the Ras signaling pathway"></a>2. Signaling pathways downstream of RTKs: the Ras signaling pathway</h3><blockquote>
<p>The GTPase <strong>Ras</strong> Mediates Signaling by Most RTKs</p>
<p>The <strong>Ras</strong> superfamily consists of various families of monomeric GTPases, but only the <font color="EE0000">Ras</font> and <font color="EE0000">Rho</font> families relay signals <u>from cell-surface receptors</u></p>
<ul>
<li><strong>Ras</strong> is often required, for example, when <u>RTKs signal to the nucleus</u> to stimulate <strong>cell proliferation</strong> or <strong>differentiation</strong>, both of which require changes in <strong>gene expression</strong>.</li>
</ul>
</blockquote>
<p><strong>Ras</strong>: name derived from Rat sarcoma virus, which infection induces sarcoma </p>
<ul>
<li>Small monomeric GTPase, ~20 KDa, weak GTP hydrolysis activity.</li>
<li><u>Highly mutated in human cancers</u>, e.g. ~ 30%, in pancreatic cancer, mutation rate <u>95%（for dominant-dominant mutation）</u>.</li>
<li>Pivotal roles in cell proliferation, survival, motility, etc.</li>
<li>Tethered on lipid membrane.</li>
</ul>
<h4 id="The-Ras-signaling-pathway-was-discovered-by-analysis-of-Drosophila-eye-development"><a href="#The-Ras-signaling-pathway-was-discovered-by-analysis-of-Drosophila-eye-development" class="headerlink" title="The Ras signaling pathway was discovered by analysis of Drosophila eye  development"></a>The Ras signaling pathway was discovered by analysis of Drosophila eye  development</h4><p>Observations:</p>
<ul>
<li>Flies lacking a RTK called sevenless (<font color="EE0000">Sev</font>) fail to detect UV light by R7 photoreceptor → Sev is required for the formation of a photoreceptor cell called <font color="FF4500">R7</font>. </li>
<li>Flies with deficiency of bride of sevenless (<font color="FF4500">Boss</font>) show the same defects</li>
</ul>
<p>Therefore identify the proteins:</p>
<ul>
<li>Sev- RTK receptor</li>
<li>Boss- Sev’s ligand</li>
</ul>
<p>Therefore: Genetic screens of Sev&#x2F;Boss partial mutant strains revealed that mutation of Ras leads to loss of R7; while hyperactive Ras rescues deficiency of both Sev&#x2F;Boss</p>
<ul>
<li>Ras locates downstream of Sev</li>
<li>Further genetic screen identified Son of sevenless (<font color="FF4500">Sos</font>) and <font color="EE0000">Drk</font> (<font color="EE0000">Grb2</font> in human)</li>
</ul>
<h4 id="Activation-of-Ras-downstream-of-Sev-RTK-requires-an-adaptor-protein-Grb2-and-the-GEF-Sos"><a href="#Activation-of-Ras-downstream-of-Sev-RTK-requires-an-adaptor-protein-Grb2-and-the-GEF-Sos" class="headerlink" title="Activation of Ras downstream of Sev RTK requires an adaptor protein Grb2 and the GEF Sos"></a>Activation of Ras downstream of Sev RTK requires an adaptor protein Grb2 <u>and</u> the GEF Sos</h4><blockquote>
<p>The GEF that mediates Ras activation by RTKs was discovered by genetic studies of eye development in Drosophila, where an RTK called <strong>Sevenless (Sev)</strong>  is required for the formation of a photoreceptor cell called R7.</p>
<p>Genetic screens for components of this signaling pathway led to the discovery of a Ras-GEF called <strong><font color="EE0000">Son-of-sevenless (Sos)</font></strong>. </p>
<p>Further genetic screens uncovered another protein, now called <strong><font color="EE0000">Grb2</font></strong>, which is an <strong>adaptor protein</strong> that links the <strong>Sev receptor</strong> to the <strong>Sos protein;</strong> </p>
<ul>
<li>the <font color="EE0000">SH2</font> domain of the <strong>Grb2</strong> adaptor binds to the activated receptor</li>
<li>while its two <font color="EE0000">SH3</font> domains bind to <strong>Sos</strong>.</li>
</ul>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708200804450.png" srcset="/img/loading.gif" lazyload alt="image-20200708200804450"></p>
<p>Activation of <font color="EE0000">Ras</font> is mediated by the Ras GEF Sos (Son of sevenless) (Ras-GDP → Ras-GTP)</p>
<p>In human, <font color="EE0000">Grb2</font> is homolog for D<strong>r</strong>k: adaptor protein</p>
<p>The adaptor protein Grb2 (growth factor receptor binding protein 2) &#x2F;Drk that binds to Sos</p>
<h4 id="RTK-activates-Ras"><a href="#RTK-activates-Ras" class="headerlink" title="RTK activates Ras"></a>RTK activates Ras</h4><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708200909222.png" srcset="/img/loading.gif" lazyload alt="image-20200708200909222"></p>
<h4 id="Ras-activation-is-transient"><a href="#Ras-activation-is-transient" class="headerlink" title="Ras activation is transient"></a>Ras activation is transient</h4><blockquote>
<p>Ras Activates a MAP Kinase Signaling Module</p>
<p>Both the tyrosine phosphorylations and the activation of Ras triggered by activated RTKs are usually <u>short-lived</u> (Figure 15–48). <strong>Tyrosine-specific protein phosphatases</strong> quickly reverse the phosphorylations</p>
<p>Protein Tyrosine Phosphatases Reverse Tyrosine Phosphorylations</p>
<ul>
<li>In all signaling pathways that use tyrosine phosphorylation, the tyrosine phosphorylations are <strong>reversed</strong> by <strong>protein tyrosine phosphatases</strong>.<ul>
<li>Whereas only a few types of <strong>serine&#x2F;threonine protein phosphatase catalytic subunits</strong> are responsible for removing phosphate groups from phosphorylated serines and threonines on proteins, there are about 100 protein tyrosine phosphatases encoded in the human genome, including some <strong>dual-specificity phosphatases</strong> that also dephosphorylate serines and threonines.</li>
</ul>
</li>
<li>Like tyrosine kinases, the tyrosine phosphatases occur in <u>both cytoplasmic and transmembrane forms.</u></li>
<li>Unlike serine&#x2F;threonine protein phosphatases, which generally have <strong>broad specificity</strong>, most tyrosine phosphatases display <strong>exquisite  specificity</strong> for their substrates, removing phosphate groups from only <strong>selected phosphotyrosines</strong> on a subset of proteins.<ul>
<li>These <strong>receptor serine&#x2F;threonine kinases</strong> activate an even more direct signaling pathway to the nucleus than does the <font color="EE0000">JAK– STAT</font> pathway. They directly phosphorylate <strong>latent transcription regulators</strong> called  <strong><font color="EE0000">Smads</font></strong>, which then translocate into the nucleus to control gene transcription.</li>
</ul>
</li>
</ul>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708200921525.png" srcset="/img/loading.gif" lazyload alt="image-20200708200921525"></p>
<h4 id="Ras-activates-a-mitogen-activated-protein-kinase-MAPK-signaling-module-Ras–Raf–Mek–Erk"><a href="#Ras-activates-a-mitogen-activated-protein-kinase-MAPK-signaling-module-Ras–Raf–Mek–Erk" class="headerlink" title="Ras activates a mitogen-activated protein kinase (MAPK) signaling module (Ras–Raf–Mek–Erk)"></a>Ras activates a mitogen-activated protein kinase (MAPK) signaling module (Ras–Raf–Mek–Erk)</h4><blockquote>
<p>Ras to inactivate itself by hydrolyzing its bound GTP to GDP. <u>To stimulate cells to <strong>proliferate</strong> or <strong>differentiate</strong></u>, these <strong>short-lived signaling events</strong> <u>must be converted  into <strong>longer-lasting ones</strong></u> that can sustain the signal and relay it downstream to the nucleus to alter the pattern of gene expression. </p>
<ul>
<li>One of the key mechanisms used for this purpose is a system of proteins called the <strong>mitogen-activated protein kinase module (MAP kinase module)</strong> (Figure 15–49).</li>
</ul>
<p>The three components of this system form a functional signaling module that has been remarkably well conserved during evolution and is <u>used, with variations, in many different signaling contexts.</u></p>
<p>The three components are all protein kinases. </p>
<ol>
<li>The final kinase in the series is called simply <strong><font color="EE0000">MAP kinase (MAPK)</font>.</strong> </li>
<li>The next one <u>upstream</u> from this is <strong><font color="EE0000">MAP kinase kinase (MAPKK)</font></strong>: it phosphorylates and thereby activates <strong>MAP kinase.</strong> </li>
<li>Next <u>above</u> that, receiving an activating signal directly from Ras, is <strong><font color="EE0000">MAP kinase kinase kinase (MAPKKK)</font></strong>: it phosphorylates and thereby activates <strong>MAPKK</strong>.</li>
</ol>
<p>In the mammalian  <strong>Ras–MAP-kinase signaling pathway</strong>, these three kinases are known by shorter names: </p>
<ul>
<li><strong><font color="EE0000">Raf</font></strong> (&#x3D; MAPKKK), <strong><font color="EE0000">Mek</font></strong> (&#x3D; MAPKK), and <strong><font color="EE0000">Erk</font></strong> (&#x3D;MAPK).</li>
</ul>
<p><strong>Erk</strong> also <strong>phosphorylates</strong> and <strong>inactivates Raf</strong>, providing another <strong>negative feedback</strong> loop that helps <strong>shut off the MAP kinase module.</strong></p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708200946686.png" srcset="/img/loading.gif" lazyload alt="image-20200708200946686" style="zoom: 33%;" />

<p>MAP kinase pathway controls both, </p>
<ul>
<li><p>protein activity</p>
</li>
<li><p>gene transcription through phosphorylation of transcription regulatory complex, which activates transcription of</p>
</li>
<li><p>immediate early genes within minutes</p>
</li>
<li><p>other genes (e.g. regulate <strong>G1 cyclin</strong>) </p>
</li>
<li><p>stimulate cell proliferation, differentiation, etc..</p>
</li>
<li><p>Extracellular signals usually activate MAP kinases only transiently (min), while sometimes can also be many hours (e.g. NGF mediated neuron differentiation)</p>
</li>
</ul>
<h5 id="1-Specificity-of-MAPK-signaling-provided-by-scaffold-proteins"><a href="#1-Specificity-of-MAPK-signaling-provided-by-scaffold-proteins" class="headerlink" title="(1) Specificity of MAPK signaling provided by scaffold proteins"></a>(1) Specificity of MAPK signaling provided by scaffold proteins</h5><blockquote>
<p><strong>Scaffold Proteins</strong> Help Prevent Cross-talk Between Parallel MAP Kinase Modules</p>
<p>Although the scaffold strategy provides precision and avoids cross-talk, it reduces the opportunities for <strong>amplification</strong> and <strong>spreading</strong> of the signal to different <u>parts of the cell</u>, which require at least some of the components to be diffusible</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708201123564.png" srcset="/img/loading.gif" lazyload alt="image-20200708201123564"></p>
<p>12 MAPKs, 7 MAPKKs, and 7 MAPKKKs</p>
<ul>
<li>At least five parallel MAP kinase modules</li>
</ul>
<p>Stress response mediated by <font color="0000EE">JNK</font> and <font color="0000EE">p38</font> are activated by different kinds of cell stresses, such as <strong>UV irradiation</strong>, <strong>heat shock</strong>, and <strong>osmotic stress</strong>, as well as by <strong>inflammatory cytokines</strong> </p>
<ul>
<li>Scaffold proteins provide precision and prevent cross-talk between parallel&#x2F;neighboring MAP kinase modules</li>
</ul>
<h4 id="Ras-superfamily-of-monomeric-GTPases"><a href="#Ras-superfamily-of-monomeric-GTPases" class="headerlink" title="Ras superfamily of monomeric GTPases"></a>Ras superfamily of monomeric GTPases</h4><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708201701621.png" srcset="/img/loading.gif" lazyload alt="image-20200708201701621"></p>
<h3 id="3-Signaling-pathways-downstream-of-RTKs-the-Rho-GTPase-signaling-pathway"><a href="#3-Signaling-pathways-downstream-of-RTKs-the-Rho-GTPase-signaling-pathway" class="headerlink" title="3. Signaling pathways downstream of RTKs: the Rho GTPase signaling pathway"></a>3. Signaling pathways downstream of RTKs: the Rho GTPase signaling pathway</h3><blockquote>
<p><font color="EE0000">Rho</font> Family GTPases Functionally Couple Cell-Surface Receptors to the Cytoskeleton</p>
<p>Besides the Ras proteins, the <strong>other class of Ras superfamily GTPases</strong> that relays signals from <strong>cell-surface receptors</strong> is the large <strong>Rho family</strong></p>
<p>Rho family monomeric GTPases regulate both the <u>actin and microtubule cytoskeletons</u>, </p>
<ul>
<li>controlling <strong>cell shape</strong>, <strong>polarity</strong>, <strong>motility</strong>, and <strong>adhesion</strong> (discussed in Chapter 16); </li>
<li>they also regulate <strong>cell-cycle progression</strong>, <strong>gene transcription</strong>, and <strong>membrane transport.</strong> </li>
<li>They play a key part in the guidance of <strong>cell migration</strong> and <strong>nerve axon outgrowth</strong>, mediating cytoskeletal responses to the activation of a special class of guidance receptors</li>
</ul>
</blockquote>
<blockquote>
<p>The three best-characterized family members are <strong><font color="EE0000">Rho</font></strong> itself, <strong><font color="EE0000">Rac</font></strong>, and <strong><font color="EE0000">Cdc42</font></strong>,  each of which affects multiple downstream target proteins. In the same way as for Ras, GEFs activate and GAPs inactivate the Rho family GTPases; there are more than 80 Rho-GEFs and more than 70 Rho-GAPs in humans</p>
<p>Unlike Ras, which is <u>membrane-associated even when inactive</u> (with GDP bound), inactive Rho family GTPases are often bound to <strong><font color="EE0000">guanine nucleotide dissociation inhibitors (GDIs)</font></strong> in the <strong>cytosol</strong>, which <u>prevent the GTPases</u> from interacting with their <strong>Rho-GEFs</strong> at the <strong>plasma membrane</strong></p>
<ul>
<li>Signaling by extracellular signaling proteins of the <strong><font color="EE0000">ephrin</font></strong> family provides an example of how RTKs can activate a Rho GTPase.</li>
</ul>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708201723934.png" srcset="/img/loading.gif" lazyload alt="image-20200708201723934" style="zoom:33%;" />



<p>Ephrin (extracellular signaling proteins)  binds and activates the <font color="EE0000">Eph</font> family of  RTKs (<font color="EE0000">EphA4</font>) on the surface of motor neurons</p>
<p>Helps guide the migrating tip of the axon  (called a growth cone) to its muscle target.</p>
<h4 id="Signaling-pathways-downstream-of-RTKs-the-PI3-kinase-PI3K-signaling-pathway"><a href="#Signaling-pathways-downstream-of-RTKs-the-PI3-kinase-PI3K-signaling-pathway" class="headerlink" title="Signaling pathways downstream of RTKs: the PI3-kinase (PI3K) signaling  pathway"></a>Signaling pathways downstream of RTKs: the PI<del>3</del>-kinase (PI<del>3</del>K) signaling  pathway</h4><blockquote>
<p>PI 3-Kinase Produces Lipid Docking Sites in the Plasma Membrane</p>
<p>One of the proteins that binds to the intracellular tail of RTK molecules is the plasma-membrane-bound enzyme <strong>phosphoinositide 3-kinase  (PI 3-kinase)</strong>.</p>
<p>This kinase principally phosphorylates <strong>inositol phospholipids</strong> rather than proteins, and both RTKs and GPCRs can activate it. It plays a central part in promoting cell <strong>survival</strong> and <strong>growth</strong>.</p>
</blockquote>
<blockquote>
<p>Notice the difference between this use of phosphoinositides and their use described earlier, in which <strong>PI(4,5)P2</strong> is cleaved by <strong><font color="0000EE">PLCβ</font></strong> (in the case of GPCRs) or <strong><font color="EE0000">PLCγ</font></strong> (in the case of RTKs) to generate soluble <font color="EE0000">IP3</font> and <strong>membrane-bound diacylglycerol</strong> (see Figures 15–28 and 15–29)</p>
<ul>
<li>By contrast, <strong>PI(3,4,5)P3</strong> is not cleaved by either PLC</li>
</ul>
<p>There are various types of PI 3-kinases. Those activated by RTKs and GPCRs  belong to <strong>class I</strong></p>
<ul>
<li>RTKs activate <strong>class <font color="0000EE">Ⅰa</font> PI 3-kinases</strong>, in which the regulatory subunit is an <strong>adaptor protein</strong> that binds to <strong>two phosphotyrosines</strong> on <strong>activated RTKs</strong> through its two <strong>SH2 domains</strong> (see Figure 15–46A). </li>
<li>GPCRs activate <strong>class <font color="0000EE">Ⅰb</font> PI 3-kinases</strong>, which have a <strong>regulatory subunit</strong> that binds to the <strong>βγ complex</strong> of an activated trimeric G protein (<font color="0000EE">G<del>q</del> </font>) when GPCRs are activated by their extracellular ligand.</li>
</ul>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708201811180.png" srcset="/img/loading.gif" lazyload alt="image-20200708201811180" style="zoom: 50%;" />

<p>There are various types of PI 3-kinases.  </p>
<p>Those activated by RTKs and GPCRs belong to class I.  </p>
<p>These are heterodimers composed of a common catalytic subunit and different regulatory subunits.  </p>
<ul>
<li><p>RTKs activate class Ia PI3K, in which the regulatory subunit is an adaptor protein that binds to two phosphotyrosines on activated  RTKs through its two SH2 domains. </p>
</li>
<li><p>GPCRs activate class Ib PI3K, which have a  regulatory subunit that binds to the βγ complex  of an activated G<del>q</del>, when GPCRs are activated.</p>
</li>
</ul>
<p>The direct binding of activated Ras can also activate the common class I catalytic subunit.</p>
<h4 id="The-PI3K-Akt-signaling-pathway"><a href="#The-PI3K-Akt-signaling-pathway" class="headerlink" title="The PI3K&#x2F;Akt signaling pathway"></a>The PI3K&#x2F;Akt signaling pathway</h4><blockquote>
<p>Intracellular signaling proteins bind to <strong>PI(3,4,5)P3</strong> produced by activated PI 3-kinase via a specific interaction domain, such as a <strong><font color="EE0000">pleckstrin homology (PH) domains</font></strong></p>
<p>One especially important PH-domain-containing protein is the serine&#x2F;threonine protein kinase <strong><font color="EE0000">Akt</font></strong>. </p>
<ul>
<li>The <strong>PI-3-kinase–Akt signaling pathway</strong> is the major pathway activated by the hormone <strong>insulin</strong>. It also plays a key part in promoting the <strong>survival and growth</strong> of many cell types</li>
</ul>
</blockquote>
<blockquote>
<p>The PI-3-Kinase–Akt Signaling Pathway Stimulates Animal Cells to Survive and Grow</p>
<p>Members of the <strong><font color="EE0000">insulin-like growth factor (IGF)</font></strong> family of signal proteins, for example, stimulate many types of animal cells to survive and grow. They bind to specific RTKs (see Figure 15–43), which activate PI 3-kinase to produce PI(3,4,5)P3</p>
<p>The PI(3,4,5)P3 recruits two protein kinases to the plasma membrane via their <strong>PH domains</strong>—<strong>Akt</strong> (also called <strong><font color="EE0000">protein kinase B</font></strong>, or <strong><font color="EE0000">PKB</font></strong>) and <strong><font color="EE0000">phosphoinositide-dependent protein kinase 1 (PDK1)</font></strong>, and this leads to the activation of <strong>Akt</strong></p>
<ul>
<li><strong>Akt</strong> phosphorylates various target proteins at the plasma membrane, as well as in the cytosol and nucleus</li>
</ul>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708201905221.png" srcset="/img/loading.gif" lazyload alt="image-20200708201905221"></p>
<p>Intracellular signaling proteins bind to PI(3,4,5)P3 produced by activated PI3K via a specific interaction domain, such as a PH domain (pleckstrin homology first identified in the platelet protein pleckstrin) (PH domain is very common, and is present in Sos as well) </p>
<p>PI3K-Akt signaling is to  </p>
<ul>
<li>promote signals for cell growth and survival </li>
<li>produces lipid binding&#x2F;docking sites for proteins at the PM</li>
</ul>
<h5 id="1-Coordination-and-integration-of-growth-factor-and-nutrient-sensing-pathways"><a href="#1-Coordination-and-integration-of-growth-factor-and-nutrient-sensing-pathways" class="headerlink" title="(1) Coordination and integration of growth factor and nutrient sensing pathways"></a>(1) Coordination and integration of growth factor and nutrient sensing pathways</h5><blockquote>
<p>The control of cell growth by the PI-3-kinase–Akt pathway depends in part on a large protein kinase called <strong>TOR</strong></p>
<ul>
<li><p><u>in mammalian cells, it is called <strong><font color="EE0000">mTOR</font></strong>, which exists in cells in two functionally distinct multiprotein complexes.</u> <strong><font color="EE0000">mTOR complex 1</font></strong> contains the protein <strong><font color="EE0000">raptor</font></strong>; this complex is <u>sensitive</u> to <strong>rapamycin</strong>, and it stimulates <strong>cell growth</strong>—both by promoting <u>ribosome production and protein synthesis</u>  and by <u>inhibiting protein degradation</u>. Complex 1 also promotes both cell growth and cell survival by <u>stimulating nutrient uptake and metabolism.</u> </p>
</li>
<li><p><strong><font color="EE0000">mTOR complex  2</font></strong> contains the protein <strong><font color="EE0000">rictor</font></strong> and is <u>insensitive to rapamycin</u>; it helps to <strong>activate Akt</strong>  (see Figure 15–53), and it regulates the <strong>actin cytoskeleton</strong> via <strong>Rho</strong> family GTPases.</p>
</li>
<li><p>Akt activates mTOR in complex 1 indirectly by phosphorylating, and thereby inhibiting a GAP called <font color="EE0000">Tsc2</font>. </p>
<ul>
<li>Tsc2 acts on a monomeric Ras-related GTPase called <strong><font color="EE0000">Rheb</font></strong> (see  Table 15–5).</li>
</ul>
</li>
</ul>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708201944542.png" srcset="/img/loading.gif" lazyload alt="image-20200708201944542" style="zoom: 25%;" />

<h5 id="2-Activation-of-mTORC1-by-the-PI3-kinase-PI3K-Akt"><a href="#2-Activation-of-mTORC1-by-the-PI3-kinase-PI3K-Akt" class="headerlink" title="(2) Activation of mTORC1 by the PI3-kinase (PI3K)-Akt"></a>(2) Activation of mTORC1 by the PI3-kinase (PI3K)-Akt</h5><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708202018066.png" srcset="/img/loading.gif" lazyload alt="image-20200708202018066"></p>
<p>mTOR1: Raptor  (regulatory subunit,  rapamycin <strong>sensitive</strong>) </p>
<p>mTOR2: rictor (regulatory subunit,  rapamycin <strong>insensitive</strong>)</p>
<h5 id="3-The-PI3K-and-the-phosphatase-and-tensin-homolog-PTEN-control-PIP3-levels-at-the-plasma-membrane"><a href="#3-The-PI3K-and-the-phosphatase-and-tensin-homolog-PTEN-control-PIP3-levels-at-the-plasma-membrane" class="headerlink" title="(3) The PI3K and the phosphatase and tensin homolog (PTEN) control PIP3 levels at the plasma membrane"></a>(3) The PI3K and the phosphatase and tensin homolog (PTEN) control PIP3 levels at the plasma membrane</h5><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708202416557.png" srcset="/img/loading.gif" lazyload alt="image-20200708202416557"></p>
<p>PI3K hyperactivation and PTEN loss-of-function occurs frequently in human cancers.</p>
<h3 id="4-RTKs-and-GPCRs-activate-overlapping-signaling-pathways"><a href="#4-RTKs-and-GPCRs-activate-overlapping-signaling-pathways" class="headerlink" title="4. RTKs and GPCRs activate overlapping signaling pathways"></a>4. RTKs and GPCRs activate overlapping signaling pathways</h3><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708202519254.png" srcset="/img/loading.gif" lazyload alt="image-20200708202519254"></p>
<p>Some pathway components like Ca2+ or phospholipase C could participate in both GPCR  and RTK pathways </p>
<p>Pathway may also integrate signal via double phosphorylation or activating complementary proteins</p>
<h3 id="5-Summary-major-downstream-signaling-of-receptor-tyrosine-kinases-RTKs"><a href="#5-Summary-major-downstream-signaling-of-receptor-tyrosine-kinases-RTKs" class="headerlink" title="5. Summary: major downstream signaling of receptor tyrosine kinases (RTKs)"></a>5. Summary: major downstream signaling of receptor tyrosine kinases (RTKs)</h3><p>Ras signaling pathway</p>
<p>Rho signaling pathway</p>
<p>PI3K&#x2F;Akt signaling pathway</p>
<h2 id="Activation-of-EGFR-epithelia-growth-factor-receptor"><a href="#Activation-of-EGFR-epithelia-growth-factor-receptor" class="headerlink" title="Activation of EGFR (epithelia growth factor receptor)"></a>Activation of EGFR (epithelia growth factor receptor)</h2><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708195855971.png" srcset="/img/loading.gif" lazyload alt="image-20200708195855971"></p>
<p>EGF binding results in a conformational change that promotes dimerization of the external domains. </p>
<p>The receptor kinase domain is not activated by trans-autophosphorylation. </p>
<p>Instead, dimerization induce an activating conformational change in one monomer (the receiver). </p>
<p>The active receiver domain then phosphorylates multiple tyrosines in the C-terminal tails of both receptors.</p>
<h2 id="The-PDGF-platelet-derived-growth-factor-receptor-and-signal-relay"><a href="#The-PDGF-platelet-derived-growth-factor-receptor-and-signal-relay" class="headerlink" title="The PDGF (platelet-derived growth factor) receptor and signal relay"></a>The PDGF (platelet-derived growth factor) receptor and signal relay</h2><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708200433727.png" srcset="/img/loading.gif" lazyload alt="image-20200708200433727"></p>
<h3 id="1-SH2-domain-structure"><a href="#1-SH2-domain-structure" class="headerlink" title="1. SH2 domain structure"></a>1. SH2 domain structure</h3><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708200453124.png" srcset="/img/loading.gif" lazyload alt="image-20200708200453124"></p>
<h2 id="Conserved-Interaction-Domains"><a href="#Conserved-Interaction-Domains" class="headerlink" title="Conserved Interaction Domains"></a>Conserved Interaction Domains</h2><blockquote>
<p>Proteins with SH2 Domains Bind to Phosphorylated Tyrosines</p>
<ul>
<li>A whole menagerie of intracellular signaling proteins can bind to the phosphotyrosines on activated RTKs (or on docking proteins such as <font color="EE0000">IRS1</font>).</li>
<li>They help to relay the signal onward（向前）, mainly through chains of protein–protein interactions mediated by modular interaction domains</li>
</ul>
<p>Some of the docked proteins are enzymes, such as <strong><font color="EE0000">phospholipase C-γ (PLCγ)</font></strong>, which functions in the same way as <strong>phospholipase C-β</strong>—activating the inositol phospholipid signaling pathway discussed earlier in connection with GPCRs</p>
<ul>
<li>Through this pathway, RTKs can increase cytosolic Ca2+ levels and activate PKC. Another enzyme that docks on these receptors is the cytoplasmic tyrosine kinase Src, which phosphorylates other signaling proteins on tyrosines.</li>
</ul>
<p>Yet another is <strong>phosphoinositide 3-kinase (PI 3-kinase)</strong>, which phosphorylates lipids rather than proteins; as we discuss later, the phosphorylated lipids then serve  as docking sites to <u>attract</u> various signaling proteins to the plasma membrane.</p>
<p>The intracellular signaling proteins that bind to phosphotyrosines have varied structures and functions. However, they usually share <u>highly conserved phosphotyrosine-binding domains</u>. </p>
<p>These can be either </p>
<ol>
<li><strong>SH2 domains</strong> (for Src homology region) </li>
<li>or, less commonly, <strong>PTB domains</strong> (for phosphotyrosine-binding)</li>
<li><strong>SH3 domain</strong>, which binds to proline-rich motifs in intracellular proteins</li>
</ol>
</blockquote>
<p>Conserved interaction domains are important for protein binding which modulates intracellular signaling proteins (Lecture 15)</p>
<ol>
<li>Src homology 2 (SH2) domains (Bind phosphotyrosine (p-$\gamma$))</li>
<li>Phosphotyrosine-binding (PTB) domains (Bind phosphotyrosine (p-$\gamma$))</li>
<li>Src homology 3 (SH3): bind proline rich domains</li>
<li>Pleckstrin homology (PH): bind phosphoinositides</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708200217084.png" srcset="/img/loading.gif" lazyload alt="image-20200708200217084"></p>
<ul>
<li>autophosphorylation of activated receptor <u>recruits IRS1</u></li>
<li>Receptor phosphorylates IRS1, which <u>recruits Grb2</u><ol>
<li>Grb2 <u>recruits Sos</u><ul>
<li>Sos acts as a GEF to activate Ras</li>
</ul>
</li>
<li>Grb2 <u>recruits scaffold protein</u></li>
</ol>
</li>
</ul>
<blockquote>
<p><u>Not</u> all proteins that bind to activated RTKs via SH2 domains help to relay the signal onward. Some act to <u>decrease the signaling process</u>, providing <strong>negative feedback</strong></p>
<ul>
<li>One example is the <strong><font color="EE0000">c-Cbl protein</font></strong>, which can dock on some activated receptors and catalyze their ubiquitylation, covalently adding one or more ubiquitin molecules to specific sites on the receptor. This promotes the endocytosis and degradation of the receptors in lysosomes—an example of receptor down-regulation<ul>
<li>Mutations that inactivate <strong>c-Cbl-dependent RTK down-regulation</strong> cause prolonged RTK signaling and thereby promote the development of cancer.</li>
</ul>
</li>
<li>Endocytic proteins that contain <strong><font color="EE0000">ubiquitin-interaction motifs (UIMs)</font></strong> recognize the ubiquitylated RTKs and direct them into <strong>clathrin-coated vesicles</strong> and, ultimately, into lysosomes (discussed in Chapter  13).</li>
</ul>
<p>As is the case for GPCRs, <strong>ligand-induced endocytosis of RTKs</strong> does <u>not</u> always  <u>decrease</u> signaling. In some cases, RTKs are endocytosed with their bound signaling proteins and <u>continue to signal</u> from <strong>endosomes</strong> or other intracellular compartments</p>
<p>Some signaling proteins are composed almost entirely of SH2 and SH3 domains and function as <strong>adaptors</strong> to couple tyrosine-phosphorylated proteins to other proteins that <u>do not have their own SH2 domains</u></p>
</blockquote>
<h1 id="二、Non–receptor-tyrosine-kinases"><a href="#二、Non–receptor-tyrosine-kinases" class="headerlink" title="二、Non–receptor tyrosine kinases"></a>二、Non–receptor tyrosine kinases</h1><blockquote>
<p>Some Enzyme-Coupled Receptors Associate with Cytoplasmic Tyrosine Kinases</p>
<p>Many cell-surface receptors depend on <strong>tyrosine phosphorylation</strong> for their activity and yet <u>lack a tyrosine kinase domain</u>. These receptors act through <strong>cytoplasmic tyrosine kinases</strong>, which are <u>associated with the receptors</u> and <u>phosphorylate various target proteins,</u> often <u>including the receptors themselves</u>, when the receptors bind their ligand.</p>
<p>These <strong>tyrosine-kinase-associated receptors</strong> thus function <u>in much the same way as <strong>RTKs</strong></u>, except that their kinase domain is encoded by a separate gene and is noncovalently associated with the receptor polypeptide  chain.</p>
</blockquote>
<h2 id="Major-types-of-cytosolic-non-receptor-tyrosine-kinases"><a href="#Major-types-of-cytosolic-non-receptor-tyrosine-kinases" class="headerlink" title="Major types of cytosolic non-receptor tyrosine kinases"></a>Major types of cytosolic non-receptor tyrosine kinases</h2><blockquote>
<p>Some of these receptors depend on members of the largest family of mammalian cytoplasmic tyrosine kinases, the <strong><font color="EE0000">Src</font></strong> family</p>
<p>which includes <strong><font color="EE0000">Src, Yes, Fgr, Fyn, Lck, Lyn, Hck, and Blk</font>.</strong> These protein kinases all contain <strong>SH2</strong> and <strong>SH3</strong> domains and are located on the cytoplasmic side of the plasma membranes</p>
<ul>
<li>In each case, the kinase is <u>activated</u> when an <strong>extracellular ligand</strong> binds to the appropriate receptor protein.  Src itself, as well as several other family members, can also bind to activated RTKs;</li>
<li>There are even <strong>some G proteins</strong> (Gs and Gi) that can activate Src, which is one way that the <strong>activation of GPCRs</strong> can lead to <strong>tyrosine phosphorylation</strong> of  intracellular signaling proteins and effector proteins.</li>
</ul>
</blockquote>
<p><strong>Src</strong> (sarcoma virus) tyrosine kinase protein family: </p>
<ul>
<li>cytosolic tyrosine kinases</li>
<li>proto-oncogenes</li>
<li>controls cytoskeleton assembly, growth and proliferation.</li>
</ul>
<p>Focal adhesion kinase (<strong><font color="EE0000">FAK</font></strong>) (“Integrin signaling” Lecture 25) </p>
<ul>
<li>mediates integrin signaling to the cytoskeleton during cell adhesion.</li>
</ul>
<p>Janus kinases (<strong><font color="EE0000">JAKs</font></strong>):</p>
<ul>
<li>named after the two-faced Roman God Janus</li>
<li>cytosolic tyrosine kinases</li>
<li>they phosphorylate and activate gene regulatory proteins, called signal transducers and activators of transcription (<strong><font color="EE0000">STAT</font></strong>)</li>
<li>mediates cytokine signaling</li>
</ul>
<h2 id="Src-tyrosine-kinase-protein-family"><a href="#Src-tyrosine-kinase-protein-family" class="headerlink" title="Src tyrosine kinase protein family"></a>Src tyrosine kinase protein family</h2><p>The largest family of cytoplasmic tyrosine kinases</p>
<ul>
<li>Family members: <font color="EE0000">Src, Yes, Fgr, Fyn, Lck, Lyn, Hck, Blk</font>, etc<ul>
<li>All contain conserved SH2, SH3 and kinase domains.</li>
<li>All within the cytoplasmic region</li>
</ul>
</li>
<li>Binds to activated RTK, mutually stimulate each other</li>
</ul>
<p>Some G-proteins can also activate Src</p>
<p>Src is also phosphorylated by focal adhesion kinase (FAK)</p>
<h3 id="1-Src-kinase-has-“on”-and-“off”-states"><a href="#1-Src-kinase-has-“on”-and-“off”-states" class="headerlink" title="1. Src kinase has “on” and “off” states"></a>1. Src kinase has “on” and “off” states</h3><p>This part also described in L12</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708202802918.png" srcset="/img/loading.gif" lazyload alt="image-20200708202802918"></p>
<ul>
<li>Three distinct domains: SH2, SH3, kinase domain </li>
<li>OFF: phosphorylation of tyrosine (Y) 527 (inhibiting phosphotyrosine) </li>
<li>ON: phosphorylation of tyrosine (Y) 416 (activating phosphotyrosine)</li>
</ul>
<h1 id="三、Receptor-Ser-Thr-kinases"><a href="#三、Receptor-Ser-Thr-kinases" class="headerlink" title="三、Receptor Ser&#x2F;Thr kinases"></a>三、Receptor Ser&#x2F;Thr kinases</h1><h2 id="Serine-Threonine-kinase-receptor"><a href="#Serine-Threonine-kinase-receptor" class="headerlink" title="Serine&#x2F;Threonine kinase receptor"></a>Serine&#x2F;Threonine kinase receptor</h2><p>Single transmembrane receptor and Ser&#x2F;Thr kinase.</p>
<ul>
<li>Two classes: Type I and Type II, which are homodimers</li>
<li>Type II dimer phosphorylates Type I dimer to form active tetramer.</li>
</ul>
<p>They are receptors for transforming growth factor-β (<font color="EE0000">TGF-β</font>) protein superfamily:</p>
<ul>
<li>secreted and dimeric proteins</li>
<li>~33 members for human</li>
<li>two categories: TGF-β&#x2F;ativin family, bone morphogenetic protein (BMP) family</li>
</ul>
<p>Control diverse activity in differentiation, proliferation, cell death, development, etc.</p>
<h3 id="1-Smad-dependent-signaling-activated-by-TGF-β"><a href="#1-Smad-dependent-signaling-activated-by-TGF-β" class="headerlink" title="1. Smad-dependent signaling activated by TGF-β"></a>1. Smad-dependent signaling activated by TGF-β</h3><blockquote>
<p>Signal Proteins of the TGFβ Superfamily Act Through Receptor Serine&#x2F;Threonine Kinases and <font color="EE0000">Smads</font></p>
<ul>
<li>The <strong>transforming growth factor-β (TGFβ) superfamily</strong> consists of a large number (33 in humans) of structurally related, secreted, dimeric proteins.</li>
</ul>
<p>During development, they regulate pattern formation and influence various cell behaviors, including <strong>proliferation</strong>, <strong>specification</strong> and <strong>differentiation</strong>, <strong>extracellular matrix production</strong>, and <strong>cell death</strong>.</p>
</blockquote>
<blockquote>
<p>The superfamily consists of the <strong><font color="FF4500">TGFβ&#x2F;activin family</font></strong> and  the larger <strong><font color="FF4500">bone morphogenetic protein (BMP)</font></strong> family</p>
<p><u>All of these proteins <strong>act through</strong> enzyme-coupled receptors that are <strong>single-pass</strong> transmembrane proteins with a serine&#x2F;threonine kinase domain on the cytosolic side of the plasma membrane</u>. </p>
<ul>
<li>There are two classes of these receptor serine&#x2F;threonine kinases—<strong>type I and type II</strong>—which are structurally similar homodimers.</li>
</ul>
<p>Each member of the TGFβ superfamily binds to a <u>characteristic combination of type-I and type-II receptor dimers</u>, bringing the kinase domains together so that the <u>type-II receptor can phosphorylate and activate the type-I  receptor</u>, forming an active tetrameric receptor complex</p>
<ul>
<li>The activated type-I receptor directly binds and phosphorylates a latent transcription regulator of the <strong>Smad family</strong> (named after the first two proteins identified, Sma in C. elegans and Mad in Drosophila)</li>
</ul>
</blockquote>
<blockquote>
<p>TGFβ&#x2F;activin与BMP调控Smad的不同方式:</p>
<blockquote>
<p>Activated <font color="FF4500">TGFβ&#x2F;activin receptors</font> phosphorylate <strong><font color="0000EE">Smad2</font></strong> or <strong><font color="0000EE">Smad3</font></strong>, while activated <font color="FF4500">BMP receptors</font> phosphorylate <font color="0000EE"><strong>Smad1</strong>, <strong>Smad5</strong>, or <strong>Smad8</strong>.</font> </p>
<p>Once one of these receptor-activated Smads (<strong>R-Smads</strong>) has been phosphorylated, it dissociates from the receptor and binds to <font color="0000EE"><strong>Smad4</strong></font> (called a <strong>co-Smad</strong>), which can form a complex with any of the five R-Smads. </p>
<ul>
<li>The Smad complex then translocates into the nucleus, where it associates with other <strong>transcription regulators</strong> and controls the transcription of specific target genes (Figure 15–57). </li>
<li>Because the partner proteins in the nucleus vary depending on the cell type and state of the cell, the genes affected vary.</li>
</ul>
</blockquote>
</blockquote>
<blockquote>
<p>Activated TGFβ receptors and their bound ligand are <strong>endocytosed</strong> by two distinct routes, one leading to <strong>further activation</strong> and the other leading to <strong>inactivation.</strong> </p>
<ol>
<li>The activation route depends on <strong>clathrin-coated vesicles</strong> and leads to <strong>early endosomes</strong> (discussed in Chapter 13), where most of the Smad activation occurs.</li>
<li>The inactivation route depends on <strong>caveolae</strong> (discussed in Chapter 13) and leads to receptor <strong>ubiquitylation</strong> and <strong>degradation</strong> in  proteasomes.</li>
</ol>
<p>During the signaling response, the <strong>Smads</strong> <u>shuttle</u> continuously between the cytoplasm and the nucleus: they are <strong>dephosphorylated</strong> in the nucleus and exported to the cytoplasm</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708202958466.png" srcset="/img/loading.gif" lazyload alt="image-20200708202958466" style="zoom:50%;" />

<p>In activated tetramer, the type-II receptors phosphorylate specific sites on the type-I receptors, and activate their kinase domains  </p>
<ul>
<li>Phosphorylation R-Smads (Smad2 and  Smad3) (<strong>BMP signaling, R-Smads: 1, 5, 8</strong>) </li>
<li>Formation of a trimeric Smad complex containing two R-Smads and the co-Smad,  Smad4)</li>
</ul>
<p>The phosphorylated Smad complex enters the nucleus and regulate specific gene transcription</p>
<blockquote>
<p>As in other signaling systems, <strong>negative feedback</strong> regulates the Smad pathway. Among the target genes activated by Smad complexes are those that encode <strong>inhibitory Smads</strong>, either <font color="0000EE"><strong>Smad6</strong></font> or <font color="0000EE"><strong>Smad7</strong></font>. </p>
<p><strong>Smad7</strong> (and possibly <strong>Smad6</strong>) binds to the cytosolic tail of the activated receptor and inhibits its signaling ability in at least three ways:</p>
<ol>
<li><u>it competes with <strong>R-Smads</strong></u> for binding sites on the receptor, decreasing R-Smad phosphorylation</li>
<li>it <u>recruits a ubiquitin ligase</u> called <font color="EE0000"><strong>Smurf</strong></font>, which ubiquitylates the receptor, leading to receptor internalization and degradation (it is because <strong>Smurfs</strong> also ubiquitylate and promote the degradation of <strong>Smads</strong> that they are called <strong>Smad ubiquitylation regulatory factors</strong>, or Smurfs); </li>
<li>it <u>recruits a <strong>protein phosphatase</strong></u> that dephosphorylates and inactivates the receptor. In addition, the <strong>inhibitory Smads</strong> bind to the co-Smad, <font color="0000EE"><strong>Smad4</strong></font>, and  inhibit it, either by preventing its binding to <strong>R-Smads</strong> or by promoting its ubiquitylation and degradation.</li>
</ol>
<p>Although receptor serine&#x2F;threonine kinases operate mainly through the Smad pathway just described, they can also stimulate other intracellular signaling proteins such as <strong>MAP kinases</strong> and <strong>PI 3-kinase</strong>. </p>
<p>Conversely, signaling proteins in other  pathways can phosphorylate <strong>Smads</strong> and thereby influence signaling along the Smad pathways</p>
</blockquote>
<h1 id="四、Cytokine-receptors-that-activates-JAK-STAT-signaling"><a href="#四、Cytokine-receptors-that-activates-JAK-STAT-signaling" class="headerlink" title="四、Cytokine receptors that activates JAK-STAT signaling"></a>四、Cytokine receptors that activates JAK-STAT signaling</h1><blockquote>
<p>Cytokine Receptors Activate the JAK–STAT Signaling Pathway</p>
<p>The large family of <strong>cytokine receptors</strong> includes receptors for many kinds of local mediators (collectively called <strong>cytokines</strong>), as well as receptors for some hormones, such as <strong>growth hormone</strong> and <strong>prolactin</strong></p>
<p>These receptors are stably associated with cytoplasmic tyrosine kinases called <strong><font color="EE0000">Janus kinases (JAKs)</font></strong> (after the two-faced Roman god), which phosphorylate and activate transcription regulators called <strong><font color="EE0000">STATs</font></strong> (signal transducers and activators of transcription).</p>
<p><strong>STAT</strong> proteins are located in the cytosol and are referred to as <strong>latent transcription regulators</strong> because they <u>migrate into the nucleus</u> and regulate gene transcription only after they are activated.</p>
<p><strong>Negative feedback</strong> regulates the responses mediated by the JAK–STAT pathway.</p>
</blockquote>
<h2 id="JAK-STAT-signaling-pathway"><a href="#JAK-STAT-signaling-pathway" class="headerlink" title="JAK-STAT signaling pathway"></a>JAK-STAT signaling pathway</h2><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708203047738.png" srcset="/img/loading.gif" lazyload alt="image-20200708203047738"></p>
<p>JAK-Janus kinase (cytosolic tyrosine kinase)</p>
<p>Mediates growth hormone, prolactin, cytokine signaling</p>
<p>STAT-Signal transducers and activators of transcription (transcription factors)</p>
<h3 id="1-JAK-STAT-signaling-pathway-activated-by-cytokines"><a href="#1-JAK-STAT-signaling-pathway-activated-by-cytokines" class="headerlink" title="1. JAK-STAT signaling pathway activated by cytokines"></a>1. JAK-STAT signaling pathway activated by cytokines</h3><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708203112934.png" srcset="/img/loading.gif" lazyload alt="image-20200708203112934"></p>
<h3 id="2-How-does-prolactin-promote-milk-production"><a href="#2-How-does-prolactin-promote-milk-production" class="headerlink" title="2. How does prolactin promote milk production?"></a>2. How does prolactin promote milk production?</h3><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708203126121.png" srcset="/img/loading.gif" lazyload alt="image-20200708203126121"></p>
<h3 id="3-How-to-deactivate-phosphorylated-tyrosines"><a href="#3-How-to-deactivate-phosphorylated-tyrosines" class="headerlink" title="3. How to deactivate phosphorylated tyrosines?"></a>3. How to deactivate phosphorylated tyrosines?</h3><p>Phosphorylated tyrosines are dephosphorylated&#x2F;deactivated by tyrosine phosphatases</p>
<p>~ 100 protein tyrosine phosphatases in human genome, each has exquisite specificity for a subset  of proteins</p>
<h1 id="五、Cytokine-receptors-that-activates-NF-kappa-B-signaling"><a href="#五、Cytokine-receptors-that-activates-NF-kappa-B-signaling" class="headerlink" title="五、Cytokine receptors that activates NF$\kappa$B signaling"></a>五、Cytokine receptors that activates NF$\kappa$B signaling</h1><blockquote>
<p>Many Stressful and Inflammatory Stimuli Act Through an NFκB-Dependent Signaling Pathway</p>
<p>The <strong>NFκB proteins</strong> are <u>latent transcription regulators</u> that are present in most animal cells and are central to many stressful, inflammatory, and innate immune responses. These responses occur as a reaction to <strong>infection</strong> or <strong>injury</strong> and help protect stressed multicellular organisms and their cells</p>
<ul>
<li>An excessive or inappropriate inflammatory response in animals can also damage tissue and cause severe pain, and chronic inflammation can lead to cancer; as in the case of <font color="EE0000">Wnt</font> and <font color="EE0000">Hedgehog</font> signaling, <strong>excessive NFκB signaling</strong> is found in a number of human cancers.</li>
<li>NFκB proteins also have important roles during normal animal development: the Drosophila NFκB family member <strong><font color="EE0000">Dorsal</font></strong>, for  example, has a crucial role in specifying the dorsal–ventral axis of the developing fly embryo (discussed in Chapter 22).</li>
</ul>
</blockquote>
<blockquote>
<p>Various cell-surface receptors activate the NFκB signaling pathway in animal cells.</p>
<ul>
<li><strong>Toll receptors</strong> in Drosophila and Toll-like receptors in vertebrates, for example, recognize pathogens and activate this pathway in triggering innate immune responses (discussed in Chapter 24).</li>
<li>The receptors for <strong>tumor necrosis factor α</strong> (<strong><font color="EE0000">TNFα</font></strong>) and <strong><font color="EE0000">interleukin-1 (IL1)</font></strong>, which are vertebrate cytokines especially important in inducing inflammatory responses, also activate this signaling pathway.</li>
</ul>
<p>The <font color="EE0000">Toll, Toll-like, and IL1 receptors</font> belong to the same family of proteins, whereas TNF receptors belong to a different family; all of them, however, act in similar ways to <strong>activate NFκB</strong></p>
<p>When activated, they trigger a multiprotein ubiquitylation and phosphorylation cascade that <u>releases NFκB from an inhibitory protein complex</u>,  so that it can translocate to the nucleus and turn on the transcription of hundreds of genes that participate in inflammatory and innate immune responses.</p>
</blockquote>
<blockquote>
<p>There are five NFκB proteins in mammals (<font color="EE0000"><strong>RelA</strong>, <strong>RelB</strong>, <strong>c-Rel</strong>, <strong>NFκB1</strong>, and  <strong>NFκB2</strong></font>), and they form a variety of <strong>homodimers</strong> and <strong>heterodimers</strong>, each of which activates its own characteristic set of genes.</p>
<ul>
<li><strong>Inhibitory proteins</strong> called <strong><font color="EE0000">IκB</font></strong> bind tightly to the dimers and hold them in an inactive state within the cytoplasm of unstimulated cells. There are three major <strong>IκB</strong> proteins in mammals (<strong><font color="EE0000">IκB α, β, and ε</font></strong>), and the signals that release NFκB dimers do so by triggering a signaling pathway that leads to the phosphorylation, ubiquitylation, and consequent degradation of the IκB proteins</li>
</ul>
<p>Among the genes activated by the released NFκB is the gene that encodes  <strong>IκBα</strong>. This activation leads to increased synthesis of <strong>IκBα protein</strong>, which binds to NFκB and inactivates it, creating a negative <strong>feedback loop</strong></p>
</blockquote>
<h2 id="NF-kappa-B-pathway"><a href="#NF-kappa-B-pathway" class="headerlink" title="NF$\kappa$B pathway"></a>NF$\kappa$B pathway</h2><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708203846491.png" srcset="/img/loading.gif" lazyload alt="image-20200708203846491"></p>
<p>NF$\kappa$B proteins are <u>latent transcription regulators</u> that are present in most animal cells and are central to many stressful, inflammatory, and innate immune responses.</p>
<p>Various cell-surface receptors can activate the NF$\kappa$B signaling pathway in animal cells. </p>
<ul>
<li>Toll receptors in fly</li>
<li>Toll-like receptors (<font color="EE0000">TLR</font>) in vertebrates (recognize pathogens and activate this pathway in triggering innate immune response)</li>
<li>Cytokines responsible for inflammatory responses <ul>
<li>Interleukin-1 (IL1 ) </li>
<li>Tumor necrosis factor a (TNF$\alpha$ )</li>
</ul>
</li>
</ul>
<blockquote>
<p>Nuclear Receptors Are Ligand-Modulated Transcription Regulators</p>
<p>Various <u>small, hydrophobic</u> signal molecules diffuse directly across the plasma membrane of target cells and bind to intracellular receptors that are <strong>transcription regulators</strong>. These signal molecules include <strong>steroid hormones</strong>, <strong>thyroid hormones</strong>,  <strong>retinoids</strong>, and <strong>vitamin D</strong>.</p>
<p>They bind to their <u>respective intracellular receptor proteins</u> and alter the ability of these proteins to control the transcription of specific genes. Thus, these proteins serve both as <u>intracellular receptors</u> and as <u>intracellular effectors</u> for the signal.</p>
<p>The receptors are all structurally related, being part of the very large nuclear receptor superfamily</p>
</blockquote>
<blockquote>
<p>Many family members have been identified by DNA sequencing only, and their ligand is not yet known; they are therefore referred to as <strong>orphan nuclear receptors</strong>, and they make up large fractions of the nuclear receptors encoded in the genomes of humans, Drosophila, and the nematode  C. elegans.</p>
<p>Some mammalian nuclear receptors are regulated by intracellular metabolites rather than by secreted signal molecules; the <strong>peroxisome proliferation-activated receptors (PPARs)</strong>, for example, bind intracellular lipid metabolites and regulate the transcription of genes involved in lipid metabolism and fat-cell differentiation</p>
<p><strong>Steroid hormones</strong>—which include <strong>cortisol</strong> (皮质(甾)醇), the <strong>steroid sex hormones</strong>, <strong>vitamin D</strong> (in vertebrates), and the molting hormone <strong>ecdysone</strong> (蜕化素；蜕皮激素) (in insects)—are all made from cholesterol</p>
<ul>
<li><strong>Cortisol</strong> is produced in the <strong>cortex of the adrenal glands</strong> and influences the <u>metabolism</u> of many types of cells. The <strong>steroid sex hormones</strong> are made in the <u>testes and ovaries</u> and are responsible for the <u>secondary sex characteristics</u> that distinguish males from females.</li>
<li><strong>Vitamin D</strong> is synthesized in the skin in response to sunlight; after it has been converted to its active form in the liver or kidneys, it regulates Ca2+ metabolism, promoting Ca2+ uptake in the gut and reducing its excretion in the kidneys</li>
<li>The <strong>thyroid hormones</strong>, which are made from the amino acid <strong>tyrosine</strong>, act to <u>increase the metabolic rate</u> of many cell types, while the <strong>retinoids</strong>, such as <strong>retinoic acid</strong>, are made from vitamin A and have important roles as local mediators in vertebrate development.</li>
</ul>
<p>Although all of these signal molecules are relatively <u>insoluble in water</u>, they are made <u>soluble</u> for transport in the bloodstream and other extracellular fluids by binding to specific carrier proteins, from which they dissociate before entering a target cell</p>
<p>The <strong>inactive receptors</strong> are usually bound to <strong>inhibitory protein complexes</strong>. Ligand binding <strong>alters the conformation</strong> of the receptor protein, causing the inhibitory complex to dissociate,  while also causing the receptor to bind <strong>coactivator</strong> proteins that stimulate gene transcription</p>
</blockquote>
<h3 id="1-Negative-feedback-in-the-NF-kappa-B-signaling-pathway-induced-in-NF-kappa-B-activation"><a href="#1-Negative-feedback-in-the-NF-kappa-B-signaling-pathway-induced-in-NF-kappa-B-activation" class="headerlink" title="1. Negative feedback in the NF$\kappa$B signaling pathway  induced in NF$\kappa$B activation"></a>1. Negative feedback in the NF$\kappa$B signaling pathway  induced in NF$\kappa$B activation</h3><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708204036410.png" srcset="/img/loading.gif" lazyload alt="image-20200708204036410"></p>
<h1 id="六、Other-signaling-Notch-Wnt-Hedgehog-and-Circadian-clock"><a href="#六、Other-signaling-Notch-Wnt-Hedgehog-and-Circadian-clock" class="headerlink" title="六、Other signaling: Notch, Wnt, Hedgehog and Circadian clock"></a>六、Other signaling: Notch, Wnt, Hedgehog and Circadian clock</h1><h2 id="Notch-pathway"><a href="#Notch-pathway" class="headerlink" title="Notch pathway"></a>Notch pathway</h2><blockquote>
<p>The Receptor Notch Is a Latent Transcription Regulatory Protein</p>
<p>it has a general role in controlling <strong>cell fate</strong> <strong>choices</strong> and regulating <strong>pattern formation</strong> during the development of most tissues,  as well as in the <strong>continual renewal of tissues</strong> such as the lining of the gut.</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708204050373.png" srcset="/img/loading.gif" lazyload alt="image-20200708204050373"></p>
<p>In fly neural cells, when a precursor cell commits to becoming a neural cell, it signals to its immediate neighbors not to do the same; the inhibited cells develop into epidermal cells instead. This process is called lateral  inhibition. </p>
<blockquote>
<p>This process, called <strong>lateral inhibition</strong>, depends on a  contact-dependent signaling mechanism that is activated by a single-pass transmembrane signal protein called <strong>Delta</strong></p>
</blockquote>
<p>This depends on a contact-dependent signaling mechanism that is activated by a single-pass transmembrane signal protein called <font color="EE0000">Delta</font> that activates <font color="EE0000">Notch receptors</font>. </p>
<p>By activation, Notch cleaved and its tail migrates to nucleus where activates responsible genes.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708204107482.png" srcset="/img/loading.gif" lazyload alt="image-20200708204107482" style="zoom:50%;" />

<blockquote>
<p>Notch is a single-pass transmembrane protein that requires <strong>proteolytic processing</strong> to function. It acts as a <strong>latent transcription regulator</strong> and provides the simplest and most direct signaling pathway known from a cell-surface receptor to the nucleus.</p>
<p>Note that, unlike most receptors, the activation of  Notch is <strong>irreversible</strong>; once activated by ligand binding, the protein cannot be used  again.</p>
<p>Both Notch and Delta are <strong>glycoproteins</strong>, and their interaction is regulated by  the <strong>glycosylation</strong> of Notch.</p>
<ul>
<li>The <strong>Fringe family of glycosyl transferases</strong>, in particular,  adds extra sugars to the <strong>O-linked oligosaccharide</strong> (discussed in Chapter 13) on Notch, which alters the specificity of Notch for its ligands.</li>
</ul>
</blockquote>
<h3 id="1-The-processing-and-activation-of-Notch"><a href="#1-The-processing-and-activation-of-Notch" class="headerlink" title="1. The processing and activation of Notch"></a>1. The processing and activation of Notch</h3><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708204131047.png" srcset="/img/loading.gif" lazyload alt="image-20200708204131047"></p>
<p>3-step proteolytic processing </p>
<ol>
<li><p>The first proteolytic processing step occurs within the trans Golgi network to generate the mature heterodimeric Notch  receptor that is then displayed on the cell surface.  </p>
</li>
<li><p>The binding to Delta on a neighboring cell triggers the next two proteolytic steps:  </p>
<ul>
<li>Delta-Notch complex is endocytosed by the Delta expressing cell, exposing the extracellular cleavage site in the transmembrane Notch subunit.</li>
</ul>
</li>
<li><p>The released Notch tail <u>migrates into the nucleus</u>, where it binds to the <font color="EE0000">Rbpsuh</font> protein, which it converts from a  <strong>transcriptional repressor</strong> to a transcriptional activator.</p>
</li>
</ol>
<h2 id="Wnt-pathway"><a href="#Wnt-pathway" class="headerlink" title="Wnt pathway"></a>Wnt pathway</h2><blockquote>
<p>Wnt Proteins Bind to <font color="EE0000"><strong>Frizzled Receptors</strong></font> and <u>Inhibit</u> the  Degradation of <strong>β-Catenin</strong></p>
<p><strong>Wnt proteins</strong> are <strong>secreted</strong> signal molecules that act as local mediators and <strong>morphogens</strong> (形态发生子) to control many aspects of development in all animals that have been studied</p>
<ul>
<li>They were discovered independently in flies and in mice: in Drosophila,  the <strong><font color="0000EE">Wingless (Wg) gene</font></strong> originally came to light because of its role as a morphogens in wing development, while in mice, the <strong><font color="0000EE">Int1 gene</font></strong> was found because it promoted the development of breast tumors when activated by the integration of a virus next to it</li>
</ul>
<p>Wnts are unusual as secreted proteins in that they have a <strong>fatty acid chain</strong> covalently attached to their <strong>N-terminus</strong>, which <u>increases their binding to cell surfaces.</u></p>
</blockquote>
<blockquote>
<p>Wnts can activate at least two types of intracellular signaling pathways. </p>
<ol>
<li>Our primary focus here is the <strong><font color="EE0000">Wnt&#x2F;β-catenin pathway</font></strong> (also known as the <strong><font color="EE0000">canonical  Wnt pathway</font></strong>), which is centered on the <u>latent transcription regulator</u> <strong>β-catenin</strong>.  </li>
<li>A second pathway, called the <strong><font color="EE0000">planar polarity pathway</font></strong>, coordinates the <u><strong>polarization</strong> of cells</u> in the plane of a developing epithelium and depends on <strong><font color="EE0000">Rho</font></strong> family GTPases. Both of these pathways begin with the binding of Wnts to <strong><font color="EE0000">Frizzled</font></strong> family cell-surface receptors<ul>
<li>when activated by Wnt binding, Frizzled proteins recruit the <strong>scaffold protein</strong> <strong><font color="EE0000">Dishevelled</font></strong>, which helps relay the signal to other signaling  molecules</li>
</ul>
</li>
</ol>
</blockquote>
<blockquote>
<p>Wnt proteins regulate <strong>β-catenin</strong> proteolysis by binding to <strong>both</strong> a <strong>Frizzled protein</strong> and a co-receptor that is related to the <strong>low-density lipoprotein (LDL) receptor</strong>  (discussed in Chapter 13) and is therefore called an <strong><font color="EE0000">LDL-receptor-related protein (LRP)</font></strong>.</p>
<ul>
<li>In a poorly understood process, the activated receptor complex recruits the <strong>Dishevelled scaffold</strong> and promotes the <u>phosphorylation of the LRP receptor</u> by the two protein kinases, <strong><font color="0000EE">GSK3</font></strong> and <strong><font color="0000EE">CK1</font></strong>.</li>
<li><strong>Axin</strong> is brought to the receptor complex and inactivated, thereby disrupting the β-catenin degradation complex in the cytoplasm. In this way, the phosphorylation and degradation of β-catenin are prevented, enabling unphosphorylated β-catenin to <u>accumulate and translocate</u>  to the nucleus, where it alters the pattern of gene transcription</li>
</ul>
</blockquote>
<blockquote>
<p>In the absence of Wnt signaling, Wnt-responsive genes are <u>kept silent</u> by an <u>inhibitory complex</u> of transcription regulatory proteins. The complex includes proteins of the <strong><font color="0000EE">LEF1&#x2F;TCF family</font></strong> bound to a co-repressor protein of the <strong><font color="0000EE">Groucho</font></strong> family (see Figure 15–60A).</p>
<ul>
<li>In response to a Wnt signal, <strong>β-catenin</strong> enters the nucleus and binds to the <strong>LEF1&#x2F;TC</strong>F proteins, <u>displacing</u> <strong>Groucho</strong>. The <strong>β-catenin</strong>  now functions as a coactivator, inducing the transcription of the Wnt target genes  (see Figure 15–60B).</li>
</ul>
<p>Thus, as in the case of Notch signaling, <strong>Wnt&#x2F;β-catenin signaling</strong> triggers a switch <u>from transcriptional repression to transcriptional activation.</u></p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709194707975.png" srcset="/img/loading.gif" lazyload alt="image-20200709194707975"></p>
<p>Co-receptor: LRP ( LDL-receptor-related protein) and  Frizzled </p>
<p>Latent transcription regulators:  $\beta$-catenin </p>
<p>Scaffold protein: </p>
<ul>
<li>Adenomatous polyposis coli  (APC) </li>
<li>axin</li>
</ul>
<blockquote>
<p>Among the genes activated by β-catenin is <strong><font color="EE0000">Myc</font></strong>, which encodes a protein (<strong>Myc</strong>) that is an important regulator of <strong>cell growth</strong> and <strong>proliferation</strong> (discussed  in Chapter 17).</p>
<p>Mutations of the <strong><font color="EE0000">Apc</font></strong> gene occur in 80% of human <u>colon cancers</u>  (discussed in Chapter 20).</p>
<p>Various secreted <strong>inhibitory proteins</strong> regulate Wnt signaling in development.  </p>
<ul>
<li>Some <u>bind to the LRP receptors and promote their down-regulation</u></li>
<li>whereas others <u>compete with <strong>Frizzled</strong> receptors</u> for secreted Wnts. </li>
<li>In Drosophila at least,  Wnts activate <strong>negative feedback loops</strong>, in which Wnt target genes encode proteins that help shut the response off; some of these proteins <u>inhibit <strong>Dishevelled</strong></u>, and  others are <u>secreted inhibitors</u>.</li>
</ul>
</blockquote>
<h2 id="Hedgehog-pathway"><a href="#Hedgehog-pathway" class="headerlink" title="Hedgehog pathway"></a>Hedgehog pathway</h2><blockquote>
<p>Hedgehog Proteins Bind to <strong>Patched</strong>, Relieving Its Inhibition of <strong>Smoothened</strong></p>
<p>Hedgehog proteins and Wnt proteins act in similar ways. Both are <strong>secreted signal molecules</strong>, which act as <strong>local mediators</strong> and <strong>morphogens</strong> in many developing invertebrate and vertebrate tissues.</p>
<ul>
<li>Both proteins are <strong>modified</strong> by covalently attached <strong>lipids</strong>, depend on secreted or cell-surface-bound <strong>heparan sulfate proteoglycans</strong> (discussed in Chapter 19) for their action, and activate latent transcription regulators by <u>inhibiting their degradation</u></li>
</ul>
<p>The <strong>Hedgehog proteins</strong> were discovered in Drosophila, where this protein family has only one member. Mutation of the Hedgehog gene produces a larva covered with <u>spiky processes (denticles)</u>, like a hedgehog.</p>
<p>At least three genes encode Hedgehog proteins in vertebrates—**<font color="EE0000">Sonic</font>**, <strong><font color="EE0000">Desert</font></strong>, and <strong><font color="EE0000">Indian hedgehog</font></strong>.  <u>The active forms of all Hedgehog proteins are covalently coupled to <strong>cholesterol</strong>, as  well as to a <strong>fatty acid chains</strong></u></p>
</blockquote>
<blockquote>
<p>The effects of Hedgehog are mediated by a <strong>latent transcription regulator</strong> called <font color="EE0000"><strong>Cubitus interruptus</strong> (<strong>Ci</strong>)</font>, the regulation of which is reminiscent of the regulation  of <strong>β-catenin</strong> by Wnts.</p>
<ul>
<li>In the absence of a Hedgehog signal, <strong>Ci</strong> is <u>ubiquitylated and proteolytically</u> cleaved in proteasomes. Instead of being completely degraded, however, Ci is processed to <u>form a smaller fragment</u>, which accumulates in the nucleus, where it acts as a transcriptional repressor, helping to <u>keep Hedgehog-responsive genes silent.</u></li>
<li>The proteolytic processing of the Ci protein depends on its phosphorylation by three protein kinases—**<font color="EE0000">PKA</font>** and two kinases also used  in the Wnt pathway, namely <strong><font color="EE0000">GSK3</font></strong> and <strong><font color="EE0000">CK1</font></strong>.</li>
<li>As in the Wnt pathway, the proteolytic processing occurs in a multiprotein complex. The complex includes the protein kinase <em><strong><font color="EE0000">Fused</font></strong></em> and a <strong>scaffold protein</strong> <em><strong><font color="EE0000">Costal2</font></strong></em>, which stably associates with <strong>Ci,</strong> recruits the three other kinases, and binds the complex to <strong>microtubules</strong>, thereby keeping unprocessed <strong>Ci</strong> out of the nucleus</li>
</ul>
</blockquote>
<blockquote>
<p>Hedgehog functions by <u>blocking the proteolytic processing of <strong>Ci</strong>,</u> thereby changing it into a <strong>transcriptional activator</strong>. It does this by a convoluted signaling process that depends on three transmembrane proteins: <font color="EE0000"><strong>Patched</strong>, <strong>iHog</strong>, and  <strong>Smoothened</strong></font></p>
<ul>
<li><strong>Patched</strong> is predicted to cross the plasma membrane 12 times, and, although much of it is in intracellular vesicles, some is on the cell surface where it can bind the Hedgehog protein.</li>
<li><strong>iHog</strong> is also on the cell surface and is thought to serve as a co-receptor for Hedgehog.</li>
<li><strong>Smoothened</strong> is a seven-pass transmembrane protein with a structure very <u>similar to a GPCR</u>, but it does not seem to act as a Hedgehog receptor or even as an activator of G proteins; <u>it is controlled by  Patched and iHog</u>.</li>
</ul>
<p>In the absence of a Hedgehog signal, Patched employs an unknown mechanism to <u>keep Smoothened sequestered</u> and inactive in intracellular vesicles (see Figure 15–61A).</p>
<p>The binding of Hedgehog to iHog and Patched inhibits the activity of Patched and induces its endocytosis and degradation.</p>
<ul>
<li>The result is that Smoothened is liberated from inhibition and translocates to the plasma membrane, where it recruits the protein complex containing <strong>Ci,</strong> <strong>Fused</strong>, and <strong>Costal2</strong>.</li>
</ul>
<p>Hedgehog signaling can <u>promote cell proliferation</u>, and excessive Hedgehog signaling can lead to cancer</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709194805512.png" srcset="/img/loading.gif" lazyload alt="image-20200709194805512"></p>
<p>Patched is predicted to cross the  plasma membrane 12 times </p>
<p>iHog is also on the cell surface  and is thought to serve as a co-receptor for Hedgehog. </p>
<p>Smoothened is a seven-pass  transmembrane  </p>
<p>It is controlled by Patched and  iHog.</p>
<h2 id="Circadian-clock"><a href="#Circadian-clock" class="headerlink" title="Circadian clock"></a>Circadian clock</h2><blockquote>
<p>Circadian Clocks Contain Negative Feedback Loops That Control Gene Expression</p>
<p>The internal oscillators that control such <strong>diurnal rhythms</strong> are called <strong>circadian clocks.</strong></p>
<p>External signals indicating the time of day cause <strong>small adjustments</strong> in the running of the clock, so as to keep the organism <strong>in synchrony</strong> with its environment.</p>
<p>in almost all multicellular organisms,  including humans, the timekeepers are <strong>individual cells</strong>. </p>
<ul>
<li>Thus, a clock that operates in each member of a specialized group of brain cells (the <strong><font color="EE0000">SCN cells</font></strong> in the <strong>suprachiasmatic nucleus</strong> of the <strong>hypothalamus</strong>) controls our diurnal cycles of sleeping and waking, body temperature, and <strong>hormone release</strong>.</li>
</ul>
<p>In the intact body, the SCN cells receive neural cues from the retina, entraining the <strong>SCN cells</strong> to the daily cycle of light and dark; they also  send information about the time of day to another brain area, the <strong>pineal gland (松果腺；松果体)</strong>,  which relays the time signal to the rest of the body by releasing the hormone <strong>melatonin（褪黑激素）</strong> in time with the clock.</p>
<ul>
<li>The key principle is that circadian clocks generally depend on <strong>negative feedback loops</strong></li>
</ul>
<p>In Drosophila and many other animals, including humans, the heart of the circadian clock is a <strong>delayed negative feedback loop</strong> based on <strong>transcription regulators</strong>: accumulation of certain gene products switches off the transcription of their own genes</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709195344032.png" srcset="/img/loading.gif" lazyload alt="image-20200709195344032"></p>
<p>The internal oscillators that  control such diurnal rhythms  are called <strong>circadian clocks</strong>. </p>
<p>A central feature of the clock  is the periodic accumulation  and decay of two transcription regulatory proteins </p>
<ul>
<li><font color="EE0000">Tim</font> (short for timeless) </li>
<li><font color="EE0000">Per</font> (short for period).</li>
</ul>
<p>Negative feedback </p>
<p>In fly, there are no photoreceptor, fly uses <strong>cryptochrome</strong> (flavoprotein) to sense the light</p>
<h3 id="1-Oscillation-of-circadian-clock-in-cyanobacteria"><a href="#1-Oscillation-of-circadian-clock-in-cyanobacteria" class="headerlink" title="1. Oscillation of circadian clock in cyanobacteria"></a>1. Oscillation of circadian clock in cyanobacteria</h3><blockquote>
<p>Three Proteins in a Test Tube Can Reconstitute a Cyanobacterial Circadian Clock </p>
<p>The core oscillator in this organism is remarkably simple, being composed of just three proteins—<font color="EE0000"><strong>KaiA, KaiB, and KaiC</strong></font>. The central player is <strong><font color="EE0000">KaiC</font></strong>, a <u>multifunctional enzyme</u> that catalyzes its own <u>phosphorylation and dephosphorylation</u> in a 24-hour cycle: </p>
<ul>
<li>it gradually phosphorylates itself sequentially at two sites during the day and <u>dephosphorylates itself during the night.</u></li>
</ul>
<p>This timing depends on interactions with the two other Kai proteins: </p>
<ol>
<li><strong>KaiA</strong>  binds to <strong>unphosphorylated KaiC</strong> and <u>stimulates KaiC autophosphosphorylation</u>,  first at one site and then, with a delay, at the other. </li>
<li>The second phosphorylation  promotes the binding of the third protein, <strong>KaiB</strong>, which <u>blocks the stimulatory effect of KaiA</u> and thereby allows KaiC to dephosphorylate itself, bringing KaiC back to its dephosphorylated state.</li>
</ol>
<p>Circadian oscillations in <strong>KaiC</strong> phosphorylation lead to parallel rhythms in the expression of large numbers of genes involved in controlling <u>metabolic activities</u>  and <u>cell division</u></p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709200018756.png" srcset="/img/loading.gif" lazyload alt="image-20200709200018756"></p>
<p>KaiC is a combined kinase and phosphatase that phosphorylates and dephosphorylates itself on two adjacent sites. </p>
<ul>
<li>In the absence of other proteins, the phosphatase activity is dominant, and the protein is mostly unphosphorylated.</li>
</ul>
<p>The binding of KaiA to KaiC suppresses the phosphatase activity and promotes the kinase activity leading to KaiC phosphorylation.</p>
<ul>
<li>This form of KaiC interacts with KaiB, which blocks the stimulatory effects of KaiA, thereby reducing the rate of KaiC phosphorylation and allowing dephosphorylation to occur.</li>
</ul>
<p>Diphosphorylated KaiC increases in abundance during the day and peaks around dusk. </p>
<p><font color="EE0000">RpaA</font>: a transcription regulator</p>
<h1 id="七、Plant-signaling"><a href="#七、Plant-signaling" class="headerlink" title="七、Plant signaling"></a>七、Plant signaling</h1><blockquote>
<p>Receptor Serine&#x2F;Threonine Kinases Are the Largest Class of Cell-Surface Receptors in Plants</p>
<p>Most cell-surface receptors in plants are enzyme-coupled. However, whereas the largest class of enzyme-coupled receptors in animals is the <strong>receptor tyrosine kinase (RTK)</strong> class, this type of receptor is extremely rare in plants. Instead, plants rely largely on a great diversity of transmembrane receptor serine&#x2F;threonine  kinases</p>
<p>The most abundant types of these receptors  have a <strong>tandem array</strong> of <strong>extracellular leucine-rich repeat</strong> structures and are therefore called <strong>leucine-rich repeat (LRR) receptor kinases</strong>.</p>
<ul>
<li>There are about 175 <strong>LRR receptor</strong> kinases encoded by the Arabidopsis genome. These include a protein called <strong><font color="EE0000">Bri1</font></strong>, which forms part of a cell-surface steroid hormone receptor</li>
<li>Plants synthesize a class of steroids that are called <strong>brassinosteroids (油菜素类固醇)</strong> because they were originally identified in the mustard family Brassicaceae, which includes Arabidopsis.<ul>
<li>Binding of a brassinosteroid to a <strong>Bri1</strong> cell-surface receptor kinase initiates an intracellular signaling cascade that uses a <font color="EE0000"><strong>GSK3</strong></font> protein kinase and a protein phosphatase to regulate the phosphorylation and degradation of specific transcription regulatory proteins in the nucleus, and thereby specific gene transcription.</li>
</ul>
</li>
</ul>
<p>The <strong>lectin receptor kinases</strong>, for example, have extracellular domains that bind <strong>carbohydrate</strong> signal molecules.</p>
</blockquote>
<h2 id="Response-to-stimuli-tropisms"><a href="#Response-to-stimuli-tropisms" class="headerlink" title="Response to stimuli: tropisms"></a>Response to stimuli: tropisms</h2><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709200405435.png" srcset="/img/loading.gif" lazyload alt="image-20200709200405435"></p>
<h2 id="Plant-hormones"><a href="#Plant-hormones" class="headerlink" title="Plant hormones"></a>Plant hormones</h2><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709200522231.png" srcset="/img/loading.gif" lazyload alt="image-20200709200522231" style="zoom:50%;" />

<p>Plant cells are not using RTKs, steroid hormones,  cyclic AMPs </p>
<p>GPCRs are known for plants, but there are few of  them </p>
<p>In contrast, plants have specific receptors for specific plant molecules—plant hormones</p>
<h2 id="Similar-signaling-pathways-exist-in-plants-to-regulate-plant-growth"><a href="#Similar-signaling-pathways-exist-in-plants-to-regulate-plant-growth" class="headerlink" title="Similar signaling pathways exist in plants to regulate plant growth"></a>Similar signaling pathways exist in plants to regulate plant growth</h2><h3 id="1-Ethylene"><a href="#1-Ethylene" class="headerlink" title="1. Ethylene"></a>1. Ethylene</h3><blockquote>
<p>Ethylene Blocks the Degradation of Specific Transcription Regulatory Proteins in the Nucleus</p>
<p>Various plant growth regulators (also called plant hormones) help to coordinate plant development. They include <strong>ethylene (乙烯)</strong>, <strong>auxin (植物生长素,植物激素)</strong>, <strong>cytokinins (细胞激肽类)</strong>, <strong>gibberellins (赤霉素)</strong>, and  <strong>abscisic acid (脱落酸)</strong>, as well as <strong>brassinosteroids (油菜素类固醇).</strong></p>
<p>The specific effect depends on <strong>environmental conditions</strong>, the <strong>nutritional state</strong> of the plant, the <strong>responsiveness</strong> of the target cells, and which <strong>other growth regulators</strong> are acting.</p>
<p><strong>Ethylene</strong> is an important example. This small gas molecule (Figure 15–69A)  can influence plant development in various ways; it can, for example, <u>promote fruit ripening, leaf abscission, and plant senescence</u>. It also functions as a stress signal in response to <u>wounding, infection, flooding, and so on</u></p>
<p>Plants have various ethylene receptors, which are located in the endoplasmic reticulum and are all structurally related. They are <strong>dimeric</strong>, <strong>multipass</strong> transmembrane proteins, with a <strong>copper-containing ethylene-binding domain</strong> and a domain that interacts with a cytoplasmic protein called <font color="EE0000"><strong>CTR1</strong></font>, which is closely related in sequence to the <strong><font color="EE0000">Raf</font></strong> <strong>MAP kinase kinase kinase</strong> discussed earlier</p>
<ul>
<li>Surprisingly, it is the empty receptors that are active and keep <strong>CTR1</strong> active. By an unknown signaling mechanism, active <strong>CTR1</strong> stimulates the <strong>ubiquitylation</strong>  and <strong>degradation</strong> in proteasomes of a nuclear transcription regulator called <strong><font color="EE0000">EIN3</font></strong>,  which is required for the transcription of <strong>ethylene-responsive genes</strong>.</li>
</ul>
</blockquote>
<ul>
<li>Enzyme signaling</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709200558060.png" srcset="/img/loading.gif" lazyload alt="image-20200709200558060" style="zoom: 43%;" height = 1500 width = 1400/>

<p>Ethylene pathway </p>
<ul>
<li>Ethylene promote fruit ripening, leaf abscission, and  plant senescence.  </li>
<li>It also functions as a stress signal in response to  wounding, infection, flooding, and so on. </li>
<li>Ethylene receptor is somewhat similar to enzyme-coupled receptors of animals but it is dimeric and  active with an absence of signal (!) </li>
<li>Ethylene presence will make receptor kinase  inactive and transcription regulators will not  degrade but start to activate transcription</li>
</ul>
<h3 id="2-The-auxin-signaling-pathway"><a href="#2-The-auxin-signaling-pathway" class="headerlink" title="2. The auxin signaling pathway"></a>2. The auxin signaling pathway</h3><blockquote>
<p>Regulated Positioning of Auxin Transporters Patterns Plant Growth</p>
<p>The plant hormone <strong>auxin</strong>, which is generally indole-3-acetic acid (Figure  15–71A), binds to receptor proteins <u>in the nucleus</u>. It helps plants grow <u>toward  light</u>, grow upward rather than branch out, and grow their roots downwards </p>
<ul>
<li>It also  regulates organ initiation and positioning and helps plants flower and bear fruit.</li>
</ul>
<p>auxin influences <strong>gene expression</strong> by controlling the <strong>degradation</strong> of <strong>transcription regulators</strong>. It works by stimulating the ubiquitylation and degradation of repressor proteins that block the transcription of auxin target genes in unstimulated cells</p>
<p>Specific plasma-membrane-bound <strong>influx transporter proteins</strong> and <strong>efflux transporter proteins</strong> move auxin into and out of plant cells, respectively.</p>
<p>The efflux transporters can be distributed <strong>asymmetrically</strong> in the plasma membrane to make the efflux of auxin <strong>directional</strong>.</p>
<p>In some regions of the plant, the localization of the auxin transporters, and therefore the direction of auxin flow, is highly <strong>dynamic</strong> and <strong>regulated</strong></p>
<ul>
<li>A cell can rapidly <u>redistribute transporters</u> by controlling the traffic of vesicles containing them.</li>
<li>A cell can redistribute these transporters on its surface by <u>inhibiting their endocytosis</u> in one domain of the plasma membrane, causing the transporters to accumulate there</li>
</ul>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709200629161.png" srcset="/img/loading.gif" lazyload alt="image-20200709200629161"></p>
<p>Auxin </p>
<ul>
<li>are a group of related hormones and are  responsible for a wide range of growth and  development effects in plants. </li>
<li>promotes initiation of lateral roots, new leaves </li>
<li>regulates elongation of stem and root cells</li>
</ul>
<h4 id="Auxin-transport-and-rot-gravitropism"><a href="#Auxin-transport-and-rot-gravitropism" class="headerlink" title="Auxin transport and rot gravitropism"></a>Auxin transport and rot gravitropism</h4><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709200827600.png" srcset="/img/loading.gif" lazyload alt="image-20200709200827600"></p>
<h3 id="3-Photoperiod"><a href="#3-Photoperiod" class="headerlink" title="3. Photoperiod"></a>3. Photoperiod</h3><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709200928835.png" srcset="/img/loading.gif" lazyload alt="image-20200709200928835"></p>
<p>Florists apply knowledge of the photoperiod of particular plants to bring us flowers out of season.</p>
<h4 id="Light-response-mediated-by-phytochromes"><a href="#Light-response-mediated-by-phytochromes" class="headerlink" title="Light response mediated by phytochromes"></a>Light response mediated by phytochromes</h4><blockquote>
<p>Phytochromes Detect Red Light, and Cryptochromes Detect Blue Light</p>
<p>Plants have thus evolved a large set of <strong>light-sensitive proteins</strong> to monitor the <strong>quantity</strong>, <strong>quality</strong>, <strong>direction</strong>, and <strong>duration</strong> of light. These are usually referred to as photoreceptors.</p>
<ul>
<li>However, because the term photoreceptor is also used for light-sensitive cells in the animal retina (see Figure 15–38), we shall use the term <strong>photoprotein</strong> instead</li>
</ul>
<p>All photoproteins sense light by means of a <u>covalently attached light-absorbing chromophore</u>, which changes its shape in response to light and then induces a change in the protein’s conformation</p>
<p>The best-known plant photoproteins are the <strong>phytochromes</strong>, which are present in all plants and in some algae but are absent in animals. These are dimeric, cytoplasmic <strong>serine&#x2F;threonine kinases</strong>, which respond differentially and reversibly to <strong>red</strong> and <strong>far-red light</strong>:</p>
<ul>
<li>whereas red light usually activates the kinase activity of the phytochrome, <u>far-red light inactivates it</u>. When activated by red light, the phytochrome is thought to <u>phosphorylate itself</u> and then to phosphorylate one or more other proteins in the cell.</li>
</ul>
</blockquote>
<blockquote>
<p>Plants sense blue light using photoproteins of two other sorts, <strong>phototropin</strong>  and <strong>cryptochromes</strong></p>
<ul>
<li><strong>Phototropin</strong> is associated with the plasma membrane and is partly responsible for phototropism, the tendency of plants to grow toward light<ul>
<li>Phototropism occurs by directional cell elongation, which is stimulated by auxin,  but the links between phototropin and auxin are unknown.</li>
</ul>
</li>
<li><strong>Cryptochromes</strong> are <strong>flavoproteins</strong> that are sensitive to blue light. They are  structurally related to blue-light-sensitive enzymes called photolyases, which  are involved in the <strong>repair</strong> of ultraviolet-induced DNA damage in all organisms,  except most mammals</li>
</ul>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709201005556.png" srcset="/img/loading.gif" lazyload alt="image-20200709201005556"></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709201013039.png" srcset="/img/loading.gif" lazyload alt="image-20200709201013039"></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709201021300.png" srcset="/img/loading.gif" lazyload alt="image-20200709201021300"></p>

                
              </div>
            
            <hr/>
            <div>
              <div class="post-metas my-3">
  
    <div class="post-meta mr-3 d-flex align-items-center">
      <i class="iconfont icon-category"></i>
      

<span class="category-chains">
  
  
    
      <span class="category-chain">
        
  <a href="/categories/Academic-Notebook/" class="category-chain-item">Academic Notebook</a>
  
  
    <span>></span>
    
  <a href="/categories/Academic-Notebook/Cell-Biology/" class="category-chain-item">Cell Biology</a>
  
  

  

      </span>
    
  
</span>

    </div>
  
  
    <div class="post-meta">
      <i class="iconfont icon-tags"></i>
      
        <a href="/tags/Cell-Biology/" class="print-no-link">#Cell Biology</a>
      
    </div>
  
</div>


              
  

  <div class="license-box my-3">
    <div class="license-title">
      <div>L17 Cell communication via enzyme-coupled receptors</div>
      <div>https://zhenyumi.github.io/posts/1362cf09/</div>
    </div>
    <div class="license-meta">
      
        <div class="license-meta-item">
          <div>作者</div>
          <div>向海</div>
        </div>
      
      
        <div class="license-meta-item license-meta-date">
          <div>发布于</div>
          <div>2020年7月18日</div>
        </div>
      
      
      
        <div class="license-meta-item">
          <div>许可协议</div>
          <div>
            
              
              
                <a class="print-no-link" target="_blank" href="https://creativecommons.org/licenses/by/4.0/">
                  <span class="hint--top hint--rounded" aria-label="BY - 署名">
                    <i class="iconfont icon-cc-by"></i>
                  </span>
                </a>
              
            
          </div>
        </div>
      
    </div>
    <div class="license-icon iconfont"></div>
  </div>



              
                <div class="post-prevnext my-3">
                  <article class="post-prev col-6">
                    
                    
                      <a href="/posts/8983d1b1/" title="L18 Cell Division- Mitosis">
                        <i class="iconfont icon-arrowleft"></i>
                        <span class="hidden-mobile">L18 Cell Division- Mitosis</span>
                        <span class="visible-mobile">上一篇</span>
                      </a>
                    
                  </article>
                  <article class="post-next col-6">
                    
                    
                      <a href="/posts/1b141cf1/" title="L01 Chemical Logic of Metabolism">
                        <span class="hidden-mobile">L01 Chemical Logic of Metabolism</span>
                        <span class="visible-mobile">下一篇</span>
                        <i class="iconfont icon-arrowright"></i>
                      </a>
                    
                  </article>
                </div>
              
            </div>

            
          </article>
        </div>
      </div>
    </div>

    <div class="side-col d-none d-lg-block col-lg-2">
      
  <aside class="sidebar" style="margin-left: -1rem">
    <div id="toc">
  <p class="toc-header">
    <i class="iconfont icon-list"></i>
    <span>目录</span>
  </p>
  <div class="toc-body" id="toc-body"></div>
</div>



  </aside>


    </div>
  </div>
</div>





  



  



  



  



  


  
  









    

    
      <a id="scroll-top-button" aria-label="TOP" href="#" role="button">
        <i class="iconfont icon-arrowup" aria-hidden="true"></i>
      </a>
    

    
      <div class="modal fade" id="modalSearch" tabindex="-1" role="dialog" aria-labelledby="ModalLabel"
     aria-hidden="true">
  <div class="modal-dialog modal-dialog-scrollable modal-lg" role="document">
    <div class="modal-content">
      <div class="modal-header text-center">
        <h4 class="modal-title w-100 font-weight-bold">搜索</h4>
        <button type="button" id="local-search-close" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body mx-3">
        <div class="md-form mb-5">
          <input type="text" id="local-search-input" class="form-control validate">
          <label data-error="x" data-success="v" for="local-search-input">关键词</label>
        </div>
        <div class="list-group" id="local-search-result"></div>
      </div>
    </div>
  </div>
</div>

    

    
  </main>

  <footer>
    <div class="footer-inner">
  
    <div class="footer-content">
       <a href="https://hexo.io" target="_blank" rel="nofollow noopener"><span>Hexo</span></a> <i class="iconfont icon-love"></i> <a href="https://github.com/fluid-dev/hexo-theme-fluid" target="_blank" rel="nofollow noopener"><span>Fluid</span></a> 
    </div>
  
  
  
</div>

  </footer>

  <!-- Scripts -->
  
  <script  src="https://lib.baomitu.com/nprogress/0.2.0/nprogress.min.js" ></script>
  <link  rel="stylesheet" href="https://lib.baomitu.com/nprogress/0.2.0/nprogress.min.css" />

  <script>
    NProgress.configure({"showSpinner":false,"trickleSpeed":100})
    NProgress.start()
    window.addEventListener('load', function() {
      NProgress.done();
    })
  </script>


<script  src="https://lib.baomitu.com/jquery/3.6.4/jquery.min.js" ></script>
<script  src="https://lib.baomitu.com/twitter-bootstrap/4.6.1/js/bootstrap.min.js" ></script>
<script  src="/js/events.js" ></script>
<script  src="/js/plugins.js" ></script>





  
    <script  src="/js/img-lazyload.js" ></script>
  




  
<script>
  Fluid.utils.createScript('https://lib.baomitu.com/tocbot/4.20.1/tocbot.min.js', function() {
    var toc = jQuery('#toc');
    if (toc.length === 0 || !window.tocbot) { return; }
    var boardCtn = jQuery('#board-ctn');
    var boardTop = boardCtn.offset().top;

    window.tocbot.init(Object.assign({
      tocSelector     : '#toc-body',
      contentSelector : '.markdown-body',
      linkClass       : 'tocbot-link',
      activeLinkClass : 'tocbot-active-link',
      listClass       : 'tocbot-list',
      isCollapsedClass: 'tocbot-is-collapsed',
      collapsibleClass: 'tocbot-is-collapsible',
      scrollSmooth    : true,
      includeTitleTags: true,
      headingsOffset  : -boardTop,
    }, CONFIG.toc));
    if (toc.find('.toc-list-item').length > 0) {
      toc.css('visibility', 'visible');
    }

    Fluid.events.registerRefreshCallback(function() {
      if ('tocbot' in window) {
        tocbot.refresh();
        var toc = jQuery('#toc');
        if (toc.length === 0 || !tocbot) {
          return;
        }
        if (toc.find('.toc-list-item').length > 0) {
          toc.css('visibility', 'visible');
        }
      }
    });
  });
</script>


  <script src=https://lib.baomitu.com/clipboard.js/2.0.11/clipboard.min.js></script>

  <script>Fluid.plugins.codeWidget();</script>


  
<script>
  Fluid.utils.createScript('https://lib.baomitu.com/anchor-js/5.0.0/anchor.min.js', function() {
    window.anchors.options = {
      placement: CONFIG.anchorjs.placement,
      visible  : CONFIG.anchorjs.visible
    };
    if (CONFIG.anchorjs.icon) {
      window.anchors.options.icon = CONFIG.anchorjs.icon;
    }
    var el = (CONFIG.anchorjs.element || 'h1,h2,h3,h4,h5,h6').split(',');
    var res = [];
    for (var item of el) {
      res.push('.markdown-body > ' + item.trim());
    }
    if (CONFIG.anchorjs.placement === 'left') {
      window.anchors.options.class = 'anchorjs-link-left';
    }
    window.anchors.add(res.join(', '));

    Fluid.events.registerRefreshCallback(function() {
      if ('anchors' in window) {
        anchors.removeAll();
        var el = (CONFIG.anchorjs.element || 'h1,h2,h3,h4,h5,h6').split(',');
        var res = [];
        for (var item of el) {
          res.push('.markdown-body > ' + item.trim());
        }
        if (CONFIG.anchorjs.placement === 'left') {
          anchors.options.class = 'anchorjs-link-left';
        }
        anchors.add(res.join(', '));
      }
    });
  });
</script>


  
<script>
  Fluid.utils.createScript('https://lib.baomitu.com/fancybox/3.5.7/jquery.fancybox.min.js', function() {
    Fluid.plugins.fancyBox();
  });
</script>


  <script>Fluid.plugins.imageCaption();</script>

  
      <script>
        if (!window.MathJax) {
          window.MathJax = {
            tex    : {
              inlineMath: { '[+]': [['$', '$']] }
            },
            loader : {
              load: ['ui/lazy']
            },
            options: {
              renderActions: {
                insertedScript: [200, () => {
                  document.querySelectorAll('mjx-container').forEach(node => {
                    let target = node.parentNode;
                    if (target.nodeName.toLowerCase() === 'li') {
                      target.parentNode.classList.add('has-jax');
                    }
                  });
                }, '', false]
              }
            }
          };
        } else {
          MathJax.startup.document.state(0);
          MathJax.texReset();
          MathJax.typeset();
          MathJax.typesetPromise();
        }

        Fluid.events.registerRefreshCallback(function() {
          if ('MathJax' in window && MathJax.startup.document && typeof MathJax.startup.document.state === 'function') {
            MathJax.startup.document.state(0);
            MathJax.texReset();
            MathJax.typeset();
            MathJax.typesetPromise();
          }
        });
      </script>
    

  <script  src="https://lib.baomitu.com/mathjax/3.2.2/es5/tex-mml-chtml.js" ></script>

  <script  src="/js/local-search.js" ></script>





<!-- 主题的启动项，将它保持在最底部 -->
<!-- the boot of the theme, keep it at the bottom -->
<script  src="/js/boot.js" ></script>


  

  <noscript>
    <div class="noscript-warning">博客在允许 JavaScript 运行的环境下浏览效果更佳</div>
  </noscript>
</body>
</html>
