---
title: L06 DNA Replication
tags:
  -  Molecular Biology
categories:
  -  Academic Notebook
  -  Molecular Biology
author: 向海
index_img: >-
 https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Molecular%20Biology/20200724_%EC%BA%AC%EB%A3%A8%ED%83%88%EC%B6%9C.png
banner_img: >-
 https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Molecular%20Biology/20200724_Sunlight%281%29.jpg
top: false
math: true
comments: true
hide: false
excerpt: No such things
abbrlink: 808992df
date: 2020-07-24 17:20:09
updated: 2020-07-24 17:20:09
---

# 一、DNA Replication

## Requirement

DNA replication requires: 

1. Unraveling or relaxation of the supercoiled structure. 
2. Initiation of DNA replication at special sites. (Primer)
3. Separation of the two strands from each other to create a single-stranded template. 
4. Replication of the two strands which run in an anti-parallel direction.

## Segregation of Chromosome

### 1. Chromosome is a segregation device

A eukaryotic chromosome is held on the mitotic **spindle** by the attachment of microtubules to the **kinetochore** that forms in its centromeric region. 

**microtubule organizing center (MTOC) –** A region from which microtubules emanate（发散）. 

+ In animal cells the centrosome is the major microtubule organizing center, but <u>plants use the nuclear envelope itself as the MTOCs.</u> 

### 2. Semiconservative Nature (Meselson-Stahl Experiment)

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105101344031.png" alt="image-20191105101344031" style="zoom: 33%;" />

The Meselson-Stahl Experiment proves the semi-conservative replication of DNA

Results matched prediction for semi-conservative replication; 

+ e.g. mixed strands were created (50% heavy, 50% light).

#### The Experiment Procedure

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191226150714743.png" alt="image-20191226150714743" style="zoom:100%;" />
E. coli cells grown in ,heavy" nitrogen to label cellular DNA. ('H' in
figure)

Transferred to normal "light" nitrogen, grown, and then DNA separated by density-gradient centrifugation.

If replication were 'conservative (at left), then after replication, only old and new types are present

If semi-conservative, after replication, mixed strands would be created (50% heavy, 50% light).

### 3. Theta Structures introduced in the circular DNA Replication

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105102035448.png" alt="image-20191105102035448" style="zoom:67%;" />

In the absence of swiveling（转动的）, the unwinding（解退） DNA in one region would cause tightening or over-winding of the other strand. 

Relaxation of this stress is performed by <u>topoisomerase enzymes.</u> 

By cutting one strand of DNA, these enzymes allow one DNA to swivel around the other.

## Direction and Progress

### 1. Direction

#### Direction for one strand

The DNA replication has a direction from <u>5’ to 3’</u>

#### Direction for the whole double-strand DNA

所以复制从一点开始，对于一条double-strand DNA来说同时向两边扩展

**The replicating strands form a “bubble”.** 

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105110349364.png" alt="image-20191105110349364" style="zoom: 50%;" />

The two sites of replication are proceeding identically, with a “fork” formed from the parent strand to the new parent/daughter hybrid strands. 

**Bidirectional（双向的） strand growth is universal to** **prokaryotes and eukaryotes.** 



### 2. dNTPs

The template stand directs which of the four <u>dNTPs are added</u> in the extension of the 3’ end.

### 3. Primes

DNA is formed from dNTP monomers; synthesis proceeds in the 5’ to 3’ direction. DNA polymerases <u>require a primers such as a short piece of RNA.</u> 

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105104212758.png" alt="image-20191105104212758" style="zoom: 50%;" />

### 4. Semidiscontinuous DNA Replication

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191226152441590.png" alt="image-20191226152441590" style="zoom:67%;" />

DNA is formed from dNTP monomers; synthesis proceeds in the 5' to 3' direction.

DNA polymerases require a primers such as a short piece of RNA.

### 5. DNA Replication Forks

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191226152602114.png" alt="image-20191226152602114" style="zoom: 67%;" />

DNA replication forks occur at many locations, these finally merge to create the completely replicated molecules

### 6. Bidirectional（双向） Replication

The replicating strands form a “bubble”. 

The two sites of replication are proceeding identically, with a “fork” formed from the parent strand to the new parent/daughter hybrid strands.

Bidirectional strand growth is universal to prokaryotes and eukaryotes. 

### 7. Replication Initation Point Mapping

Newly synthesized DNA is used as template.

Primer extension stops at DNA-RNA junctions on the nascent strand because the DNA polymerase included in the reaction cannot use RNA as a template.

TP = transition point, location of shift from discontinuous to continuous synthesis.

## DNA polymerase

DNA polymerases often have a 3′–5′ exonuclease activity that is used to <u>excise（切除）</u> incorrectly paired bases. 

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105102952533.png" alt="image-20191105102952533" style="zoom: 67%;" />

Bacterial DNA polymerases scrutinize the base pair at the end of the growing chain and excise the nucleotide added in the case of a misfit. 

### 1. holoenzyme 全酶

The DNA polymerase holoenzyme consists of subcomplexes

A **clamp loader** places the processivity subunits on DNA, where they form a circular **clamp** around the nucleic acid. 

One catalytic core is associated with each template strand. 

## Untwisting of Supercoiled DNA

The untwisting of DNA is needed for DNA replication.

This localized unwinding creates tension further along the molecule, particularly in a circular DNA molecule in which the tension cannot be released at the ends.

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105103948076.png" alt="image-20191105103948076" style="zoom: 50%;" />

## Characteristic Structures or Molecules

### 1. Origins of Replication 

Origins of Replication differs in Prokaryotic and Eukaryotic cells

+ Single origin for bacterial chromosomes 

+ Many origins for eukaryotic chromosomes, multiple replication forks.

### 2. Origin Reading Complex, ORC

Histone removal at the site of replication to allow access to DNA.

Histone deposition, with nucleosomes reassembled on nascent（初期的，开始存在的） DNA <u>via interaction with chromatin assembly factor.</u> 

Prereplication complex formation at the origin by the association of the origin recognition complex(ORC).

#### Replication licensing

Once the ORC binds, additional proteins are recruited（ORC结合后，会吸引结合更多的蛋白质）

#### The Assembly of Prereplication Complex

ATP hydrolysis by the origin recognition complex (ORC) stimulates prereplication complex assembly, which is inhibited by ATP-γS-bound ORC (a non-hydrolyzable form of ATP).

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105111343545.png" alt="image-20191105111343545" style="zoom: 50%;" />

### 3. Helicase（解旋酶） and Topoisomerase

1. Helicase <u>initiates local melting/unwinding of DNA</u> at “origin of replication” (ori). 
 + Single-strand binding proteins stabilize strands
 + Primase (RNA polymerase) forms short RNA primer.
 + DNA polymerase uses RNA primer to continue synthesis (switch of polymerases).
2. Topoisomerase <u>relieves supercoils</u>.

### 4. Primase(RNA polymerase) and RNA primer

Single-strand binding proteins stabilize strands 

1. Primase (RNA polymerase) <u>forms short RNA primer.</u> 
2. DNA polymerase <u>uses RNA primer to continue synthesis</u> (switch of polymerases).

### 5. Lagging Strand – Leading Strand and Okazaki Fragment

On lagging strand, ‘Okazaki fragments’ are started every few hundred bases. 

When one Okazaki fragment meets RNA primer of next strand, RNA is removed and DNA strands are ligated together by ‘ligase’. 

Filing in of gaps left by removal of RNA primers 

+ Joining of Okazaki fragments by **DNA ligase I** (DNA 连接酶 I)
+ Human DNA Ligase I encircles（环绕）DNA during ligation

### 6. Proliferating cell nuclear antigen 增殖细胞核抗原（PCNA）

“Proliferating cell nuclear antigen” (PCNA) originally identified in dividing cells. Interacts with many other proteins. 

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105111648152.png" alt="image-20191105111648152" style="zoom: 67%;" />

PCNA is a <u>“sliding clamp”</u> that encircles DNA and allows proteins to stay adhered to DNA during processing.

### 7. DNA Ligase

+ Joining of Okazaki fragments by **DNA ligase I** (DNA 连接酶 I)
+ Human DNA Ligase I encircles（环绕）DNA during ligation

The ends of linear DNA are a problem for replication

Special arrangements must be made to replicate the DNA strand with a 5’ end

#### DNA End Replication Problem

Because of the requirement for a primer on the lagging strand, when the final primer is removed, this leaves an 8-12 nt region uncopied at the 5’ end. 

Terminal proteins enable initiation at the ends of <u>viral DNAs</u>



## Activation of DNA replication

### 1. CDK1

Origin of replication ativated once during the cell cycle

DNA synthesis is restricted to only the S (‘synthesis’) phase of the cell cycle. 

Replication is therefore carefully regulated.

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191226153938217.png" alt="image-20191226153938217" style="zoom: 67%;" />

### 2. Assembly of The Prereplication Complex

The data suggest that ATP hydrolysis by the origin recognition complex (ORC) stimulates prereplication complex assembly, which is inhibited by AFR-vS-bound ORC (a non-hydrolyzable form of AFP).

## Assembly of Nucleosomes

Nucleosome assembly after DNA replication

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105112442136.png" alt="image-20191105112442136" style="zoom:67%;" />

Replication fork passage displaces histone octamers from DNA. 

<u>Histones may be added back to the DNA strands as either entire new tetramers, or mixed pairs of old and new dimers.</u> 

## Simple Biological System – Rolling Cycle

Simple Biological system use “**Rolling Circle**” or “**Covalent extension**” Replication

This produces large amounts of DNA, which is a “concatamer（串联体）” of many molecules of the same template. 

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105112859365.png" alt="image-20191105112859365" style="zoom:67%;" />

## Telomere

A telomere consists of a simple repeat where a C+A-rich strand has the sequence $C_{>1}(A/T)_{1-4}$

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191226155808125.png" alt="image-20191226155808125" style="zoom:100%;" />
A typical telomere has a simple repeating structure with a G-T-rich strand that extends beyond the C-A-rich strand. 

Telomere have simple repeat sequences which seal the chromosome ends

Mutation in telomerase causes telomeres to shorten in each cell division. <u>Eventual loss of the telomere causes chromosome breaks and rearrangements.</u>

The **telomere** is required for the stability of the chromosome end.

### 1. Telomerase

Telomerase puts the ends of eukaryotic chromosomes

+ **Telomerase** uses the 3′-OH of the G+T telomeric strand to prime synthesis of tandem TTGGGG repeats. 

+ The RNA component of telomerase has a sequence that pairs with the C+A-rich repeats. 

+ One of the protein subunits is a reverse transcriptase that uses the RNA as template to synthesis the G+T-rich sequence. 

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105114151585.png" alt="image-20191105114151585" style="zoom: 67%;" />



RNA in telomerase pairs with DNA and polymerase extends DNA at 3’ end, which can then be filled in either by lagging strand synthesis or T-loop formation.

### 2. TRF2

The protein TRF2 catalyzes a reaction in which the 3′ repeating unit of the G+T-rich strand forms a loop by displacing its homolog in an upstream region of the telomere. 

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105114438065.png" alt="image-20191105114438065" style="zoom:50%;" />

The 3' single-stranded end of the telomere (TTAGGG)n displaces the homologous repeats from duplex DNA to form a t-loop. The reaction is catalyzed byTRF2.

### 3. The length of telomeres

The length of the telomere is determined by concentration of the <u>POT1 (Protection of Telomeres1) protein bound to the DNA.</u> 

This is sensed by the telomerase and the level either represses or activates telomerase activity.

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191226160554553.png" alt="image-20191226160554553" style="zoom:67%;" />

### 4. The existence of telomerases

Telomerase is expressed in actively dividing cells and is not expressed in differentiated cells. 

Loss of telomeres results in senescence（衰老）. 

Escape from senescence can occur if telomerase is reactivated, or via unequal homologous recombination to restore telomeres. 

# 二、DNA Proofreading

## DNA polymerase I

DNA polymerase I has a 3’ to 5’ exonuclease activity to remove erroneous bases. The normal replication direction is 5’ to 3’. 

The DNA polymerase I functions usually from 3’ to 5’

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105111905318.png" alt="image-20191105111905318" style="zoom: 67%;" />

DNA polymerase I has a 3' to 5' exonuclease activity to remove erroneous bases. The normal replication direction is 5' to 3’.