

<!DOCTYPE html>
<html lang="zh-CN" >



<head>
  <meta charset="UTF-8">

  <link rel="apple-touch-icon" sizes="76x76" href="/img/Logo.png">
  <link rel="icon" href="/img/Logo.png">
  

  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=5.0, shrink-to-fit=no">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  
  <meta name="theme-color" content="#2f4154">
  <meta name="author" content="向海">
  <meta name="keywords" content="">
  
    <meta name="description" content="一、Intracellular compartmentation All Eukaryotic Cells Have the Same Basic Set of Membrane-enclosed Organelles Intracellular membrane system increases membrane area to host biochemical reactions, intra">
<meta property="og:type" content="article">
<meta property="og:title" content="L9 Intracellular transport I - Nucleus, Mitochondria， Chloroplast, Peroxisome">
<meta property="og:url" content="https://zhenyumi.github.io/posts/12aa050d/index.html">
<meta property="og:site_name" content="鹘横海">
<meta property="og:description" content="一、Intracellular compartmentation All Eukaryotic Cells Have the Same Basic Set of Membrane-enclosed Organelles Intracellular membrane system increases membrane area to host biochemical reactions, intra">
<meta property="og:locale" content="zh_CN">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Cell_Biology/Cell_Biology_01.jpg">
<meta property="article:published_time" content="2020-07-02T11:32:00.000Z">
<meta property="article:modified_time" content="2020-07-17T13:36:00.000Z">
<meta property="article:author" content="向海">
<meta property="article:tag" content="Cell Biology">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Cell_Biology/Cell_Biology_01.jpg">
  
  
  
  <title>L9 Intracellular transport I - Nucleus, Mitochondria， Chloroplast, Peroxisome - 鹘横海</title>

  <link  rel="stylesheet" href="https://lib.baomitu.com/twitter-bootstrap/4.6.1/css/bootstrap.min.css" />



  <link  rel="stylesheet" href="https://lib.baomitu.com/github-markdown-css/4.0.0/github-markdown.min.css" />

  <link  rel="stylesheet" href="https://lib.baomitu.com/hint.css/2.7.0/hint.min.css" />

  <link  rel="stylesheet" href="https://lib.baomitu.com/fancybox/3.5.7/jquery.fancybox.min.css" />



<!-- 主题依赖的图标库，不要自行修改 -->
<!-- Do not modify the link that theme dependent icons -->

<link rel="stylesheet" href="//at.alicdn.com/t/c/font_1749284_5i9bdhy70f8.css">



<link rel="stylesheet" href="//at.alicdn.com/t/c/font_1736178_k526ubmyhba.css">


<link  rel="stylesheet" href="/css/main.css" />


  <link id="highlight-css" rel="stylesheet" href="/css/highlight.css" />
  




  <script id="fluid-configs">
    var Fluid = window.Fluid || {};
    Fluid.ctx = Object.assign({}, Fluid.ctx)
    var CONFIG = {"hostname":"zhenyumi.github.io","root":"/","version":"1.9.8","typing":{"enable":true,"typeSpeed":70,"cursorChar":"_","loop":false,"scope":["home"]},"anchorjs":{"enable":true,"element":"h1,h2,h3,h4,h5,h6","placement":"left","visible":"hover","icon":""},"progressbar":{"enable":true,"height_px":3,"color":"#29d","options":{"showSpinner":false,"trickleSpeed":100}},"code_language":{"enable":true,"default":"TEXT"},"copy_btn":true,"image_caption":{"enable":true},"image_zoom":{"enable":true,"img_url_replace":["",""]},"toc":{"enable":true,"placement":"right","headingSelector":"h1,h2,h3,h4,h5,h6","collapseDepth":0},"lazyload":{"enable":true,"loading_img":"/img/loading.gif","onlypost":false,"offset_factor":2},"web_analytics":{"enable":false,"follow_dnt":true,"baidu":null,"google":{"measurement_id":null},"tencent":{"sid":null,"cid":null},"leancloud":{"app_id":null,"app_key":null,"server_url":null,"path":"window.location.pathname","ignore_local":false},"umami":{"src":null,"website_id":null,"domains":null,"start_time":"2024-01-01T00:00:00.000Z","token":null,"api_server":null}},"search_path":"/local-search.xml","include_content_in_search":true};

    if (CONFIG.web_analytics.follow_dnt) {
      var dntVal = navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack;
      Fluid.ctx.dnt = dntVal && (dntVal.startsWith('1') || dntVal.startsWith('yes') || dntVal.startsWith('on'));
    }
  </script>
  <script  src="/js/utils.js" ></script>
  <script  src="/js/color-schema.js" ></script>
  


  
<meta name="generator" content="Hexo 7.0.0"></head>


<body>
  

  <header>
    

<div class="header-inner" style="height: 70vh;">
  <nav id="navbar" class="navbar fixed-top  navbar-expand-lg navbar-dark scrolling-navbar">
  <div class="container">
    <a class="navbar-brand" href="/">
      <strong>鹘横海</strong>
    </a>

    <button id="navbar-toggler-btn" class="navbar-toggler" type="button" data-toggle="collapse"
            data-target="#navbarSupportedContent"
            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <div class="animated-icon"><span></span><span></span><span></span></div>
    </button>

    <!-- Collapsible content -->
    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav ml-auto text-center">
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/" target="_self">
                <i class="iconfont icon-home-fill"></i>
                <span>首页</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/archives/" target="_self">
                <i class="iconfont icon-archive-fill"></i>
                <span>归档</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/categories/" target="_self">
                <i class="iconfont icon-category-fill"></i>
                <span>分类</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/tags/" target="_self">
                <i class="iconfont icon-tags-fill"></i>
                <span>标签</span>
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/about/" target="_self">
                <i class="iconfont icon-user-fill"></i>
                <span>关于</span>
              </a>
            </li>
          
        
        
          <li class="nav-item" id="search-btn">
            <a class="nav-link" target="_self" href="javascript:;" data-toggle="modal" data-target="#modalSearch" aria-label="Search">
              <i class="iconfont icon-search"></i>
            </a>
          </li>
          
        
        
      </ul>
    </div>
  </div>
</nav>

  

<div id="banner" class="banner" parallax=true
     style="background: url('https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Cell_Biology/Cell_Biology_02.png') no-repeat center center; background-size: cover;">
  <div class="full-bg-img">
    <div class="mask flex-center" style="background-color: rgba(0, 0, 0, 0.3)">
      <div class="banner-text text-center fade-in-up">
        <div class="h2">
          
            <span id="subtitle">L9 Intracellular transport I - Nucleus, Mitochondria， Chloroplast, Peroxisome</span>
          
        </div>

        
          
  <div class="mt-3">
    
    
      <span class="post-meta">
        <i class="iconfont icon-date-fill" aria-hidden="true"></i>
        <time datetime="2020-07-02 19:32" pubdate>
          2020年7月2日 晚上
        </time>
      </span>
    
  </div>

  <div class="mt-1">
    
      <span class="post-meta mr-2">
        <i class="iconfont icon-chart"></i>
        
          993 字
        
      </span>
    

    
      <span class="post-meta mr-2">
        <i class="iconfont icon-clock-fill"></i>
        
        
        
          9 分钟
        
      </span>
    

    
    
  </div>


        
      </div>

      
    </div>
  </div>
</div>

</div>

  </header>

  <main>
    
      

<div class="container-fluid nopadding-x">
  <div class="row nomargin-x">
    <div class="side-col d-none d-lg-block col-lg-2">
      

    </div>

    <div class="col-lg-8 nopadding-x-md">
      <div class="container nopadding-x-md" id="board-ctn">
        <div id="board">
          <article class="post-content mx-auto">
            <h1 id="seo-header">L9 Intracellular transport I - Nucleus, Mitochondria， Chloroplast, Peroxisome</h1>
            
            
              <div class="markdown-body">
                
                <h1 id="一、Intracellular-compartmentation"><a href="#一、Intracellular-compartmentation" class="headerlink" title="一、Intracellular compartmentation"></a>一、Intracellular compartmentation</h1><blockquote>
<p>All Eukaryotic Cells Have the Same Basic Set of Membrane-enclosed Organelles</p>
<p>Intracellular membrane system increases membrane area to host biochemical reactions, intracellular membrane systems form enclosed compartments that are separate from the cytosol, thus creating functionally specialized aqueous spaces within the cell.</p>
</blockquote>
<p>Relative volumes for each compartment in a liver cell: </p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315120629782.png" srcset="/img/loading.gif" lazyload alt="image-20200315120629782" style="zoom:67%;" />

<ul>
<li>Volume of compartments<ul>
<li>represents about <u>50%</u> of the total volume of the cell</li>
</ul>
</li>
<li>Almost half of that of that are mitochondria (in this liver cell)</li>
<li>Each organelle contains its unique set of proteins that are characteristic of its distinct functions.</li>
</ul>
<p>Different cell has different compartment volume ratio.</p>
<blockquote>
<p>The size, shape, composition, and location are all important and regulated features of these organelles that ultimately contribute to the organelle’s function.</p>
<p>The <strong>eukaryotic cell</strong> has a <u>much smaller ratio of surface area to volume,</u> and its plasma membrane therefore presumably has too small an area to sustain the many vital functions that membranes perform.</p>
</blockquote>
<h2 id="Origin-of-internal-membrane-and-membrane-bound-organelles"><a href="#Origin-of-internal-membrane-and-membrane-bound-organelles" class="headerlink" title="Origin of internal membrane and membrane-bound organelles"></a>Origin of internal membrane and membrane-bound organelles</h2><blockquote>
<p>The precursors of the first eukaryotic cells are thought to have been relatively simple cells that—like <u>most bacterial and archaeal cells—have a plasma membrane but no internal membranes</u>. The plasma  membrane in such cells provides all membrane-dependent functions, including  the <u>pumping of ions, ATP synthesis, protein secretion, and lipid synthesis.</u></p>
<p>A hypothetical scheme for how the first eukaryotic cells, with a nucleus and ER, might have evolved by the invagination  and pinching off of the plasma membrane of an ancestral cell is illustrated in Figure 12–3.This process would create membrane-enclosed organelles with an  interior or <strong>lumen</strong> that is <strong>topologically equivalent</strong>.</p>
<ul>
<li><p>Compartments are said to be <strong>topologically equivalent</strong> if they can  <u>communicate with one another</u></p>
</li>
<li><p>We shall see that this topological relationship holds for all of the organelles involved in the secretory and endocytic pathways, including the <strong>ER</strong>, <strong>Golgi apparatus</strong>, <strong>endosomes</strong>, <strong>lysosomes</strong>, and <strong>peroxisomes</strong>. We can therefore think of all of these organelles as members of the <strong>same topologically equivalent compartment</strong></p>
</li>
<li><p>Because this double membrane of nucleus is penetrated by nuclear pore complexes, the <strong>nuclear compartment</strong> is  \topologically equivalent to the <strong>cytosol</strong>.</p>
</li>
<li><p>In  contrast, the lumen of the <strong>ER</strong> is continuous  with the space between the inner and outer nuclear membranes, and it is topologically  equivalent to the <strong>extracellular space</strong></p>
</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200701162557676.png" srcset="/img/loading.gif" lazyload alt="image-20200701162557676"></p>
</blockquote>
<ol>
<li><p>Endogenous (e.g. invagination of plasma membrane)</p>
</li>
<li><p>Symbiotic (外源的)</p>
</li>
</ol>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315121122308.png" srcset="/img/loading.gif" lazyload alt="image-20200315121122308" style="zoom: 80%;" />



<blockquote>
<p>Mitochondria and plastids differ from the other membrane-enclosed organelles because they contain their own genomes</p>
<p>The nature of these genomes, and the close resemblance of the proteins in these organelles to those in some present-day bacteria, strongly suggest that mitochondria and plastids evolved from bacteria that were engulfed by other cells with which they initially lived in <strong>symbiosis</strong></p>
</blockquote>
<blockquote>
<p>Most Organelles Cannot Be Constructed <em>De Novo</em>（再次）: They Require Information in the Organelle Itself</p>
<ul>
<li>Information in the form of at least one distinct protein that preexists in the organelle membrane is also required</li>
<li>this information is passed from parent cell to daughter cells in the form of the organelle itself.</li>
</ul>
</blockquote>
<h2 id="Proteins-Transport-Pathway"><a href="#Proteins-Transport-Pathway" class="headerlink" title="Proteins Transport Pathway"></a>Proteins Transport Pathway</h2><blockquote>
<p>Proteins Can Move Between Compartments in Different Ways</p>
</blockquote>
<p>Proteins are imported into organelles from cytosol or transported between organelles</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315121339123.png" srcset="/img/loading.gif" lazyload alt="image-20200315121339123" style="zoom:67%;" />

<p>Almost all proteins are synthesized in the cytosol and are transported later to their final destination due to their specific <strong>sorting signals</strong></p>
<ul>
<li>Exception: some proteins from mitochondria and plastids!</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123121491.png" srcset="/img/loading.gif" lazyload alt="image-20200315123121491"></p>
<h3 id="1-Protein-sorting-mechanisms-of-import"><a href="#1-Protein-sorting-mechanisms-of-import" class="headerlink" title="1. Protein sorting: mechanisms of import"></a>1. Protein sorting: mechanisms of import</h3><blockquote>
<p>proteins are transported into other membrane-enclosed organelles <u>only after their synthesis is complete</u>, they are transported into the ER as they are synthesized.</p>
</blockquote>
<p><u>3 types of mechanisms</u> for <strong>macromolecule transport</strong> to cellular compartments:</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315121521893.png" srcset="/img/loading.gif" lazyload alt="image-20200315121521893" style="zoom: 50%;" />

<ol>
<li><p><strong>“gated” transport</strong> (cytosal $\larr \rarr$ Nucleus) $\rarr$ “nuclear import&#x2F;export signal (Lecture 9)</p>
<blockquote>
<p>In <strong>gated transport</strong>, proteins and RNA molecules move between the cytosol and the nucleus through nuclear pore complexes in the nuclear envelope.  </p>
<p>The nuclear pore complexes function as <strong>selective gates</strong> that support the active transport of specific macromolecules and macromolecular assemblies between the two topologically equivalent spaces, although they also allow <u>free diffusion</u> of smaller molecules.</p>
</blockquote>
</li>
<li><p><strong>Protein translocation</strong> (unfolded protein “snake” through transmembrane translocators) $\rarr$ signal peptides, sorting signals&#x2F;motifs (Lecture 9 and 10)</p>
<blockquote>
<p>In protein translocation, <strong>transmembrane protein translocators</strong> directly transport specific proteins across a membrane from the cytosol into a  space that is <strong>topologically distinct.</strong> </p>
<p>The transported protein molecule usually must <strong>unfold</strong> to snake through the translocator. The initial transport of selected proteins from the cytosol into the ER lumen or mitochondria,  for example, occurs in this way. </p>
<ul>
<li><strong>Integral membrane proteins</strong> often use the same translocators but translocate only partially across the membrane, so  that the protein becomes embedded in the lipid bilayer.</li>
</ul>
</blockquote>
</li>
<li><p><strong>Vesicular transport</strong> $\rarr$sorting signals&#x2F;motifs (Lecture 11)</p>
<blockquote>
<p>In vesicular transport, membrane-enclosed transport intermediates— which may be small, spherical transport vesicles or larger, irregularly shaped organelle fragments—ferry proteins <strong>from one topologically equivalent compartment to another</strong>. </p>
<p>The transport vesicles and fragments become  loaded with a cargo of molecules derived from the lumen of one compartment as they bud and pinch off from its membrane; they discharge their cargo into a second compartment by fusing with the membrane enclosing that compartment (Figure 12–6). </p>
<ul>
<li>The transfer of soluble proteins from the ER to the Golgi apparatus, for example, occurs in this way. Because the transported proteins do not cross a membrane, <u>vesicular transport can move proteins only between compartments that are <strong>topologically equivalent</strong></u></li>
</ul>
</blockquote>
</li>
</ol>
<p>Principles:</p>
<ul>
<li>Cargo recognition depends on sorting signals (signal sequences).</li>
<li>Cargo transportation requires transport machinery.</li>
<li><u>Protein without any kind of sorting signal remain in the cytosol.</u></li>
</ul>
<h1 id="二、Signal-sequences-and-targeted-transport"><a href="#二、Signal-sequences-and-targeted-transport" class="headerlink" title="二、Signal sequences and targeted transport"></a>二、Signal sequences and targeted transport</h1><h2 id="Signal-sequences"><a href="#Signal-sequences" class="headerlink" title="Signal sequences"></a>Signal sequences</h2><h4 id="Definition"><a href="#Definition" class="headerlink" title="Definition"></a>Definition</h4><p><strong>Specific sorting signals</strong>: a stretch of amino acid sequence (typically 15-60 aa) which indicate where to put this protein.</p>
<blockquote>
<p><u>Such signal sequences are often found at the N-terminus of the polypeptide chain</u>, and in many cases specialized <strong>signal peptidases</strong> remove the signal sequence from the finished protein once the sorting process is complete</p>
<p>Signal sequences can also  be internal stretches of amino acids, which remain part of the protein.</p>
<p><u>Sorting signals can also be composed of multiple internal amino acid sequences that form a specific <strong>three-dimensional arrangement of atoms</strong> on the protein’s surface</u>; such <strong>signal patches</strong> are sometimes used for nuclear import and in vesicular transport.</p>
</blockquote>
<ul>
<li>Often found at the <strong>N-terminus</strong> of the polypeptide chain</li>
<li>In many cases can be removed by signal peptidases upon “arrival” (i. e. the sorting process is complete)</li>
<li>Signal patches: a specific 3-dimensional arrangement of atoms on protein’s surface</li>
</ul>
<blockquote>
<p>Signal sequences are recognized by <strong>complementary sorting receptors</strong> that guide proteins to their appropriate destination, where the receptors unload their cargo.</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315121935467.png" srcset="/img/loading.gif" lazyload alt="image-20200315121935467" style="zoom:80%;" />

<p>Protein without any kind of sorting signal remain in the cytosol.</p>
<h4 id="Organization-arrangement-of-sorting-signals"><a href="#Organization-arrangement-of-sorting-signals" class="headerlink" title="Organization&#x2F;arrangement of sorting signals"></a>Organization&#x2F;arrangement of sorting signals</h4><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122008515.png" srcset="/img/loading.gif" lazyload alt="image-20200315122008515"></p>
<p>图示的signal sequences在<strong>大部分情况是成立</strong>的，不代表具有相应的sequence就一定确定了转运方向，同样，没有图示的序列也不代表一定不是该转运方向。</p>
<h1 id="三、Nuclear-mitochondria-chloroplast-and-peroxisome-import"><a href="#三、Nuclear-mitochondria-chloroplast-and-peroxisome-import" class="headerlink" title="三、Nuclear, mitochondria, chloroplast and peroxisome import"></a>三、Nuclear, mitochondria, chloroplast and peroxisome import</h1><h2 id="Nucleus-cytosol-transport"><a href="#Nucleus-cytosol-transport" class="headerlink" title="Nucleus-cytosol transport"></a>Nucleus-cytosol transport</h2><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122155352.png" srcset="/img/loading.gif" lazyload alt="image-20200315122155352" style="zoom:80%;" />

<p>Nuclear envelope is a double lipid bilayer with supporting cytoskeleton and nuclear pores</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122213167.png" srcset="/img/loading.gif" lazyload alt="image-20200315122213167" style="zoom:67%;" />

<p>The Nuclear Lamina 提供一定的支撑作用</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122252978.png" srcset="/img/loading.gif" lazyload alt="image-20200315122252978" style="zoom:80%;" />

<p>Phosphorylation of lamins leads to disassembly of nuclear envelope</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122319288.png" srcset="/img/loading.gif" lazyload alt="image-20200315122319288" style="zoom:67%;" />

<h2 id="Nuclear-import-and-export"><a href="#Nuclear-import-and-export" class="headerlink" title="Nuclear import and export"></a>Nuclear import and export</h2><h3 id="1-Structural-components-of-nuclear-membrane-envelope"><a href="#1-Structural-components-of-nuclear-membrane-envelope" class="headerlink" title="1. Structural components of nuclear membrane&#x2F;envelope"></a>1. Structural components of nuclear membrane&#x2F;envelope</h3><p>Inner nuclear membrane proteins that bind to chromosome-binding proteins and nuclear lamina.</p>
<blockquote>
<p>During Mitosis the Nuclear Envelope Disassembles</p>
<p>The <strong>nuclear lamina</strong>, located on the nuclear side of the inner nuclear membrane,  is a meshwork of interconnected protein subunits called <strong>nuclear lamins</strong>.</p>
<ul>
<li>The lamina also interacts directly with <strong>chromatin</strong>, which itself <u>interacts with transmembrane proteins</u> of the inner nuclear membrane. </li>
<li>Together with  the lamina, these inner membrane proteins provide <strong>structural links</strong> between the DNA and the nuclear envelope.</li>
</ul>
<p>Later in mitosis, the nuclear envelope reassembles <u>on the surface of the daughter chromosomes.</u></p>
</blockquote>
<p>Outer membrane is normally continuous with ER.</p>
<p>Structure of nucleus:</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123201816.png" srcset="/img/loading.gif" lazyload alt="image-20200315123201816"></p>
<p>Inner membrane:</p>
<ul>
<li>anchor sites for chromatin and the nuclear lamina.</li>
</ul>
<p>Outer membrane: </p>
<ul>
<li>continuation of ER<ul>
<li>with ribosomes: rough ER</li>
<li>without ribosomes: smooth ER</li>
</ul>
</li>
</ul>
<h4 id="Nuclear-pore-complex-NPC-the-gate-of-nuclear-envelope"><a href="#Nuclear-pore-complex-NPC-the-gate-of-nuclear-envelope" class="headerlink" title="Nuclear pore complex (NPC): the gate of nuclear envelope"></a>Nuclear pore complex (NPC): the gate of nuclear envelope</h4><blockquote>
<p>Nuclear Pore Complexes Perforate (穿孔于,在…上打眼) the Nuclear Envelop</p>
<ul>
<li>Large and elaborate <strong>nuclear pore complexes (NPCs)</strong> perforate the nuclear envelope in all eukaryotes. Each NPC is composed of a set of approximately 30 different proteins, or <strong>nucleoporins</strong></li>
<li>Some of the scaffold nucleoporins (see Figure 12–8) are structurally related to <strong>vesicle coat protein</strong> complexes, such as <strong>clathrin</strong> and <strong>COPII coatomer</strong></li>
</ul>
</blockquote>
<p>Transport between cytosol and nucleus occurs via the nuclear pore</p>
<p>The nuclear pore complex (NPC) is the gate into the nucleus, <strong>∼</strong> 3000-4000 NPC&#x2F;cell</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123358834.png" srcset="/img/loading.gif" lazyload alt="image-20200315123358834"></p>
<p>negative stained NPCs after detergent extraction</p>
<h4 id="Nuclear-pores-complex-structural-organization-Features"><a href="#Nuclear-pores-complex-structural-organization-Features" class="headerlink" title="Nuclear pores complex structural organization&#x2F;Features"></a>Nuclear pores complex structural organization&#x2F;Features</h4><p>Each NPC is a protein complex containing <strong>∼</strong> 30 types proteins called <strong>nucleoporins</strong> (phentolamine-glycine: FG-repeat). </p>
<ul>
<li>Each nucleoporin is present in <u>multiple copies</u>, which in total is 500-1000 nucleoporins&#x2F;NPC (internal structural symmetry)</li>
</ul>
<p>Some formed <strong>passages</strong> for small molecules, some participate in floating structures, some form a central channel with meshwork in the middle</p>
<p><strong>Cytoplasmic fibrils</strong>（细胞质原纤维） and <strong>nuclear basket fibrils</strong>（核篮纤维） are proteins located around the entrance of pore</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123552995.png" srcset="/img/loading.gif" lazyload alt="image-20200315123552995"></p>
<h4 id="Nuclear-pores-are-size-selective-channels"><a href="#Nuclear-pores-are-size-selective-channels" class="headerlink" title="Nuclear pores are size selective channels"></a>Nuclear pores are size selective channels</h4><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123636586.png" srcset="/img/loading.gif" lazyload alt="image-20200315123636586" style="zoom:67%;" />

<p>Features:</p>
<ul>
<li><p>Transport at amazing speed: up to <strong>∼</strong>1000 macromolecules&#x2F;sec, in <u>bi-directional manner</u></p>
<blockquote>
<p> each NPC can transport up to 1000 macromolecules per second and can transport in both directions at the same time. </p>
<p>Each NPC contains <strong>aqueous passages</strong>（水通道）, through which small water-soluble molecules can diffuse passively</p>
</blockquote>
</li>
<li><p>Small molecules (&lt;5 KDa) passive diffusion freely</p>
<blockquote>
<p>Small  molecules (5000 daltons or less) diffuse in so fast that we can consider the nuclear envelope freely permeable to them</p>
</blockquote>
</li>
<li><p>Big molecules (&gt; 60 KDa) cannot enter by passive diffusion</p>
<blockquote>
<p>Proteins larger than 60,000 daltons cannot enter by passive diffusion</p>
</blockquote>
</li>
<li><p>How are big proteins (DNA polymerases, RNA polymerases, ribosomal proteins, etc.) transported across nuclear envelope?</p>
</li>
</ul>
<h4 id="Nuclear-import-needs-special-proteins-and-sequences"><a href="#Nuclear-import-needs-special-proteins-and-sequences" class="headerlink" title="Nuclear import needs special proteins and sequences"></a>Nuclear import needs special proteins and sequences</h4><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123757815.png" srcset="/img/loading.gif" lazyload alt="image-20200315123757815" style="zoom:67%;" />

<p><strong>Nuclear importing signals (nuclear localization signal, NLS):</strong></p>
<ul>
<li>One or two short sequences that are rich in K (Lysine) and R (Arginine)</li>
<li>Can be anywhere in the amino acid sequence.</li>
<li>Can form loops or patches on the protein surface (3-D structure, signal patch)</li>
</ul>
<blockquote>
<p>Nuclear Localization Signals Direct Nuclear Proteins to the  Nucleus</p>
<ul>
<li>Sorting signals called <strong>nuclear localization signals (NLSs)</strong> are responsible for the selectivity of this active nuclear import process</li>
<li>Nuclear localization signals can be located almost anywhere in the amino acid sequence and are thought to form loops or patches on the protein surface</li>
</ul>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123828344.png" srcset="/img/loading.gif" lazyload alt="image-20200315123828344" style="zoom:67%;" />

<p>Mutation of key amino acids in the NLS destroys the signal</p>
<ul>
<li>the mutant protein will lose its nuclear localization and remains in the cytosol!</li>
</ul>
<h5 id="1-Visualizing-active-import-through-NPCs"><a href="#1-Visualizing-active-import-through-NPCs" class="headerlink" title="(1) Visualizing active import through NPCs"></a>(1) Visualizing active import through NPCs</h5><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315124108601.png" srcset="/img/loading.gif" lazyload alt="image-20200315124108601" style="zoom: 80%;" />

<p>Gold particles were coated with NLS-peptides (time-course experiment)</p>
<p>Coated gold particles were injected into cells. Cells were fixed at different time points after injection to track gold particle movement</p>
<p>Coated gold particle can be seen to:</p>
<ul>
<li>interact with NPC (A-C)</li>
<li>get through the NPC (D)</li>
</ul>
<h3 id="2-Substances-That-Can-be-Transported"><a href="#2-Substances-That-Can-be-Transported" class="headerlink" title="2. Substances That Can be Transported"></a>2. Substances That Can be Transported</h3><p>Molecules that are transport (imported) into the nucleus: </p>
<ol>
<li>histones</li>
<li>DNA&#x2F;RNA polymerases</li>
<li>gene regulatory proteins (transcription factors) </li>
<li>RNA processing proteins</li>
<li>ribosomal proteins, etc.</li>
</ol>
<p>Molecules that are transported (exported) out of the nucleus:</p>
<ol>
<li>mature RNAs </li>
<li>tRNAs</li>
<li>pre-ribosome subunits </li>
<li>proteins, etc</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123102956.png" srcset="/img/loading.gif" lazyload alt="image-20200315123102956"></p>
<h3 id="3-Nuclear-import-receptors-importins"><a href="#3-Nuclear-import-receptors-importins" class="headerlink" title="3. Nuclear import receptors (importins)"></a>3. Nuclear import receptors (importins)</h3><blockquote>
<p>To initiate nuclear import, most nuclear localization signals must be recognized by <strong>nuclear import receptors</strong>, sometimes called <strong>importins</strong>, most of which are encoded by a family of related genes</p>
<p>Nuclear import receptors do not always bind to nuclear proteins directly. Additional <strong>adaptor proteins</strong>  can form a bridge between the import receptors and the nuclear localization signals on the proteins to be transported</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315131625378.png" srcset="/img/loading.gif" lazyload alt="image-20200315131625378"></p>
<p>Features</p>
<ol>
<li>Soluble cytosolic proteins </li>
<li>Bridging NLS-cargo and FG-repeats of nucleoporins at the NPC</li>
<li>Some receptors use adaptor proteins for “bridging”: the adaptors recognize NLS-cargo proteins and have also an own NLS to interact with the receptor</li>
</ol>
<h3 id="4-Nuclear-export"><a href="#4-Nuclear-export" class="headerlink" title="4. Nuclear export"></a>4. Nuclear export</h3><blockquote>
<p>The transport system relies on <strong>nuclear export signals</strong> on the macromolecules to be exported, as well as on complementary <strong>nuclear export receptors</strong>,  or <strong>exportins</strong></p>
<p><u>Many nuclear export receptors are structurally related to nuclear import receptors</u>, and they are encoded by the same gene family of <strong>nuclear transport receptors</strong>, or <strong>karyopherins（胡萝卜素）</strong></p>
</blockquote>
<p>Features</p>
<ul>
<li><p>Nuclear export signals relatively poorly characterized, not many facts are known</p>
</li>
<li><p>Some have a <strong><u>leucine</u> rich sequence</strong> </p>
</li>
<li><p><strong>Nuclear export receptors</strong> facilitates export.</p>
</li>
<li><p>Nuclear export receptors are structurally <u>related</u> to nuclear import receptors.</p>
</li>
</ul>
<h4 id="Role-of-GTPase"><a href="#Role-of-GTPase" class="headerlink" title="Role of GTPase"></a>Role of GTPase</h4><p>Nuclear import&#x2F;export consume energy, which is controlled by Ran GTPase (nuclear transport cycle)</p>
<p>The “GTPase cycle” of RAN:</p>
<ul>
<li>Ran-GDP (cytosol)</li>
<li>Ran-GTP (nucleus)</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315132930210.png" srcset="/img/loading.gif" lazyload alt="image-20200315132930210" style="zoom:120%;" />

<blockquote>
<p>A cytosolic <strong>GTPase-activating protein (GAP)</strong> triggers GTP hydrolysis and thus converts Ran-GTP to Ran-GDP, </p>
<p>A nuclear <strong>guanine exchange factor (GEF)</strong>  promotes the exchange of GDP for GTP and thus converts Ran-GDP to Ran-GTP.</p>
<p>Because Ran-GAP is located in the cytosol and Ran-GEF is located in the nucleus  where it is anchored to chromatin, the cytosol contains mainly Ran-GDP, and the nucleus contains mainly Ran-GTP</p>
<p>This gradient of the two conformational forms of Ran drives nuclear transport in the appropriate direction</p>
<ul>
<li>This occurs whether or not these receptors are loaded with appropriate cargo</li>
</ul>
<p>Because the Ran-GDP in the cytosol does not bind to import (or export)  receptors, unloading occurs only on the nuclear side of the NPC</p>
</blockquote>
<p>Transport occurs according to gradient cytosol&#x2F;nucleus</p>
<ol>
<li>Cytosolic Ran-GDP is “activated” by a Ran-GEF (Guanine nucleotide exchange factor) in the nucleus. This converts Ran-GDP to Ran-GTP</li>
<li>In the cytosol, Ran-GTP is stimulated by Ran-GAP (GTPase-activating protein) to hydrolyse its GTP (conversion to Ran-GDP)</li>
</ol>
<blockquote>
<p>In addition to its crucial role in nuclear transport, the Ran GTPase also acts as a positional marker for chromatin during cell division,  when the nuclear and cytosolic components intermix. </p>
<p><u>Because Ran-GEF remains bound to <strong>chromatin</strong> when the nuclear envelope breaks down, Ran molecules  close to chromatin are mainly in their GTP-bound conformation</u></p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133059493.png" srcset="/img/loading.gif" lazyload alt="image-20200315133059493"></p>
<blockquote>
<p>Nuclear export occurs by a similar mechanism, except that Ran-GTP in the nucleus promotes cargo <u>binding to the export receptor</u>, rather than promoting cargo dissociation</p>
</blockquote>
<h3 id="5-Shuttling-Proteins"><a href="#5-Shuttling-Proteins" class="headerlink" title="5. Shuttling Proteins"></a>5. Shuttling Proteins</h3><blockquote>
<p>Some proteins contain both nuclear localization signals and nuclear export signals. These proteins continually shuttle back and forth between the nucleus and the cytosol. </p>
<p>The relative rates of their import and export determine the steady state localization of such <strong>shuttling proteins</strong></p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133140728.png" srcset="/img/loading.gif" lazyload alt="image-20200315133140728" style="zoom: 50%;" />

<h4 id="Shuttling-proteins-can-be-regulated-through-protein-phosphorylation"><a href="#Shuttling-proteins-can-be-regulated-through-protein-phosphorylation" class="headerlink" title="Shuttling proteins can be regulated through protein phosphorylation"></a>Shuttling proteins can be regulated through protein phosphorylation</h4><blockquote>
<p>In many cases, cells  control transport by regulating nuclear localization and export signals—turning them on or off, often by <u>phosphorylation of amino acids close to the signal sequences</u></p>
</blockquote>
<p>The control of nuclear import of the transcription factor NF-AT (nuclear factor of activated T cells) during T cell activation is due to altered Ca2+ levels: </p>
<ul>
<li>high Ca2+ activates</li>
<li>declining&#x2F;low Ca2+ levels deactivate</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133302836.png" srcset="/img/loading.gif" lazyload alt="image-20200315133302836" style="zoom:80%;" />

<p>For example: </p>
<ul>
<li><p>Nuclear <u>import (X)</u> and nuclear <u>export (X)</u> signal can be masked by its phosphorylation states.</p>
</li>
<li><p>Regulation over time, not via the location, due to altered Ca2+ levels dependent on the physiological condition</p>
</li>
<li><p>signal sequence 附近被磷酸化之后，NF-AT 蛋白上的 nuclear export signal 显现出来，nuclear import signal 被block掉，因此蛋白被转运出细胞；去磷酸化之后 nuclear import signal 显现出来，nuclear export signal 被block掉，因此蛋白被转运进细胞。</p>
</li>
</ul>
<h4 id="Shuttling-proteins-can-be-regulated-metabolically"><a href="#Shuttling-proteins-can-be-regulated-metabolically" class="headerlink" title="Shuttling proteins can be regulated metabolically"></a>Shuttling proteins can be regulated metabolically</h4><blockquote>
<p>Other transcription regulators are bound to <strong>inhibitory cytosolic proteins</strong> that either <u>anchor them in the cytosol</u> (through interactions with the cytoskeleton or specific organelles) or <u>mask their nuclear localization signals</u> so that they cannot interact with nuclear import receptors.</p>
</blockquote>
<p>e.g. cholesterol level in the cell regulate cholesterol biogenesis: <u>negative feedback</u></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133453297.png" srcset="/img/loading.gif" lazyload alt="image-20200315133453297" style="zoom:67%;" />

<p>SREBP: steroid response element binding protein</p>
<p>SCAP: SREBP cleavage activation protein</p>
<h3 id="7-Nucleic-Acid-Transport"><a href="#7-Nucleic-Acid-Transport" class="headerlink" title="7. Nucleic Acid Transport"></a>7. Nucleic Acid Transport</h3><blockquote>
<p>Cells control the export of RNAs from the nucleus in a similar way. snRNAs, miRNAs, and tRNAs bind to the <u>same family of nuclear export receptors</u> just discussed, and they use the <u>same Ran-GTP gradient</u> to fuel the transport process.</p>
<p>The export of mRNAs out of the nucleus uses a different mechanism. mRNAs are exported as <strong>large assemblies with nuclear proteins</strong></p>
<ul>
<li>Newly transcribed messenger RNA and ribosomal RNA are exported  from the nucleus as parts of <strong>large ribonucleoprotein complexes.</strong> </li>
<li>Because nuclear localization signals are not removed, <strong>nuclear proteins</strong> can be imported repeatedly,  as is required each time that the nucleus reassembles after mitosis</li>
</ul>
</blockquote>
<h3 id="8-Summary-Nuclear-and-cytoplasmic-Protein-Transport"><a href="#8-Summary-Nuclear-and-cytoplasmic-Protein-Transport" class="headerlink" title="8. Summary: Nuclear and cytoplasmic Protein Transport"></a>8. Summary: Nuclear and cytoplasmic Protein Transport</h3><p>Nuclear Pores ~9 nm formed from nucleoporin proteins</p>
<p>Smaller proteins (~5 kDa or less) may diffuse freely; larger proteins must be imported by “gated pore” mechanism</p>
<p>Proteins targeted for nucleus have nuclear localization signals </p>
<p>Most NLS must be recognized &amp; bound by nuclear import receptors or nuclear export receptors</p>
<h2 id="Transport-to-mitochondria-and-chloroplasts"><a href="#Transport-to-mitochondria-and-chloroplasts" class="headerlink" title="Transport to mitochondria and chloroplasts"></a>Transport to mitochondria and chloroplasts</h2><blockquote>
<p>Translocation into Mitochondria Depends on Signal Sequences and Protein Translocators</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133702482.png" srcset="/img/loading.gif" lazyload alt="image-20200315133702482" style="zoom:67%;" />

<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133717771.png" srcset="/img/loading.gif" lazyload alt="image-20200315133717771" style="zoom:67%;" />

<h3 id="1-Protein-import-into-mitochondria"><a href="#1-Protein-import-into-mitochondria" class="headerlink" title="1. Protein import into mitochondria"></a>1. Protein import into mitochondria</h3><blockquote>
<p>New mitochondria and chloroplasts are produced by the growth of pre-existing organelles, followed by fission (discussed in Chapter 14). <u>The growth depends mainly on the import of proteins from the cytosol</u></p>
</blockquote>
<p>Why is targeting to mitochondria and chloroplasts important?</p>
<ul>
<li>Even though mitochondria and chloroplast have their own genome and protein synthesis machinery, the majority of proteins are encoded in the nucleus and are therefore synthesized in the cytosol and then imported afterwards.</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133808204.png" srcset="/img/loading.gif" lazyload alt="image-20200315133808204" style="zoom:80%;" />

<blockquote>
<p><strong>mitochondrial proteins</strong> are first fully synthesized as mitochondrial precursor proteins in the cytosol and then translocated into mitochondria by a <strong>post-translational</strong> mechanism</p>
<ul>
<li>Many proteins entering the matrix space contain a signal sequence at their N-terminus that a signal peptidase rapidly removes after import. </li>
<li>Other imported proteins, including all outer membrane and many inner membrane and intermembrane space proteins, have internal signal sequences that are <strong>not</strong> removed.</li>
</ul>
</blockquote>
<h4 id="Transport-to-mitochondria-and-chloroplasts-Features"><a href="#Transport-to-mitochondria-and-chloroplasts-Features" class="headerlink" title="Transport to mitochondria and chloroplasts Features"></a>Transport to mitochondria and chloroplasts Features</h4><p>Features</p>
<ul>
<li>Proteins with appropriate signal sequence squeeze through protein <strong>translocators</strong>.</li>
<li>They unfold and re-fold later with the help of chaperones proteins</li>
</ul>
<h4 id="Molecular-chaperons-facilitate-protein-folding"><a href="#Molecular-chaperons-facilitate-protein-folding" class="headerlink" title="Molecular chaperons facilitate protein folding"></a>Molecular chaperons facilitate protein folding</h4><p>Many molecular chaperones are called <u>heat-shock proteins (hsp)</u> 温度升高时会增量表达</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133958465.png" srcset="/img/loading.gif" lazyload alt="image-20200315133958465"></p>
<h4 id="Protein-localization-in-mitochondria"><a href="#Protein-localization-in-mitochondria" class="headerlink" title="Protein localization in mitochondria"></a>Protein localization in mitochondria</h4><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134029646.png" srcset="/img/loading.gif" lazyload alt="image-20200315134029646"></p>
<p>Mitochondria have two independent reaction rooms:</p>
<ul>
<li><p>IMS (intermembrane space) and matrix</p>
</li>
<li><p>Targeting to mitochondria includes targeting to:</p>
<ul>
<li>OM: Outer membrane</li>
<li>IMS: Intermembrane space</li>
<li>IM: Inner membrane</li>
<li>matrix</li>
</ul>
</li>
</ul>
<h3 id="2-Mitochondria-localization-signal"><a href="#2-Mitochondria-localization-signal" class="headerlink" title="2. Mitochondria localization signal"></a>2. Mitochondria localization signal</h3><blockquote>
<p>The signal sequences that direct precursor proteins <u>into the mitochondrial matrix space</u> are best understood. </p>
<ul>
<li>They all form an <strong>amphiphilic α helix</strong>, in which positively charged residues cluster on one side of the helix, while uncharged  hydrophobic residues cluster on the opposite side.</li>
</ul>
</blockquote>
<p>Features:</p>
<p>Mitochondria precursor proteins are <u>fully synthesized in the cytosol</u> before their import</p>
<p>Chaperones keep the proteins in an <u>unfolded</u> “import-competent” state and guide them to the TOM&#x2F;TOC complexes, their import is referred to as “transmembrane transport”</p>
<p>An <strong>N-terminal peptide</strong> directs many precursor proteins to matrix, then, this signal peptide it is <strong>cleaved upon arrival</strong></p>
<p>An internal peptide directs all precursor proteins into the outer membrane, many to be in the inner membrane.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134242797.png" srcset="/img/loading.gif" lazyload alt="image-20200315134242797"></p>
<h4 id="need-for-a-receptor-protein"><a href="#need-for-a-receptor-protein" class="headerlink" title="need for a receptor protein"></a>need for a receptor protein</h4><blockquote>
<p><strong>Specific receptor proteins</strong> that  initiate protein translocation recognize this configuration rather than the precise amino acid sequence of the signal sequence</p>
</blockquote>
<p>The mitochondrial signal an amino acid sequence that forms an <strong>amphiphilic α-helix.</strong></p>
<ul>
<li>one side of the helix is hydrophilic the other side is hydrophobic</li>
</ul>
<p><u>Positively charged amino acid on one side, Nonpolar amino acid on other side</u></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134330394.png" srcset="/img/loading.gif" lazyload alt="image-20200315134330394"></p>
<p>The amphiphilic α-helix of the signal binds <u>with its hydrophobic face to a hydrophobic groove</u> in the receptor</p>
<h4 id="Different-Protein-Translocators"><a href="#Different-Protein-Translocators" class="headerlink" title="Different Protein Translocators"></a>Different Protein Translocators</h4><p>Different protein translocators facilitate protein import and transport across the membranes in mitochondria</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134441231.png" srcset="/img/loading.gif" lazyload alt="image-20200315134441231" style="zoom:80%;" />

<ol>
<li><strong>TOM</strong> (Translocase <u>outer membrane</u> mitochondria) complex transfers proteins across the outer mitochondria membrane.</li>
<li><strong>TIM23</strong> (Translocase <u>inner membrane</u> mitochondria) complex transfer proteins across the inner mitochondria membrane. <ul>
<li>For <strong>single pass</strong> transmembrane protein</li>
</ul>
</li>
<li><strong>TOM &amp; TIM23</strong>, both function as receptors for signal peptides<ul>
<li>can form a transmembrane channels spanning both membranes</li>
</ul>
</li>
<li><strong>SAM</strong> (sorting and assembly complex mitochondria) complex helps to insert&#x2F;fold properly the <strong>β-barrel</strong> proteins <u>in outer membrane.</u> </li>
<li><strong>TIM22</strong>, acts similar to TIM23<ul>
<li>Acts for <strong>multi pass</strong> transmembrane protein</li>
</ul>
</li>
<li><strong>OXA</strong> (<u>Oxidase</u> assembly machinery) complex<ul>
<li>helps to <u>insert mitochondria-synthesized proteins</u> and others transported proteins <u>from the matrix into the inner membrane</u></li>
</ul>
</li>
</ol>
<h5 id="1-Proteins-import-into-the-mitochondrial-matrix-via-TOM-and-TIM23"><a href="#1-Proteins-import-into-the-mitochondrial-matrix-via-TOM-and-TIM23" class="headerlink" title="(1) Proteins import into the mitochondrial matrix via TOM and TIM23"></a>(1) Proteins import into the mitochondrial matrix via TOM and TIM23</h5><p>The precursor of the matrix protein is <u>passing both membranes at the same time;</u> the signal sequence works for TOM and for TIM23.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134803868.png" srcset="/img/loading.gif" lazyload alt="image-20200315134803868"></p>
<h6 id="Protein-import-into-the-mitochondrial-matrix-consume-energy"><a href="#Protein-import-into-the-mitochondrial-matrix-consume-energy" class="headerlink" title="Protein import into the mitochondrial matrix consume energy"></a>Protein import into the mitochondrial matrix consume energy</h6><p><strong>ATP hydrolysis</strong> and the <strong>membrane potential</strong> drive the protein import into the mitochondrial matrix.</p>
<ul>
<li><strong>ATP</strong> is consumed by the <strong>chaperones</strong> (<u>cytosolic <strong>hsp70</strong></u>) for “<u>releasing</u>” the precursor protein<ul>
<li>Chaperones <u>hydrolase ATP</u> via an ATPase domain!</li>
</ul>
</li>
</ul>
<p>Translocation of the <strong>positively charged</strong> signal sequence <u>depends on the membrane potential.</u></p>
<p>Mitochondrial <strong><u>hsp70 chaperon</u></strong> also consumes ATP during “pulling” (binding &amp; release cycles) the precursor protein inside</p>
<blockquote>
<p>The <strong>mitochondrial hsp70</strong> is part of a multisubunit protein assembly that is <u>bound to the <strong>matrix side</strong></u> of the <strong>TIM23</strong> complex and <u>acts as a motor to pull the precursor protein into the matrix space.</u></p>
<p>The <strong>hsp70</strong> then undergoes a conformational change and releases  the protein chain in an ATP-dependent step, exerting a <strong>ratcheting&#x2F;pulling force</strong>  on the protein being imported</p>
</blockquote>
<blockquote>
<p>After the initial interaction with <strong>mitochondrial hsp70</strong>, many imported matrix proteins are passed on to another chaperone protein, <strong>mitochondrial hsp60</strong>. </p>
<p>As  discussed in Chapter 6, <strong>hsp60</strong> helps the <u>unfolded polypeptide chain to fold</u> by binding and releasing it through cycles of ATP hydrolysis.</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134920072.png" srcset="/img/loading.gif" lazyload alt="image-20200315134920072"></p>
<h5 id="2-Outer-membrane-porins-β-barrel-proteins-are-inserted-with-the-help-of-SAM-via-the-intermembrane-space-IMS"><a href="#2-Outer-membrane-porins-β-barrel-proteins-are-inserted-with-the-help-of-SAM-via-the-intermembrane-space-IMS" class="headerlink" title="(2) Outer membrane porins (β-barrel proteins) are inserted with the help of SAM via the intermembrane space (IMS)"></a>(2) Outer membrane porins (β-barrel proteins) are inserted with the help of SAM via the intermembrane space (IMS)</h5><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134950433.png" srcset="/img/loading.gif" lazyload alt="image-20200315134950433"></p>
<p><u>Unfolded <strong>β</strong>-barrel proteins</u> are translocated via TOM to the intermembrane space (IM) where they are stabilized by <strong>chaperones</strong>.</p>
<p>Then, they are inserted in to the outer membrane via the <strong>SAM complex</strong>, which also <u>aids folding (finally).</u></p>
<blockquote>
<p>the <strong>TOM complex</strong> cannot integrate porins into the lipid bilayer</p>
<p>They then bind to the <strong>SAM complex</strong> in the outer membrane, which both inserts them into the outer membrane and helps them fold properly</p>
</blockquote>
<p>Bacteria and mitochondria use similar mechanisms to insert porins (β-barrel proteins) into the outer membrane, but…</p>
<ul>
<li>The proteins have <u>different</u> origins in mitochondria compared to bacteria.<ul>
<li>Mitochondria: proteins reach the IMS from the “outside” via <strong>TOM</strong></li>
<li>Bacteria: proteins reach the peri-bacterial space（细菌周围的空间） from “inside” via <strong>Sec complex</strong>, the bacteria use the <strong>BAM complex</strong> to translocate the protein from intermembrane space into outer membrane</li>
</ul>
</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135243281.png" srcset="/img/loading.gif" lazyload alt="image-20200315135243281"></p>
<h5 id="3-Protein-import-into-the-inner-membrane-IM"><a href="#3-Protein-import-into-the-inner-membrane-IM" class="headerlink" title="(3) Protein import into the inner membrane (IM)"></a>(3) Protein import into the inner membrane (IM)</h5><h6 id="Targeting-of-single-pass-transmembrane-proteins"><a href="#Targeting-of-single-pass-transmembrane-proteins" class="headerlink" title="Targeting of single-pass transmembrane proteins:"></a>Targeting of <u>single-pass</u> transmembrane proteins:</h6><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135343086.png" srcset="/img/loading.gif" lazyload alt="image-20200315135343086" style="zoom: 67%;" />

<ul>
<li>TOM</li>
<li>TIM23</li>
<li>signal sequence</li>
<li>hydrophobic <u>stop-transfer sequence</u></li>
</ul>
<blockquote>
<p>A <strong>hydrophobic amino acid sequence,</strong> strategically placed after the <strong>N-terminal signal sequence</strong>, acts as a <strong>stop-transfer sequence</strong>, <u>preventing further translocation across the inner membrane</u></p>
<ul>
<li>A matrix signal peptidase then <strong>removes</strong> the N-terminal signal  sequence, <u>exposing</u> a <strong>hydrophobic sequence</strong> at the new N-terminus.</li>
<li>Many proteins that use these pathways to the inner membrane <u>remain anchored there through their hydrophobic signal sequence</u></li>
</ul>
</blockquote>
<h6 id="Targeting-of-multi-pass-transmembrane-proteins"><a href="#Targeting-of-multi-pass-transmembrane-proteins" class="headerlink" title="Targeting of multi-pass transmembrane proteins"></a>Targeting of multi-pass transmembrane proteins</h6><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135433254.png" srcset="/img/loading.gif" lazyload alt="image-20200315135433254" style="zoom:67%;" />

<ul>
<li><p>TOM</p>
</li>
<li><p>Chaperones</p>
</li>
<li><p>TIM22</p>
</li>
<li><p>Internal signal sequence</p>
</li>
</ul>
<p>Multi-pass IM proteins that function as metabolite transporters contain internal signal sequences and sneak through the <strong>TOM</strong> complex as <u>a loop</u>. </p>
<p>IMS <strong>chaperones</strong> bind the precursor in the intermembrane space and guide the protein to the <strong>TIM22</strong> complex, which is specialized for the insertion of multi-pass IM proteins.</p>
<h5 id="4-Protein-import-into-the-inner-membrane-IM-from-the-matrix"><a href="#4-Protein-import-into-the-inner-membrane-IM-from-the-matrix" class="headerlink" title="(4) Protein import into the inner membrane (IM) from the matrix"></a>(4) Protein import into the inner membrane (IM) from the matrix</h5><p>Protein import into the IM from the matrix (e.g. oxidase on the electron transport chain)</p>
<p>Extremely important strategy for insertion of proteins that are synthesized in the matrix!</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135542184.png" srcset="/img/loading.gif" lazyload alt="image-20200315135542184" style="zoom:67%;" />

<ul>
<li>TOM</li>
<li>TIM23</li>
<li>OXA</li>
<li>signal sequence</li>
<li>signal sequence</li>
</ul>
<h5 id="5-Protein-import-into-the-inter-membrane-space-IMS"><a href="#5-Protein-import-into-the-inter-membrane-space-IMS" class="headerlink" title="(5) Protein import into the inter membrane space (IMS)"></a>(5) Protein import into the inter membrane space (IMS)</h5><h6 id="Import-of-soluble-proteins"><a href="#Import-of-soluble-proteins" class="headerlink" title="Import of soluble proteins"></a>Import of soluble proteins</h6><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135634079.png" srcset="/img/loading.gif" lazyload alt="image-20200315135634079"></p>
<ul>
<li><p>TOM</p>
</li>
<li><p>TIM23</p>
</li>
<li><p>IM-localizing protease&#x2F;peptidase</p>
</li>
</ul>
<h6 id="Targeting-of-soluble-protein-to-the-IMS-via-TOM-folding-in-the-IMS-by-Mia40"><a href="#Targeting-of-soluble-protein-to-the-IMS-via-TOM-folding-in-the-IMS-by-Mia40" class="headerlink" title="Targeting of soluble protein to the IMS via TOM &amp; folding in the IMS by Mia40"></a>Targeting of soluble protein to the IMS via TOM &amp; folding in the IMS by Mia40</h6><blockquote>
<p>Certain intermembrane-space proteins that contain <strong>cysteine motifs</strong> are  imported by a yet different route. </p>
<p>These proteins form a transient covalent <strong>disulfide bond</strong> to the <strong>Mia40 protein</strong></p>
<p>The imported proteins are  then released in an oxidized form containing intrachain disulfide bonds. <strong>Mia40</strong> becomes reduced in the process, and is <u>then reoxidized by passing electrons to the electron transport chain in the inner mitochondrial membrane.</u></p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135713963.png" srcset="/img/loading.gif" lazyload alt="image-20200315135713963" style="zoom:67%;" />

<ul>
<li><p>TOM</p>
</li>
<li><p>Mia40 (Mia40: Mitochondrial IMS assembly machinery )</p>
</li>
</ul>
<p>Mia40 forms a covalent intermediate through an intermolecular disulfide bond, which helps folding&#x2F;pulling of the transported protein through the TOM complex.</p>
<p>Mia40 is reduced in the process, and is then reoxidized by the electron transport chain.</p>
<h4 id="Summary-Protein-import-into-the-inner-membrane-IM-and-the-intermembrane-space-IMS-of-mitochondria"><a href="#Summary-Protein-import-into-the-inner-membrane-IM-and-the-intermembrane-space-IMS-of-mitochondria" class="headerlink" title="Summary: Protein import into the inner membrane (IM) and the intermembrane space (IMS) of mitochondria"></a>Summary: Protein import into the inner membrane (IM) and the intermembrane space (IMS) of mitochondria</h4><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135838566.png" srcset="/img/loading.gif" lazyload alt="image-20200315135838566"></p>
<h4 id="Summary-Protein-import-into-mitochondria"><a href="#Summary-Protein-import-into-mitochondria" class="headerlink" title="Summary: Protein import into mitochondria"></a>Summary: Protein import into mitochondria</h4><p>Mitochondrial proteins that are encoded by nuclear genes possess mitochondrial import sequences.</p>
<p>Transport from the cytosol into the mitochondria is mediated by a TOM complex and two TIM complexes.</p>
<p>The SAM complex mediates proper folding of outer membrane proteins with β-barrel structures.</p>
<p>Another complex (OXA) mediates insertion of mitochondrial-gene encoded proteins into inner membrane &amp; also contributes to insertion of some nuclear-gene encoded proteins.</p>
<p>Proteins must be unfolded, either by chaperones or by specialized unfolding proteins.</p>
<h3 id="2-Protein-localization-in-plastids"><a href="#2-Protein-localization-in-plastids" class="headerlink" title="2. Protein localization in plastids"></a>2. Protein localization in plastids</h3><blockquote>
<p>Protein transport into <strong>chloroplasts</strong> resembles transport into mitochondria.  </p>
<p>Both processes occur <strong>post-translationally</strong>, use separate translocation complexes in each membrane, require energy, and use <strong>amphiphilic N-terminal signal sequences</strong> that are <u>removed after use.</u></p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135938745.png" srcset="/img/loading.gif" lazyload alt="image-20200315135938745"></p>
<p>Chloroplasts have three independent reaction rooms:</p>
<ul>
<li>IMS (intermembrane space)</li>
<li>Stroma</li>
<li>Thylakoids</li>
</ul>
<p>Targeting to chloroplasts includes targeting to: </p>
<ul>
<li>OM: Outer Membrane</li>
<li>IMS: Intermembrane Space</li>
<li>IM: Inner membrane</li>
<li>Stroma</li>
<li>TM: Thylakoid membrane</li>
<li>thylakoid lumen</li>
</ul>
<h4 id="Plastid-localization-signal"><a href="#Plastid-localization-signal" class="headerlink" title="Plastid localization signal"></a>Plastid localization signal</h4><p>Features</p>
<ul>
<li><p>Chloroplast precursor proteins are fully synthesized in the cytosol before their import.</p>
</li>
<li><p>Chaperones keep the proteins in an <strong>unfolded</strong> “import-competent” state and guide them to the <strong>TOC complexes,</strong> their import is referred to as “transmembrane transport”.</p>
</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140043854.png" srcset="/img/loading.gif" lazyload alt="image-20200315140043854"></p>
<p>An <strong>N-terminal peptide</strong> directs many precursor proteins to matrix, then, this signal peptide it is <strong>cleaved upon arrival.</strong></p>
<p>An internal peptide directs all precursor proteins into the outer membrane, many to be in the inner membrane and intermembrane (thylakoids).</p>
<h4 id="Translocation-of-chloroplast-precursor-proteins-into-the-thylakoid-lumen"><a href="#Translocation-of-chloroplast-precursor-proteins-into-the-thylakoid-lumen" class="headerlink" title="Translocation of chloroplast precursor proteins into the thylakoid lumen"></a>Translocation of chloroplast precursor proteins into the thylakoid lumen</h4><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140106818.png" srcset="/img/loading.gif" lazyload alt="image-20200315140106818" style="zoom:67%;" />

<h4 id="Translocation-into-the-thylakoid-lumen-or-thylakoid-membrane"><a href="#Translocation-into-the-thylakoid-lumen-or-thylakoid-membrane" class="headerlink" title="Translocation into the thylakoid lumen or thylakoid membrane"></a>Translocation into the thylakoid lumen or thylakoid membrane</h4><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140136609.png" srcset="/img/loading.gif" lazyload alt="image-20200315140136609"></p>
<ol>
<li><strong>Sec pathway</strong>, so called because it uses components that are <u>homolog</u>s of <strong>Sec proteins</strong> which mediate protein translocation across the bacterial plasma membrane</li>
<li><strong>SRP (Signal Recognition Particle)-like pathway</strong>, so called because it uses a chloroplast homolog of the signal-recognition particle (SRP)</li>
<li><strong>TAT (twin arginine translocation) pathway</strong>, so called because two arginines are critical in the signal sequences that direct proteins into this pathway, which depends on the <strong>H+ gradient</strong> across the thylakoid membrane</li>
<li><strong>Spontaneous insertion pathway</strong> that seems not to require any protein translocator</li>
</ol>
<h4 id="Summary-Protein-import-into-chloroplast"><a href="#Summary-Protein-import-into-chloroplast" class="headerlink" title="Summary: Protein import into chloroplast"></a>Summary: Protein import into chloroplast</h4><p>Chloroplasts have 2 further targets for proteins:</p>
<ul>
<li>Thylakoid membrane</li>
<li>Thylakoid lumen</li>
<li>$\rarr$<u>Two signal peptides</u> direct proteins to the thylakoids (For signal peptide is cleaved after traanslocation):<ul>
<li>the first to reach the stroma</li>
<li>the second to reach the thylakoids (thylakoid-specific).</li>
</ul>
</li>
</ul>
<p>Import receptors are chloroplast–specific:</p>
<ul>
<li><strong>TOC</strong>: Translocase <strong>outer</strong> membrane chloroplast</li>
<li><strong>TIC</strong>: Translocase <strong>inner</strong> membrane chloroplast</li>
<li>$\rarr$ some chloroplast-specific signal peptides &amp; signals</li>
</ul>
<p>Import is <strong>NOT</strong> driven by electrochemical energy, and is <u>driven by <strong>ATP and GTP hydrolysis</strong>.</u> </p>
<h3 id="3-Transport-to-peroxisomes"><a href="#3-Transport-to-peroxisomes" class="headerlink" title="3. Transport to peroxisomes"></a>3. Transport to peroxisomes</h3><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140329409.png" srcset="/img/loading.gif" lazyload alt="image-20200315140329409" style="zoom:50%;" />

<p>Features</p>
<ul>
<li>Proteins with appropriate signal sequence squeeze through protein translocators</li>
</ul>
<h4 id="Function-of-Peroxisomes"><a href="#Function-of-Peroxisomes" class="headerlink" title="Function of Peroxisomes"></a>Function of Peroxisomes</h4><blockquote>
<p><strong>Virtually all eukaryotic cells have peroxisomes.</strong> </p>
<p>They contain oxidative enzymes, such as <strong>catalase</strong> (过氧化氢酶) and <strong>urate oxidase</strong> (尿酸氧化酶), at such high concentrations that,  in some cells, the peroxisomes stand out in electron micrographs because of the presence of a crystalloid protein core</p>
</blockquote>
<p>Peroxisomes detoxify chemicals, metabolize fatty acids and synthesize key lipid</p>
<blockquote>
<p>Peroxisomes are so named because they usually contain one or more enzymes  that use <strong>molecular oxygen</strong> to remove <strong>hydrogen atoms</strong> from specific organic substrates (designated here as R) in an oxidation reaction that produces <strong>hydrogen peroxide (H2O2)</strong>: $RH_2 + O_2 → R + H_2O_2$</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140411652.png" srcset="/img/loading.gif" lazyload alt="image-20200315140411652" style="zoom:80%;" />

<p>(TEM image of peroxisomes in a liver cell from a rat with paracrystalline, electron dense inclusions (enzyme urate oxidase))</p>
<p>Peroxisome plays several vital functions:</p>
<ol>
<li><p>Contain catalase and oxidase</p>
<ul>
<li>Catalase use H2O2 to oxidize a variety of substrates by the “peroxidation” reaction : $H_2O_2 + R′ H_2 → R′ + 2H_2O$</li>
<li>Remove excess H2O2 by : $2H_2O_2 → O_2 + 2H_2O$</li>
</ul>
</li>
<li><p>Oxidize fatty acid by <strong>β</strong>-oxidation, producing <u>acetyl-CoA</u></p>
<blockquote>
<p>A major function of the oxidation reactions performed in peroxisomes is the breakdown of fatty acid molecules. The process, called <strong>β oxidation</strong></p>
<ul>
<li>β oxidation shortens the <strong>alkyl chains</strong> (烷基链) of fatty acids sequentially in blocks of two carbon atoms at a time,  thereby converting the fatty acids to acetyl CoA</li>
</ul>
<ol>
<li><p>In mammalian cells, β oxidation occurs in <strong>both mitochondria and peroxisomes</strong>; </p>
</li>
<li><p>in yeast and plant cells,  however, this essential reaction occurs <strong>exclusively in peroxisomes</strong>.</p>
</li>
</ol>
</blockquote>
</li>
<li><p>Metabolize many harmful molecules</p>
<ul>
<li>Phenols 酚类</li>
<li>Formaldehyde 甲醛</li>
<li>Alcohol</li>
</ul>
</li>
<li><p>It is particularly important in liver and kidney cells, where peroxisomes <strong>detoxify</strong> various toxic molecules that enter the bloodstream.</p>
</li>
</ol>
<p><strong>∼</strong> 25% of the ethanol we drink is oxidized to acetaldehyde in this way!</p>
<h5 id="1-Peroxisomes-are-important-for-the-synthesis-of-plasmalogen-缩醛磷脂"><a href="#1-Peroxisomes-are-important-for-the-synthesis-of-plasmalogen-缩醛磷脂" class="headerlink" title="(1) Peroxisomes are important for the synthesis of plasmalogen 缩醛磷脂"></a>(1) Peroxisomes are important for the synthesis of plasmalogen 缩醛磷脂</h5><blockquote>
<p>An essential biosynthetic function of animal peroxisomes is to catalyze the first reactions in the formation of plasmalogens, which are the most abundant class of phospholipids in b</p>
<ul>
<li>Plasmalogen deficiencies cause profound abnormalities in the myelination of nerve-cell axons, which is one reason why many peroxisomal disorders lead to neurological disease.</li>
</ul>
</blockquote>
<p>peroxisomes catalyze the initial step in the synthesis of a special lipid called <strong>plasmalogen</strong>. </p>
<p>Plasmalogen localizes to the cell membrane of cells that <u>form myelin sheaths around axons.</u></p>
<p>Important substance for myelin sheath to protect&#x2F;insulate axons of nerve cells. </p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140747340.png" srcset="/img/loading.gif" lazyload alt="image-20200315140747340"></p>
<p>Plasmalogen(缩醛磷脂): </p>
<ul>
<li>Ethanolamine head group</li>
<li>Glycerol phosphate backbone with attached long-chain fatty acid</li>
<li>Contain unusual fatty alcohol that is attached via an <u>ether linkage（醚键）</u></li>
</ul>
<h5 id="2-Peroxisomes-in-plant-cells-carry-out-unique-functions"><a href="#2-Peroxisomes-in-plant-cells-carry-out-unique-functions" class="headerlink" title="(2) Peroxisomes in plant cells carry out unique functions"></a>(2) Peroxisomes in plant cells carry out unique functions</h5><blockquote>
<p>Peroxisomes are also important in plants. Two types of plant peroxisomes have been studied extensively. </p>
<ol>
<li>One is present in leaves, where it participates in <strong>photorespiration</strong></li>
<li>The other type of peroxisome is present in <strong>germinating seeds</strong>, where it <u>converts the fatty acids stored in seed lipids into the sugars</u> needed for the growth of the young plant</li>
</ol>
<p>Because this conversion of fats to sugars is accomplished by a series of reactions known as the <strong>glyoxylate cycle</strong> (乙醛酸循环), these <strong>peroxisomes</strong> are also called <strong>glyoxysomes</strong></p>
</blockquote>
<p>Peroxisome: in plant leaves close association with chloroplasts to facilitate the exchange of materials between these organelles during <strong>photorespiration</strong> (for details about photorespiration, see lecture 14). </p>
<p>Glyoxysome (乙醛酸循环体) : in germinating cells association with the lipid droplets that store fat.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140918958.png" srcset="/img/loading.gif" lazyload alt="image-20200315140918958"></p>
<h4 id="Proteins-inserted-post-translationally-into-peroxisomes"><a href="#Proteins-inserted-post-translationally-into-peroxisomes" class="headerlink" title="Proteins inserted post-translationally into peroxisomes"></a>Proteins inserted post-translationally into peroxisomes</h4><p>Features</p>
<ul>
<li>Require a <strong>signal sequence</strong> recognized by another set of proteins in the peroxisome membrane which form a channel through with peroxisomal proteins enter a peroxisome.</li>
</ul>
<p>The proteins that target proteins to peroxisomes and form the channel are called <strong>Pex</strong> proteins.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140955411.png" srcset="/img/loading.gif" lazyload alt="image-20200315140955411" style="zoom:80%;" />

<h5 id="1-Proteins-transport-into-peroxisomes"><a href="#1-Proteins-transport-into-peroxisomes" class="headerlink" title="(1) Proteins transport into peroxisomes"></a>(1) Proteins transport into peroxisomes</h5><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141010362.png" srcset="/img/loading.gif" lazyload alt="image-20200315141010362"></p>
<p>Short C-terminal signal peptide (<strong>Ser–Lys–Leu</strong>) directs protein import into peroxisome</p>
<blockquote>
<p><strong>A specific sequence of three amino acids (Ser–Lys–Leu)</strong> located at the C-terminus of many peroxisomal proteins functions as an import signal</p>
<p>Numerous distinct proteins, called <strong>peroxins</strong>, participate in the import process, which is driven by ATP hydrolysis. A complex of at least six different peroxins forms a protein translocator in the peroxisome membrane.</p>
</blockquote>
<p> <img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141032145.png" srcset="/img/loading.gif" lazyload alt="image-20200315141032145"></p>
<p>Cytosolic receptor protein recognize signal sequence. </p>
<p>Several peroxins dock on cytosolic surface as a membrane translocator</p>
<p>Import requires ATP. </p>
<h6 id="Mutations-in-Pex-proteins-lead-to-loss-of-peroxisomes"><a href="#Mutations-in-Pex-proteins-lead-to-loss-of-peroxisomes" class="headerlink" title="Mutations in Pex proteins lead to loss of peroxisomes"></a>Mutations in Pex proteins lead to loss of peroxisomes</h6><p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141054126.png" srcset="/img/loading.gif" lazyload alt="image-20200315141054126"></p>
<blockquote>
<p>One soluble import receptor, the peroxin <strong>Pex5</strong> recognizes the <u><strong>C-terminal</strong> peroxisomal import signal</u>. It accompanies its cargo all the way into peroxisomes and, after cargo release, cycles back to the cytosol</p>
<ul>
<li>After delivering its cargo to the peroxisome lumen, Pex5 undergoes <strong>ubiquitylation</strong>.</li>
<li>This  modification is required to release <strong>Pex5</strong> back into the cytosol, where the ubiquitin is removed. An ATPase composed of <strong>Pex1</strong> and <strong>Pex6</strong> harnesses the energy of ATP hydrolysis to <u>help release Pex5 from peroxisomes.</u></li>
</ul>
<p> A defect in <strong>Pex7</strong>, the receptor for the <strong>N-terminal import signal</strong>, causes a <strong>milder peroxisomal disease</strong>.</p>
</blockquote>
<h6 id="Pex-mutations-cause-Zelleweger-Syndrome-and-demyelination-of-nerves"><a href="#Pex-mutations-cause-Zelleweger-Syndrome-and-demyelination-of-nerves" class="headerlink" title="Pex mutations cause Zelleweger Syndrome and demyelination of nerves"></a>Pex mutations cause Zelleweger Syndrome and demyelination of nerves</h6><p>Zelleweger Syndrome </p>
<ul>
<li>Disease is named after Hans Zellweger (1909–1990), a Swiss-American pediatrician</li>
<li>inherited genetic disease due to mutation in peroxin <strong>Pex5</strong></li>
<li>The mutation affects protein import in peroxisomes and the biogenesis of peroxisomes</li>
<li>severe abnormality in brain, liver and kidney development</li>
<li>rarely survives till 6-month old</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141149036.png" srcset="/img/loading.gif" lazyload alt="image-20200315141149036"></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141156980.png" srcset="/img/loading.gif" lazyload alt="image-20200315141156980"></p>

                
              </div>
            
            <hr/>
            <div>
              <div class="post-metas my-3">
  
    <div class="post-meta mr-3 d-flex align-items-center">
      <i class="iconfont icon-category"></i>
      

<span class="category-chains">
  
  
    
      <span class="category-chain">
        
  <a href="/categories/Academic-Notebook/" class="category-chain-item">Academic Notebook</a>
  
  
    <span>></span>
    
  <a href="/categories/Academic-Notebook/Cell-Biology/" class="category-chain-item">Cell Biology</a>
  
  

  

      </span>
    
  
</span>

    </div>
  
  
    <div class="post-meta">
      <i class="iconfont icon-tags"></i>
      
        <a href="/tags/Cell-Biology/" class="print-no-link">#Cell Biology</a>
      
    </div>
  
</div>


              
  

  <div class="license-box my-3">
    <div class="license-title">
      <div>L9 Intracellular transport I - Nucleus, Mitochondria， Chloroplast, Peroxisome</div>
      <div>https://zhenyumi.github.io/posts/12aa050d/</div>
    </div>
    <div class="license-meta">
      
        <div class="license-meta-item">
          <div>作者</div>
          <div>向海</div>
        </div>
      
      
        <div class="license-meta-item license-meta-date">
          <div>发布于</div>
          <div>2020年7月2日</div>
        </div>
      
      
      
        <div class="license-meta-item">
          <div>许可协议</div>
          <div>
            
              
              
                <a class="print-no-link" target="_blank" href="https://creativecommons.org/licenses/by/4.0/">
                  <span class="hint--top hint--rounded" aria-label="BY - 署名">
                    <i class="iconfont icon-cc-by"></i>
                  </span>
                </a>
              
            
          </div>
        </div>
      
    </div>
    <div class="license-icon iconfont"></div>
  </div>



              
                <div class="post-prevnext my-3">
                  <article class="post-prev col-6">
                    
                    
                      <a href="/posts/fe4294e6/" title="L10 Intracellular transport II - ER and Golgi">
                        <i class="iconfont icon-arrowleft"></i>
                        <span class="hidden-mobile">L10 Intracellular transport II - ER and Golgi</span>
                        <span class="visible-mobile">上一篇</span>
                      </a>
                    
                  </article>
                  <article class="post-next col-6">
                    
                    
                      <a href="/posts/b6a297cf/" title="L8 Ion gradients across the membrane">
                        <span class="hidden-mobile">L8 Ion gradients across the membrane</span>
                        <span class="visible-mobile">下一篇</span>
                        <i class="iconfont icon-arrowright"></i>
                      </a>
                    
                  </article>
                </div>
              
            </div>

            
          </article>
        </div>
      </div>
    </div>

    <div class="side-col d-none d-lg-block col-lg-2">
      
  <aside class="sidebar" style="margin-left: -1rem">
    <div id="toc">
  <p class="toc-header">
    <i class="iconfont icon-list"></i>
    <span>目录</span>
  </p>
  <div class="toc-body" id="toc-body"></div>
</div>



  </aside>


    </div>
  </div>
</div>





  



  



  



  



  


  
  









    

    
      <a id="scroll-top-button" aria-label="TOP" href="#" role="button">
        <i class="iconfont icon-arrowup" aria-hidden="true"></i>
      </a>
    

    
      <div class="modal fade" id="modalSearch" tabindex="-1" role="dialog" aria-labelledby="ModalLabel"
     aria-hidden="true">
  <div class="modal-dialog modal-dialog-scrollable modal-lg" role="document">
    <div class="modal-content">
      <div class="modal-header text-center">
        <h4 class="modal-title w-100 font-weight-bold">搜索</h4>
        <button type="button" id="local-search-close" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body mx-3">
        <div class="md-form mb-5">
          <input type="text" id="local-search-input" class="form-control validate">
          <label data-error="x" data-success="v" for="local-search-input">关键词</label>
        </div>
        <div class="list-group" id="local-search-result"></div>
      </div>
    </div>
  </div>
</div>

    

    
  </main>

  <footer>
    <div class="footer-inner">
  
    <div class="footer-content">
       <a href="https://hexo.io" target="_blank" rel="nofollow noopener"><span>Hexo</span></a> <i class="iconfont icon-love"></i> <a href="https://github.com/fluid-dev/hexo-theme-fluid" target="_blank" rel="nofollow noopener"><span>Fluid</span></a> 
    </div>
  
  
  
</div>

  </footer>

  <!-- Scripts -->
  
  <script  src="https://lib.baomitu.com/nprogress/0.2.0/nprogress.min.js" ></script>
  <link  rel="stylesheet" href="https://lib.baomitu.com/nprogress/0.2.0/nprogress.min.css" />

  <script>
    NProgress.configure({"showSpinner":false,"trickleSpeed":100})
    NProgress.start()
    window.addEventListener('load', function() {
      NProgress.done();
    })
  </script>


<script  src="https://lib.baomitu.com/jquery/3.6.4/jquery.min.js" ></script>
<script  src="https://lib.baomitu.com/twitter-bootstrap/4.6.1/js/bootstrap.min.js" ></script>
<script  src="/js/events.js" ></script>
<script  src="/js/plugins.js" ></script>





  
    <script  src="/js/img-lazyload.js" ></script>
  




  
<script>
  Fluid.utils.createScript('https://lib.baomitu.com/tocbot/4.20.1/tocbot.min.js', function() {
    var toc = jQuery('#toc');
    if (toc.length === 0 || !window.tocbot) { return; }
    var boardCtn = jQuery('#board-ctn');
    var boardTop = boardCtn.offset().top;

    window.tocbot.init(Object.assign({
      tocSelector     : '#toc-body',
      contentSelector : '.markdown-body',
      linkClass       : 'tocbot-link',
      activeLinkClass : 'tocbot-active-link',
      listClass       : 'tocbot-list',
      isCollapsedClass: 'tocbot-is-collapsed',
      collapsibleClass: 'tocbot-is-collapsible',
      scrollSmooth    : true,
      includeTitleTags: true,
      headingsOffset  : -boardTop,
    }, CONFIG.toc));
    if (toc.find('.toc-list-item').length > 0) {
      toc.css('visibility', 'visible');
    }

    Fluid.events.registerRefreshCallback(function() {
      if ('tocbot' in window) {
        tocbot.refresh();
        var toc = jQuery('#toc');
        if (toc.length === 0 || !tocbot) {
          return;
        }
        if (toc.find('.toc-list-item').length > 0) {
          toc.css('visibility', 'visible');
        }
      }
    });
  });
</script>


  <script src=https://lib.baomitu.com/clipboard.js/2.0.11/clipboard.min.js></script>

  <script>Fluid.plugins.codeWidget();</script>


  
<script>
  Fluid.utils.createScript('https://lib.baomitu.com/anchor-js/5.0.0/anchor.min.js', function() {
    window.anchors.options = {
      placement: CONFIG.anchorjs.placement,
      visible  : CONFIG.anchorjs.visible
    };
    if (CONFIG.anchorjs.icon) {
      window.anchors.options.icon = CONFIG.anchorjs.icon;
    }
    var el = (CONFIG.anchorjs.element || 'h1,h2,h3,h4,h5,h6').split(',');
    var res = [];
    for (var item of el) {
      res.push('.markdown-body > ' + item.trim());
    }
    if (CONFIG.anchorjs.placement === 'left') {
      window.anchors.options.class = 'anchorjs-link-left';
    }
    window.anchors.add(res.join(', '));

    Fluid.events.registerRefreshCallback(function() {
      if ('anchors' in window) {
        anchors.removeAll();
        var el = (CONFIG.anchorjs.element || 'h1,h2,h3,h4,h5,h6').split(',');
        var res = [];
        for (var item of el) {
          res.push('.markdown-body > ' + item.trim());
        }
        if (CONFIG.anchorjs.placement === 'left') {
          anchors.options.class = 'anchorjs-link-left';
        }
        anchors.add(res.join(', '));
      }
    });
  });
</script>


  
<script>
  Fluid.utils.createScript('https://lib.baomitu.com/fancybox/3.5.7/jquery.fancybox.min.js', function() {
    Fluid.plugins.fancyBox();
  });
</script>


  <script>Fluid.plugins.imageCaption();</script>

  
      <script>
        if (!window.MathJax) {
          window.MathJax = {
            tex    : {
              inlineMath: { '[+]': [['$', '$']] }
            },
            loader : {
              load: ['ui/lazy']
            },
            options: {
              renderActions: {
                insertedScript: [200, () => {
                  document.querySelectorAll('mjx-container').forEach(node => {
                    let target = node.parentNode;
                    if (target.nodeName.toLowerCase() === 'li') {
                      target.parentNode.classList.add('has-jax');
                    }
                  });
                }, '', false]
              }
            }
          };
        } else {
          MathJax.startup.document.state(0);
          MathJax.texReset();
          MathJax.typeset();
          MathJax.typesetPromise();
        }

        Fluid.events.registerRefreshCallback(function() {
          if ('MathJax' in window && MathJax.startup.document && typeof MathJax.startup.document.state === 'function') {
            MathJax.startup.document.state(0);
            MathJax.texReset();
            MathJax.typeset();
            MathJax.typesetPromise();
          }
        });
      </script>
    

  <script  src="https://lib.baomitu.com/mathjax/3.2.2/es5/tex-mml-chtml.js" ></script>

  <script  src="/js/local-search.js" ></script>





<!-- 主题的启动项，将它保持在最底部 -->
<!-- the boot of the theme, keep it at the bottom -->
<script  src="/js/boot.js" ></script>


  

  <noscript>
    <div class="noscript-warning">博客在允许 JavaScript 运行的环境下浏览效果更佳</div>
  </noscript>
</body>
</html>
