---
title: L3  Isolation of DNA and RNAs
tags:
  - Genetic Engineering
categories:
  - Academic Notebook
  - Genetic Engineering
author: 向海
banner_img: >-
  https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/post/post2.jpg
top: false
math: true
comments: true
hide: false
abbrlink: 44c53e21
date: 2020-11-11 08:33:51
updated: 2020-11-11 08:33:52
excerpt: 2020-11-11 08:33:52
---

# 一、The Basic Lab Equipment

Getting to know common lab equipment

1. Centrifuge
2. Fridge and Freezer
3. Vortex
4. Water Bath
5. PCR cycler, Thermo cycler
6. Gel tank, Gel tray and comb
7. Electrophoresis Power Supply
8. Pipette and Pipette Tip 
9. Rotator
10. Petri-dish
11. Incubator
12. Dry bath
13. Sonicator
14. Shaker
15. Tube rack
16. Filter
17. Eppendorf Tube
18. Falcon Tube
19. Nanodrop
20. Rotor, Fixed Angle,
21. Swinging Bucket

# 二、Isolation of DNA and RNAs

## General Considerations

The considerations behind a DNA extraction procedure:

1. Maximize DNA recovery
2. Remove inhibitors (which may interfere with down-stream applications)
3. Remove or inhibit nucleases
4. Maximize the quality of DNA

### 1. The Amount of DNA

The amount of DNA we can get

1. The PCR reactions call for on average 1 ng of DNA (single or double stranded)
2. Many of the commercially available kits are sensitive below 1 ng of DNA(100-250 pg)
3. A southern blot will need ~ 1 to 10 ug of DNA.

The amount of DNA we can recover (The DNA in different subjects)

1.  A haploid human genome contains 3.2*10^9^ nucleotides
2. Molecular weight of dsDNA=(No. of Nucleotides x 607.4) + 157.9, MW of human genome = 1.94 * 10^12^ g/mol
   + 6.02×10²³ molecule per mol
3.  A haploid human cell contains (1.94 * 10^12^ / 6.02×10²³)*10^12^ = 3.2 pg of DNA
4. The average WBC of an adult is 5 - 10 * 10^3^ Cells of 1μl Blood, equals to 60ng DNA

However, the yield of DNA is not a big problem for we have PCR.

### 2. The Major Steps

1. Cell Lysis
   + Add lysis buffer to lyse the cell, protect the DNA/RNA from degradation
2. Separate the nucleic acids from Proteins/saccharide
   + Phenol-chloroform extraction / binding of DNA/RNA to column
3. Obtaining DNA/RNA which is dissolved in small amount of solution.
   + Elution, precipitation


1. Cell Lysis: 

   1. Purpose: Break the cell and release the cell content.

   2. Some important reagents’ function: 

      1. EDTA: EDTA buffer inhibits some metal-dependent enzymes such as DNAseI.

      2. Tris-Cl: The buffering reagent, maintain the overall pH.

      3. Salt: (Nacl and Kcl) Provide salt and proper osmotic pressure.

      4. SDS: The ionic detergent. Disrupt the cell membrane and denature the protein. In order to promote the release of cell content.
2. Phenol-Chloroform Extraction: 
   1. Function: Separate the DNA molecules from other unnecessary substances (protein, carbohydrates, lipids, etc.)
   2. Some important reagents’ function: 
      1. Phenol: Insolubilize protein and denature proteins.
      2. Chloroform: Denature lipids and separate the organic and aqueous phase.
      3. Isoamyl alcohol (Alternative): Prevent foaming in the interphase and emusification, therefore we can get clear sample.
3. DNA Precipitation: 
   1. Function: Collect the DNA molecule for the future experiments.
   2. Some important reagents’ function: 
      1. Salts: Naturalize the negative charge in DNA molecule and therefore make it less hydrophilic. 
      2. 70% Ethanol: Wash away the unnecessary reagents used in previous step, such as salts, organic reagents, etc. 
      3. TE buffer: Dissolves DNA and then protect it from degradation.

### 3. The Reagents Used in DNA/RNA Isolation

#### Cell Lysis Buffer

**Cell lysis buffer** **– typical lysis buffer**

1. Buffering reagent 
   + Tris, HEPES, MES, PIPES. 10-100mM
2. Detergent Nonionic detergent, Ionic detergent 
   + Nonionic detergent: Triton X-100, NP40, Tween 20
   + Ionic detergent: SDS, CTAB, Deoxycholate, Concentration: 0.5%-2%
   + Function: Disrupt membrane and denature proteins.
3. Salt 
   + NaCl, KCl
     + physiological concentration – 100mM to 1M
     + Provide salt and osmotic pressure.
4. Chelating reagent 
   +  EDTA
     +  EDTA buffer inhibits metal-dependent enzymes such as **DNase I**
5. Antioxidant
   + 2-Mercaptoethanol, DTT
     + providing reducing environment, or to reduce disulfide bonds

**Definition of** **nonionic detergent**

+ Any of a class of synthetic detergents (as long-chain ether derivatives or esters of alcohols or phenols) that are <u>neither anionic nor cationic</u> but produce electrically neutral colloidal particles in solution

<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20200926134112694.png" alt="image-20200926134112694" style="zoom: 50%;" />

**Definition of** **ionic detergent**

+ Ionic detergents have a **hydrophilic head group** that is <u>charged</u> and can be either negatively (anionic) or positively (cationic) charged. Ionic detergents are used for the <u>complete disruption</u> of <u>cellular structures</u> and <u>denaturation of proteins</u> for separation during gel electrophoresis. Ionic detergents bind with protein molecules, masking their native charge and rendering the protein molecules the overall charge of the ionic detergent.

<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20200926134202581.png" alt="image-20200926134202581" style="zoom:50%;" />

#### Used in DNA/RNA Isolation

**Separate the nucleic acids from Proteins/polysaccharide**

1. Organic Extraction (Phenol-Chloroform)
2. Non-Organic (Proteinase K and Salting out)
3. Column based method

The method utilized may be sample dependent, technique dependent, or analyst preference

<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20200926134701804.png" alt="image-20200926134701804" style="zoom:100%;" />

##### (1) Phenol-chloroform extraction

**Deproteinization:**

+ Phenol can denature the proteins, which will then be precipitated at the interphase or <u>trapped in the organic phase.</u>

![image-20200926134813350](https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20200926134813350.png)

**Phenol:**

+ DNA is insoluble in **phenol** because phenol is a **non-polar** **solution**
+ **Protein** has both polar and non-polar group present in it because of the long chain of different amino acids. Different amino acids have different groups present on their side chain. Also, the folding of the protein into the secondary, tertiary and quaternary structure depends on the polarity of the amino acids. 
  + <u>The bonds between amino acids are broken by the addition of phenol and protein get denatured</u>
+ Denature DNA and insolubilize protein. 

**Chloroform:**

+ chloroform allows <u>proper separation of the organic phase and aqueous phase</u> which <u>keeps DNA protected into the aqueous phase</u>. Chloroform denatures the lipid as well.

**Isoamyl alcohol**:

+ In the phenol-chloroform DNA extraction method, Isoamyl alcohol helps in <u>reducing foaming</u>（防止起泡沫） between interphase. It prevents the emulsification of a solution. 
+ The **liquid phase** contains **DNA** and the **organic phase** contains <u>lipid, proteins and other impurities</u>. The **precipitated protein** denatured and coagulated between both these phases (**interface**). This will create the cloudy, whitishfoam between interphase. The anti-foaming agent, isoamyl alcohol stabilized the interphase by removing the foaming. This will increase the purity of DNA.

#### Used in DNA/RNA Extraction

Obtaining DNA/RNA which is dissolved in small amount of solution. → elution, precipitation

The overall principle of **ethanol precipitation** : **elution, precipitation**

+ Nucleic acids are <u>hydrophilic</u> due to the <u>negatively charged phosphate</u> (PO3-) groups along the sugar phosphate backbone.
+ Because of these charges, polar molecules, like DNA or RNA, can <u>interact electrostatically with the water molecules</u>, allowing them to easily dissolve in water.

Role of Salts:

+ **positively charged** sodium ions <u>neutralize</u> the **negative charge** on the PO3- groups on the nucleic acids, making the molecule far <u>less hydrophilic</u>, and therefore much less soluble in water
+ H~2~O is very polar $\rarr$ salt is hard to meet DNA $\rarr$ ethanol

The role of the ethanol (70%):

+ Water has a high **dielectric constant**, which makes it fairly <u>difficult</u> for the Na+ and PO3- to come together. 
+ **Ethanol** on the other hand has <u>a much lower dielectric （非传导性的） constant</u>, making it much easier for Na+ to interact with the PO3-, shield it’s charge and make the nucleic acid less hydrophilic, causing it to drop out of solution.

The role of temperature in ethanol precipitation:

+ Incubation of the nucleic acid/salt/ethanol mixture at low temperatures (e.g. -20 or -80C) is commonly cited in protocols as necessary in protocols. 
+ However, according to (Molecular Cloning, A Laboratory Manual 2nd Edition), this is not required, as nucleic acids at concentrations as low as 20ng/mL will precipitate at 0-4C so incubation for 15-30 minutes on ice is sufficient

**Glycogen**:

+ Glycogen is a polymer which is used to <u>trap the nucleic acids</u> (DNA, rRNA,RNA...). In ethanol, Glycogen is insoluble so it forms polymer structure and traps the nucleic acids and forms pellet with nucleic acids trapped during the centrifugation... 

Choice of salt:

1. Use **Sodium acetate** (0.3M final conc, pH 5.2) for routine DNA precipitations
2. Use **Sodium chloride** (0,2M final conc) for DNA samples <u>containing SDS</u> since NaCl keeps SDS soluble in 70% ethanol so it won’t precipitate with the DNA.
3. Use **Ammonium acetate** (2M final conc) for the <u>removal of dNTPs</u>, but do not use for preparation of DNA for T4 polynucleotide kinase reactions as ammonium ions inhibit the enzyme
4. Use **Lithium Chloride** (0.8M final conc) for <u>RNA</u>. This is because 2.5-3 volumes of ethanol should be used for RNA precipitation and LiCl is more soluble in ethanol than NaAc so will not precipitate
   + but beware – chloride ions will inhibit protein synthesis and DNA polymerase so LiCl is no good for RNA preps for in vitro translation or reverse transcription. In these cases, use NaAc.

### 4. Column based purification

![image-20200926141158907](https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20200926141158907.png)

+ Step 1: Cell Lysis
+ Step 2: Purification – Binding nucleic acids to the column
+ Step 3: Washing
+ Step 4: Dry Spin for ethanol-free DNA and RNA
+ Step 5: The final frontier – Elution

#### The Cell Lysis

what’s the difference compared with phenol/chloroform method ?

+ The key is **chaotropic salts**（促溶盐；离液盐；离序盐） - guanidine HCL, guanidine thiocyanate, urea, and lithium perchlorate.

<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20200926141428211.png" alt="image-20200926141428211" style="zoom:50%;" />

+ The term "**chaotropic**" means **chaos-forming**, a term which in biochemistry usually refers to a compound's ability to <u>disrupt the regular hydrogen bond structures in water</u>
+ Hydrogen bonding profoundly affects the **secondary structure of polymers** such as DNA, RNA, and proteins
+ Hydrogen bonding also affects how water-soluble a molecule is. With the addition of chaotropic ions to the **nucleic acid**, the ordered structure of water molecules of its <u>**hydrate shell** is destroyed.</u>

The Lysis Buffers:

+ Lysis buffers contain a high concentration of **chaotropic salts**
+ Chaotropes have two important roles in nucleic acid extraction
  1. Firstly, they <u>destabilize</u> hydrogen bonds, van der Waals forces and hydrophobic <u>interactions</u>, leading to <u>destabilization of proteins, including nucleases</u>.
  2. Secondly, they <u>disrupt the association of nucleic acids with water</u>, thereby <u>providing optimal conditions for their transfer to silica.</u>

One possible explanation for functions of salt:

![image-20200926142530844](https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20200926142530844.png)

The exact mechanism how the silicon adsorption columns work is not quite clear. One idea is as follows. Chaotropic agent guandium HCl, which disrupt water structure, allow positively charged ions to form <u>a salt bridge between the negatively charged silica</u> and <u>the negatively charged DNA backbone under high salt conditions.</u>

#### Purification

Step 2: Purification – Binding nucleic acids to the column

1. Addition of **ethanol** in the cleared lysate. 
   + The addition of alcohol (or sometimes isopropanol) will further enhance and influence the binding of nucleic acids to the silica.
   + Too much ethanol will bring down degraded material and small species that will influence UV260 readings.
   + Too little ethanol may impede washing of the salt from the membrane

#### Washing

Washing-To wash away protein and salt residues

1. There are typically two wash steps. 
   2. The first wash will often include a low concentration of chaotropic salts to remove residual proteins and pigments. 
   2. This is always followed with an ethanol wash to remove the salts.

####  Dry Spin for ethanol

Dry Spin for ethanol-free DNA and RNA

#### Elution

For DNA extraction, 10 mM Tris at pH 8-9 is typically used. DNA is more stable at a slightly basic pH and will dissolve faster in a buffer than water. This is true even for DNA pellets. Water tends to have a lower pH of 4-5, and high molecular weight DNA may not completely rehydrate in the short time used for elution

##  DNA extraction vs. RNA extraction

For DNA – things to avoid: **shearing force**, DNase, Get rid of RNA if necessary.

+ Avoid sharing force is to keep the target DNA fragment intact 

For RNA – **Inactivate endogenous RNase and avoid exogenous RNase**, DNase digestion is almost always necessary for downstream applications.

## Trizol RNA extraction

<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20200926143239125.png" alt="image-20200926143239125" style="zoom:50%;" />

Trizol RNA extraction: An variety of Phenol/Chloroform method

Chomczynski P, Sacchi N. Single-step method of RNA isolation by acid guanidinium thiocyanate-phenol-chloroform extraction. Anal Biochem. 1987 Apr;162(1):156-9.

4 M guanidinium thiocyanate， 25 mM sodium citrate， 0.5% (wt/vol) N-laurosylsarcosine (Sarkosyl)， 0.1 M 2-mercaptoethanol，50% Phenol, sodium acetate pH 4.0

![image-20200926143247971](https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20200926143247971.png)