---
title: L9 DNA&RNA Modifying Enzymes
tags:
  - Genetic Engineering
categories:
  - Academic Notebook
  - Genetic Engineering
author: 向海
banner_img: >-
  https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/post/post2.jpg
top: false
math: true
comments: true
hide: false
abbrlink: c74aa795
date: 2020-11-11 08:33:51
updated: 2020-11-11 08:33:59
excerpt: 2020-11-11 08:33:59
---

# 一、Polynucleotide Kinase & Alkaline Phosphatase

## T4 polynucleotide Kinase

**Origin: T4 phage**

**Feature:** 

1. Catalyzes the <u>transfer</u> of the <u>gamma-phosphate</u> from **ATP** to the **5'-OH group** of singleand double-stranded DNAs and RNAs.
2. The enzyme is also a 3'-phosphatase under appropriate condition

**Usage:**

1. 5' phosphorylation of DNA/RNA for subsequent ligation
2. End labeling DNA or RNA for probes and DNA sequencing
3. Phosphorylation of PCR primers
4. Removal of 3' phosphoryl groups

![image-20201108111839815](https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201108111839815.png)

## Alkaline Phosphatase

**Origin: Calf or Shrimp**

**Feature:**

1. Alkaline Phosphatase catalyzes the release of 5'and 3'-phosphate groups from DNA, RNA, and nucleotides.
2. This enzyme also removes phosphate groups from proteins
3. CIAP (Calf intestinal AP) & SAP (Shrimp AP).
4. Phosphatase needs to be inactivated before the DNA is used for ligation.

**Usage:** 

1. removes 3´ and 5´ phosphate group from DNA, RNA and dNTP
2. To prevent self-ligation

<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201108112342662.png" alt="image-20201108112342662" style="zoom: 50%;" />

# 二、Terminal Deoxynucleotidyl Transferase & Poly(A) Polymerase

## Terminal Deoxynucleotidyl Transferase

**Origin: Calf thymus**

**Feature:** 

1. TdT is a <u>template-independent</u> DNA polymerase that catalyzes the <u>repetitive addition</u> of <u>deoxyribonucleotides</u> to the **3'-OH** of oligodeoxyribonucleotides and single-stranded and double-stranded **DNA**

**Usage: **

1. To create single-stranded overhang of dsDNA – for gene cloning
2. DNA 3’end labling.
   + Creats a sticky end
   + ![image-20201108112512796](https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201108112512796.png)

## Poly(A) Polymerase

**Origin:** **E.coli** **or Yeast**

**Feature:** 

1. catalyzes the addition of <u>adenosine</u> to the <u>3' end</u> (-OH) of **RNA** in a sequence-independent fashion

**Usage: **

1. Labeling RNA 3’ end; facilitate reverse transcription, cloning & sequencing.

<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201108112749646.png" alt="image-20201108112749646" style="zoom: 33%;" />

# 三、Nucleases – DNase I, Exonuclease I, RNase A, RNase I, RNase, H, MNase

## Nucleases – DNase I

**Origin:** **E.coli** **or Yeast**

**Feature**: 

1. A nonspecific **endonuclease** that <u>degrades dsDNA, ssDNA and DNA in RNA:DNA hybrid.</u> 
2. It functions by hydrolyzing phosphodiester linkages, producing mono and oligonucleotides with a 5'-phosphate and a 3'-hydroxyl group

**Usage:** 

1. Preparation of DNA-free RNA. 
2. DNA labeling by nick-translation in conjunction with DNA Polymerase I 
3. Studies of DNA-protein interactions by DNase I footprinting

DNase I foot-printing to study DNA-protein interaction:

<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201108112907147.png" alt="image-20201108112907147" style="zoom:80%;" />

## Exonuclease I (*E. coli*) 

**Origin:** **E.coli**

**Feature:**

1. **3’ to 5’ exonuclease of ssDNA**
2. Catalyzes the removal of nucleotides from **linear single stranded** DNA in the 3' to 5' direction

<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201108113249263.png" alt="image-20201108113249263" style="zoom: 33%;" />

**Usage:**

1. <u>Remove excessive primers from PCR product</u>
   + Exo-SAP treatment
     + The **exonuclease I** removes leftover primers, while the S**hrimp Alkaline Phosphatase** removes any remaining dNTPs

![image-20201108113309170](https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201108113309170.png)

## RNase A

Degrades **single-stranded** <u>RNA at C and U residues.</u> 

Cleaves the phosphodiester bond between the 5'-ribose of a nucleotide and the phosphate group attached to the 3'-ribose of an adjacent pyrimidine nucleotide (C or U). 

Leave a **3’ end phosphate group** after clevage. 

<u>Resistant to heat, easy to refold – the enzyme that we should worry about in RNA extraction.</u>

![image-20201108113515554](https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201108113515554.png)

## RNase I

RNase I <u>degrades single-stranded</u> RNA to nucleoside 3´ monophosphates via 2´,3´ cyclic monophosphate intermediates by <u>cleaving between all dinucleotide pairs</u> 

The enzyme is completely <u>inactivated by heating</u> at 70°C for 15 minutes. 

A more controllable choice to digest RNA than RNase A

## RNase H

specifically <u>degrades the RNA</u> strand of an **RNA-DNA hybrid** to produce 5' phosphate terminated oligoribonucleotides and single-stranded DNA

**Application**: 

+ remove RNA from cDNA

<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201108113629276.png" alt="image-20201108113629276" style="zoom:50%;" />

## Micrococcal nuclease (MNase)

Exhibits exoand endo-5'-phosphodiesterase activities against DNA and RNA. ds/ss DNA/RNA. 

+ cut them all 

Preference for ATor AU-rich regions. 

<u>Strictly dependent on calcium</u> for digestion of RNA and DNA substrates. Precisely inactive the enzyme by adding EGTA

**Application: remove RNA from cDNA**

+ Remove nucleic acids from solution 
+ Release nucleosome from chromatin

## Proteinase K

Cloned from Engyodontium album, is a **nonspecific serine protease** that is useful for general digestion of proteins. 

Optimal activity between 6.5 and 9.5 

<u>Active under denaturing conditions</u>, SDS or urea 

<u>Active under high temperature</u>, 55-65 degree

**Application:** 

+ remove proteins from solution/cell extract (e.g. DNA or RNA extraction)

# Summary

1. **Polynucleotide Kinase** (Both a Kinase and phosphatase) 
2. **Alkaline Phosphatase** (You need to remove it before ligation) 
3. **Terminal Deoxynucleotidyl Transferase** (For add a stretch of A on DNA) 
4. Poly(A) Polymerase (For add a stretch of A on RNA). 
5. **Nucleases** – DNase I, Exonuclease I, RNase A, RNase I, RNase H, MNase (They all cut nucleic acids with different features，think of a scenario that you may use it).

